## SorghumBase Release 8.0
### Released: October 2024

SorghumBase (https://www.sorghumbase.org) is a USDA-ARS funded resource released to the public in 2021. The SorghumBase team works closely with the community to support stewardship of sorghum genomics data, establish best-practices on managing the data, and provide opportunities for networking and capacity building within the community. We maintain a database of scientific publications, genome sequences, gene structure annotations, and comparative genomic analyses integrated with curated data from [EBI's Gene Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments), [Bio-Analytical Resources' eFPseq browser](https://bar.utoronto.ca/), [Gramene's Plant Reactome](https://plantreactome.gramene.org), and [OZ Sorghum QTL Atlas](https://aussorgm.org.au/sorghum-qtl-atlas/).


## Summary

In its eighth release, SorghumBase (SB) features 46 new _Sorghum bicolor_ ssp. _bicolor_ breeding lines with an expired Plant Variety Protection (ex-PVP) Act certificate sequenced by a team led by Dr. Todd Michael at the Salk Institute to a total of 74 sorghum genomes. A total of 1,450,260 sorghum gene models are provided for the 37 sorghum genomes that are not ex-PVP, and 66,928 sorghum pan-genes were created using annotations from the 28 sorghum genomes in SB release 7. The sorghum pan-genes were in turn mapped to each of the 46 ex-PVP accessions by a team led by Dr. Doreen Ware at CSHL. 

We are also providing variation sampled with nearly 23 million additional variants in over 2,500 sorghum accessions. The set by [Kumar and colleagues (2024)](https://www.sorghumbase.org/paper/24501) consisted of 18.7 million single nucleotide polymorphism (SNPs) plus insertion deletions (indels) assesed by whole-genome sequencing (WGS) in 365 accessions including 332 individuals from the bioenergy association panel (BAP). The [Lasky et al (2015)](https://www.sorghumbase.org/paper/24502) set consisted on characterizing 2,327 landraces (1,943 of which are georeferenced) at 404,627 SNPs.

In addition, a large genotyping data set from the Gates-funded **Sorghum Genomics Toolbox** (SGT) Project (also known as the [Global Sorghum Initiative](https://www.globalsorghuminitiative.org/)) was provided by Dr. Nadia Shakoor from the Donald Danforth Plant Science Center, Dr. John Lovell from the HudsonAlpha Institute for Biotechnology, and Dr. Geoff Morris from Colorado State University. The set consists of 36.1 million SNPs called in 940 resequenced genomes, and is mapped to the JGI_V5.1 assembly. Please note that the JGI_V5.1 assembly is subject to the [Fort Lauderdale accord](https://www.sanger.ac.uk/wp-content/uploads/fortlauderdalereport.pdf) restrictions for not publishing major analyses before the data generators do.

In an effort to harmonize germplasm identifiers within SorghumBase and based on the recommendation of best practices used by the community and curators of major sorghum germplasm repositories such as GRIN and ICRISAT, where available, we provide genotyping data using standard germplasm identifiers (e.g., PI and IS numbers) and standard SNP identifiers (e.g., rsIDs). 

Accessions with genetic variants causing the loss of a gene's function (LOFs) are now organized by study or population, type of variant (e.g., stop gained/lost, frameshift, splice acceptor/donor) and homozygous/heterozygous state in the Germplasm tab of the gene.

Besides genomic resources, SorghumBase provides weekly updates of news items, conferences, community events and funded research projects. Our publications database now includes 1,026 sorghum papers, 13 of which have been [highlighted in News posts](https://www.sorghumbase.org/posts?categories=research-highlights) including an article co-authored by members of the SorghumBase Team describing the high-quality chromosome-scale genome assemblies of inbred lines Tx2783 and RTx436 [(Wang et al, 2024)](URL). News items and project pages are indexed and searchable, and include information on how to access related data on SorghumBase. A recently updated [user guide](https://www.sorghumbase.org/guides) is available from the site and feedback may be provided via a [contact form](https://www.sorghumbase.org/contact). Individual training sessions may be scheduled upon request. Please check the upcoming [events](https://www.sorghumbase.org/events) on the latest webinars, conferences, meetings, etc. relevant to the sorghum research community. You are always welcome to [contact us](https://www.sorghumbase.org/contact), if you are interested in hosting a training event, collaborating with the SorghumBase team, would like to provide feedback or ask questions. 

Funding for the project is provided by the Agricultural Research Service of the [U.S. Department of Agriculture](http://www.usda.gov/) (USDA ARS 8062-21000-051-00D). 


## What's New in release 8?

### New Data

***Genomes & Gene Annotations***

1) We are providing in this release, the genomes of 46 ex-PVP breeding lines (44 bicolor and 2 drummondii). Ex-PVP accessions are breeding lines with an expired Plant Variety Protection (PVP) Act certificate. These accessions were sequenced and assembled by the groups of Dr. Michael Todd of the Salk Institute and Dr. Nadia Shakoor of the Donald Danforth Plant Science Center. 

  - _Sorghum bicolor (L.) Moench subsp. bicolor_: PI543243 (PH256), PI543246 (PHB279), PI543247 (PHB349), PI544069 (PH352), PI554646 (PH288), PI554647 (PH355), PI554648 (PH356), PI554649 (PH410), PI554650 (PH449), PI554652 (PHB429), PI554654 (PHB432), PI555457 (PHB331), PI561926 (R160), PI562621 (PH538), PI562622 (PHB122), PI562623 (PHB118), PI562624 (PHA118), PI562625 (PHA122), PI574398 (PHA651), PI574406 (PHA151), PI574407 (PHB151), PI594354 (PHWIABQE), PI594355 (PHB1CGQF), PI596332 (PH387LM), PI596567 (PH4KNYE), PI601264 (PH227), PI601415 (PH232), PI601552 (A8087C), PI601553 (PH257), PI601554 (PH289), PI601555 (PH308), PI601556 (PH309), PI601557 (PH310), PI601716 (PH328), PI601717 (PH333), PI601718 (PHA82), PI601719 (PHA86), PI601720 (PHB82), PI601721 (PHB86), PI601743 (HP150), PI601744 (R159), PI601756 (R145), PI602599 (PHBCWNL), and PI602600 (PHBEPOYJ).

  - _S. bicolor (L.) Moench nothosubsp. drummondii (Steud.) de Wet ex Davidse_: PI564085 (WKMIII) and PI595221 (EXCELS235).


2) 66,928 sorghum pan-genes created using annotations from the 28 sorghum genomes in SB release 7 were mapped to each of the 46 ex-PVP accessions and are provided as a track on the corresponding ex-PVP browser pages.


***Genetic & Phenotypic Variation***

- Sorghum BTx623 (NCBIv3)

  1) [Kumar et al (2024)](https://www.sorghumbase.org/paper/24501) - 18.7 million SNPs and indels in 365 accessions including 332 BAP by WGS.

  2) [Lasky et al (2015)](https://www.sorghumbase.org/paper/24502) - approximately 400,000 SNPs in 1943 georeferenced landraces.

- Sorghum BTx623 (JGI_V5.1)

  1) Approximately 41 million rsIDs from the European Variation Archive (EVA) were mapped to SNPs lifted from the NCBIv3 to the JGI_V5.1 assembly.

  2) 36.1 million SNPs were called in 940 resequenced genomes from the Gates-funded [Global Sorghum Initiative](https://www.globalsorghuminitiative.org/). 
     
_Disclaimer_: The chromosome-scale assembly of Sorghum bicolor BTx623 known as Phytozome JGI version 5.1 (JGI_V5.1) is subject to the [Fort Lauderdale accord](https://www.sanger.ac.uk/wp-content/uploads/fortlauderdalereport.pdf) restrictions for publication of articles containing analyses of genomic data or genes prior to publication of a comprehensive genome analysis by JGI and/or its collaborators.

- Ex-PVP breeding lines

  * 41 million rsIDs were mapped from BTx623 v3.1 to the 46 ex-PVP breeding lines by the SorghumBase Team. 


***Regulation***

- Sorghum BTx623 (NCBIv3): ATAC-seq and Chip-seq data from [Lou et al (2019)](https://sorghumbase.org/paper/24376) for sorghum, maize, rice and barley.


### New Features

The Germplasm tab of the results of a gene query was reorganized to show LOF accessions grouped by study or population, type of variant, and homozygous/heterozygous state.


### New Papers & Funded Projects

- 73 sorghum papers were added to the [Publications](https://www.sorghumbase.org/publications) database with 13 of them described in more detail in [research highlights](https://www.sorghumbase.org/posts?categories=research-highlights). Corresponding title, abstract, authors, keywords, PubMed id, and doi are searchable.

- 16 funded projects for Sorghum research have been added to the site and integrated into the site-wide search to a total of 305. A [Projects](https://www.sorghumbase.org/projects) page shows a map of institutions involved, and a searchable table of awards. Each project has its own page with basic information from the funding agency; we invite PIs to work with us to enrich their project pages to include links to publications, blog entries, project web pages, etc.


## Release Contents -- Databases

### Genomes & Gene Annotations

This release includes a total of 74 sorghum assemblies including two versions of BTx623 (NCBIv3 and 5.1 from Phytozome, which was released under the [Fort Lauderdale Accord](https://www.sanger.ac.uk/wp-content/uploads/fortlauderdalereport.pdf) for restricted use prior to publication), and seven plant outgroup species. See [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for more details. 


### Variation

Standard rsID identifiers assigned by the EVA were mapped to nearly 41 million SNPs in both BTx623 assemblies, NCBIv3 and JGIv5.1, as well as remapped to 46 ex-PVP breeding lines. In this release, rsIDs were also assigned to Japonica rice and B73 maize SNPs.


**BTx623 (NCBIv3)**

Over 78 million sorghum variants including more than 65 million naturally occurring SNPs genotyped in over 3,500 accessions, and almost 13 million chemically-induced variants (i.e., ethyl methanesulfonate (EMS)-induced mutations) in a mutagenized population BTx623 background. This data includes the following studies:

***Natural variation***

* [Kumar et al (2024)](https://www.sorghumbase.org/paper/24501) - 18.7M SNPs and indels, whole-genome sequencing 365 accessions (including 332 BAP)
* [Lasky et al (2015)](https://www.sorghumbase.org/paper/24502) - 400K SNPs and indels, 1943 georeferenced landraces
* [Boatwright et al (2022)](https://www.sorghumbase.org/paper/20741) - 33M SNPs and 5M indels genotyped by sequencing 400 SAP lines
* [Lozano et al (2021)](https://www.sorghumbase.org/paper/comparative-evolutionary-genetics-of-deleterious-load-in-sorghum-and-maize) - nearly 13M SNPs, 499 sorghum accessions from the TERRA-MEPP, TERRA-REF population panels, and lines previously genotyped by [Mace et al (2013)](https://www.sorghumbase.org/paper/21275)


***Chemically-induced mutagenesis screens***

* [Jiao et al (2023)](https://www.sorghumbase.org/paper/23165) - 8.6 million EMS-induced mutations, 890 lines
* [Addo-Quaye et al (2018)](https://www.sorghumbase.org/paper/19942) - 2.5 million EMS-indiced mutations, 486 M3 lines
* [Jiao et al (2016)](https://www.sorghumbase.org/paper/a-genomic-resource-for-the-development-improvement-and-exploitation-of-sorghum-for-bioenergy) - 1.7 million EMS-induced mutations, 252 M4 mutant pools

_Note_: 41 million rsIDs were mapped from EVA release 5 (July 2023) to the genetic variants in sorghum BTx623 NCBIv3. In the near future, we aim to have rsIDs assigned to all naturally occuring variants in BTx623 in SorghumBase.


**BTx623 (JGI_v5.1)**

* 36.1M SNPs, 940 resequenced sorghum genomes from the [Global Sorghum Initiative](https://www.globalsorghuminitiative.org/)

The [Global Sorghum Initiative](https://www.globalsorghuminitiative.org/) is founded by the Gates Foundation. Libraries for the 940 genomes were resequenced using Illumina HiSeq X10 and Illumina NovaSeq 6000 paired-end sequencing (2 × 150 bp) at HudsonAlpha Institute for Biotechnology and aligned to the BTx623 reference with BWA-mem. The resulting .bam file was filtered for duplicates using Picard (http://broadinstitute.github.io/picard) and realigned around indels using GATK 3.056. Multi-sample SNP calling was done using SAMtools mpileup and Varscan V2.4.089 with a minimum coverage of eight, a minimum alternate allele count of four, and a minimum minor allele frequency of 0.001. The full 2144-library VCF was subsetted to the Gates Foundation samples using bcftools 1.9.

Approximately 41 million BTx623 rsIDs from EVA release 5 (July 2023) were remapped from NCBIv3 to JGI_V5.1.


***BTx623 rsIDs mapped to other sorghum accessions***

Nearly 41 million standard rsIDs were mapped from BTx623 (NCBIv3) to the 46 ex-PVP breeding lines resequenced by the team lead by Dr. Todd Michael of the Salk Institute.

In addition, the Lasky variation set (Lasky et al, 2015) was projected from version 1 (v1.4) to NCBIv3 (v3.1), and corresponding rsIDs were assigned.


***Loss of function mutations***

No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206).

***GWAs***

No updates in the current release, please see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207) for the GWAs hits from the meta-analysis of 234 phenotypic trait datasets of 40 studies (25 qualified for inclusion) on 406 SAP accessions performed by Mural et al (2021).

***QTL data***

No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for the 5,843 sorghum QTLs for 148 sorghum traits imported from the [Sorghum QTL Atlas](https://aussorgm.org.au/).


 **Tx2783**

No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for the ~32.5 million Tx2873 SNPs called in the 400 SAP with the HPC-based genome variant calling GATK4 pipeline [Zhou et al (2023)](https://sorghumbase.org/paper/23243).


### Expression

No updates in the current release, please see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207) for the gene expression data for the _S. bicolor_ BTx623 (NCBIv3) genome reference curated and processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments) consisting of 8 studies with baseline expression and 3 with differential expression. Baseline experiments, as well as expression of paralogs for the reference study and eFP expression visualizations, are available through our Gene Search (Expression panel). [Differential experiments](https://www.ebi.ac.uk/gxa/experiments?kingdom=Plants&species=Sorghum+bicolor&experimentType=Differential) are available at the Expression Atlas. Baseline gene expression is also available from the gene pages of the Genome Browser, see for example [Expression for SORBI_3006G095600](https://ensembl.sorghumbase.org/Sorghum_bicolor/Gene/ExpressionAtlas?g=SORBI_3006G095600;r=6:46566240-46571064). Data for the eFP browser on the Expression tab of the SorghumBase search results projects gene expression from a Developmental and a Stress Atlas by McCormick et al (2018).

### Pathways

No updates in the current release, please see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207) for the 271 orthology-based sorghum pathways for _S. bicolor_ BTx623 (NCBIv3), which were projected from curated Nipponbare rice pathways by [Gramene’s Plant Reactome](https://plantreactome.gramene.org/) Team as of August 2023.

### Regulation

In addition to the non-coding features (lnc_RNA, ncRNA, ncRNA_gene, pre_miRNA, RNase_MRP_RNA, rRNA, snoRNA, snRNA, SRP_RNA, and tRNA) available as annotations and visualizations for BTx623 (NCBIv3) and lifted to other sorghum accessions, see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207), we are offering in this release, ATAC-seq and Chip-seq data from [Lou et al (2019)](https://sorghumbase.org/paper/24376) for sorghum.


### Comparative Genomics

[**Gene Trees**](https://ensembl.sorghumbase.org/prot_tree_stats.html). No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=PRelease%206) for the 42,926 protein-coding gene family trees and 24,055 putative split genes for the 36 plant genomes in SorghumBase.

[**Synteny**](https://ensembl.sorghumbase.org/compara_analyses.html). No updates in the current release, please see [release notes for version 5.0](https://www.sorghumbase.org/relnotes?section=PRelease%205) for the 83 synteny maps in SorghumBase.

[**Whole-Genome Alignments**](https://ensembl.sorghumbase.org/compara_analyses.html). No updates in the current release, please see [release notes for version 5.0](https://www.sorghumbase.org/relnotes?section=PRelease%205) for the 15 pairwise DNA alignments in SorghumBase.


### Curated Publications

Our database of sorghum research papers has grown to 1,026 publications. The integrated search interface includes matching papers when searching by author, keywords, pubmed ID, or any other text within the title or abstract. Publications in the database can be linked to news items and funding sources, for example: [Boatwright et al (2021)](https://www.sorghumbase.org/post/whole-genome-sequencing-of-400-sorghum-association-panel-sap-accessions-establishes-a-crucial-resource-for-dissecting-genomic-diversity-in-sorghum).

### Funded Projects

Our database of funded sorghum research includes 305 projects.


## References

- Kumar N, Boatwright JL, Boyles RE, Brenton ZW, Kresovich S. 2024. "Identification of pleiotropic loci mediating structural and non-structural carbohydrate accumulation within the sorghum bioenergy association panel using high-throughput markers". Front Plant Sci. 15:1356619. doi: [10.3389/fpls.2024.1356619](https://doi.org/10.3389/fpls.2024.1356619).

- Lasky JR, Upadhyaya HD, Ramu P, Deshpande S, Hash CT, Bonnette J, Juenger TE, Hyma K, Acharya C, Mitchell SE, Buckler ES, Brenton Z, Kresovich S, Morris GP. 2015. "Genome-environment associations in sorghum landraces predict adaptive traits". Sci Adv. 1(6):e1400218. doi: [10.1126/sciadv.1400218](https://doi.org/10.1126/sciadv.1400218).

-  Wang B, Chougule K, Jiao  Y, Olson A, Kumar V, Gladman N, Huang J, Llaca V, Fengler K, Wei X, Wang L, Wang  X, Regulski M, Drenkow J, Gingeras T, Hayes C, Armstrong JS, Huang Y, Xin Z, Ware D. 2024. "High-quality chromosome scale genome assemblies of two important Sorghum inbred lines, Tx2783 and RTx436". NAR Genom Bioinform. 6(3):lqae097. doi: [10.1093/nargab/lqae097](https://doi.org/10.1093/nargab/lqae097).
