## SorghumBase Release 4.0
### Released: July 2022
## Summary

For the first time, this release includes the genome assembly and annotation of the poplar tree, _Populus trichocarpa_ v4.1 from JGI Phytozome, a dicot model plant that like sorghum, is also studied for drought tolerance and bioenergy production. Poplar is the eighth outgroup species that together with the genomes of 18 sorghum varieties was used to generate 31,210 protein-coding gene family trees to support phylogenetic analyses and cross-species comparisons.

At the instance of the sorghum community, we updated the gene and transcript names of the sorghum BTx623 reference models as follows: 

1) Gene names of the form Sobic.nnn (in use at the Phytozome database) were added as synonyms of their corresponding SORBI_3nnn counterparts, and are now given as the display name for gene models in the SorghumBase search interface and genome browser.

2) Transcript names were modified to the conventional format of appending to Ensembl gene names of the form SORBI_3nnn, a dot followed by a digit suffix. For example, transcripts for the gene ‘SORBI_3004G141800’ are SORBI_3004G141800.1, SORBI_3004G141800.2, SORBI_3004G141800.3, and SORBI_3004G141800.4. This is in contrast to their corresponding (GenBank) names such as KXG30174, OQU84910, OQU84911, and OQU84912, currently in use by the Gramene and Plant Ensembl database, see [SORBI_3004G141800 example](http://plants.ensembl.org/Sorghum_bicolor/Gene/Summary?g=SORBI_3004G141800;r=4:41625307-41663480).

Here are the corresponding [Gene](https://ensembl.sorghumbase.org/Sorghum_bicolor/Gene/Summary?g=SORBI_3004G141800;r=4:41625307-41663480) and [Transcript](https://ensembl.sorghumbase.org/Sorghum_bicolor/Transcript/Similarity?db=core;g=SORBI_3004G141800;r=4:41625307-41663480;t=SORBI_3004G141800.1) pages for this gene as an example in this release.

We have included a [mapping file](https://ftp.sorghumbase.org/release-4/Sorghum_BTx623_gene_id_mapping_v3.1.1.txt) to convert sorghum gene identifiers of the forms Sobic.nnn, SORBI_3nnn, and SbNNN. This file is available at [SorghumBase’s FTP site](https://ftp.sorghumbase.org/).

Our publications database has grown to 420 papers, 16 of which have been highlighted in recent [News](https://www.sorghumbase.org/posts?categories=research-highlights) posts, together with 71 relevant [Events](https://www.sorghumbase.org/events) for the sorghum research community.

Core funding for the project is provided by the Agricultural Research Service of the [U.S. Department of Agriculture](http://www.usda.gov/) (USDA ARS 8062-21000-041-00D).

## Release Information
- [Overall Highlights](#overall-highlights)
- [Databases](#databases)
- [References](#references)

## Overall Highlights

**New Genomes**

- [_Populus trichocarpa_](https://ensembl.sorghumbase.org/Populus_trichocarpa) (poplar tree)

**New Data**

- [Mapping file](https://ftp.sorghumbase.org/release-4/Sorghum_BTx623_gene_id_mapping_v3.1.1.txt) to interconvert sorghum gene identifiers of the forms Sobic.nnn, SORBI_3nnn, and SbNNN.

**Other Updates and Highlights**

- The link to [SorghumBase’s FTP site](https://ftp.sorghumbase.org/) has been made more prominent in the [Homepage](https://www.sorghumbase.org).

- An additional 105 relevant sorghum papers in our [Publications](https://www.sorghumbase.org/publications) database. Ten of those are described in more detailed in news items that include author quotes, images and links to relevant websites.


## Databases
### Comparative Genomics

[**Gene Trees.**](https://ensembl.sorghumbase.org/prot_tree_stats.html) A total of
31,210 protein-coding gene family trees were constructed using the peptide encoded by
the canonical transcript (i.e., a representative transcript for a given gene) for each
of 829,431 individual genes (870,922 input proteins) from 26 plant genomes.

### Variation

No updates since release 3. Genetic variation data sets are available for over 17 million sorghum variants including almost 13 million single nucleotide
polymorphisms (SNPs) [(Lozano _et al_, 2021)](https://sorghumbase.org/paper/comparative-evolutionary-genetics-of-deleterious-load-in-sorghum-and-maize) and a combined 4.3 million chemically induced by ethyl methanesulfonate (EMS) point mutations [(Addo-Quaye _et al_, 2018)](https://www.sorghumbase.org/paper/19942) and [(Jiao _et al_, 2016)](https://sorghumbase.org/paper/a-sorghum-mutant-resource-as-an-efficient-platform-for-gene-discovery-in-grasses); and nearly 6,000 QTLs from the [Sorghum QTL Atlas](https://aussorgm.org.au/).

### Expression

Gene expression data for the _S. bicolor_ BTx623 genome reference was curated and
processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments). No other updates since release 2.

### Pathways

Orthology-based sorghum pathways for _S. bicolor_ BTx623 projected from curated Japonica rice pathways by [Gramene’s Plant Reactome](https://plantreactome.gramene.org/). No other updates since release 2.

### Curated Publications

Our database of sorghum research papers has grown to 420 publications. The integrated search interface includes matching papers when searching by author, keywords, pubmed ID, or any other text within the title or abstract. Publications in the database can be linked to news items and funding sources, for example: [(Lozano _et al_, 2021)](https://sorghumbase.org/paper/comparative-evolutionary-genetics-of-deleterious-load-in-sorghum-and-maize).


## References

Tuskan GA, Difazio S, Jansson S, Bohlmann J, Grigoriev I, Hellsten U, Putnam N, Ralph S, Rombauts S, Salamov A et al. 2006. _The genome of black cottonwood, Populus trichocarpa (Torr. & Gray)_. Science. 313:1596-1604. 
PMID: 16973872. doi: [10.1126/science.1128691](https://doi.org/10.1126/science.1128691).
