## SorghumBase Release 6.0
### Released: May 2023
## Summary

Release 6 features 29 sorghum genomes including two assembly versions of the BTx623 reference: version 3 from the National Center for Biotechonology Information (NCBI_v3), and version 5.1 from Phytozome (JGI_v5), and 7 additional species, including Arabidopsis, rice, and maize, which serve as sources for curated gene function and phylogenetic analyses. Over 43K protein-coding gene trees support information on orthology, paralogy, and candidate copy numbers variation. As of this release, SorghumBase hosts over 59 million genetic variants for BTx623 version 3, including naturally occuring SNPs, and novel EMS-induced point mutations, as well as 32.5 million SNPs called in Tx2783. All variants can be visualized in the context of a gene and the gene's protein domains, and their impact on a transcript is assessed with a Variant Effect Predictor (VEP) and SIFT scores (BTx623 variants only [confirm with Sharon]), capturing functional effects in over 48.1K BTx623 transcripts and XXX [ask Sharon] Tx2783 transcripts. In addition to the genomic resources, SorghumBase provides weekly updates of news items, conferences, and community events. In the last six months, our publications database has grown to over 739 papers, 53 of which have been highlighted in recent News posts. News items are indexed and searchable, and include information on how to access data related to the articles from the site. 

A [user guide](https://www.sorghumbase.org/guides) is available from the site and feedback may be provided via a [contact form](https://WWW.sorghumbase.org/contact). Individual training sessions may be scheduled upon  request. Please check the upcoming [Events](https://www.sorghumbase.org/events) on the latest webinars, conferences, meetings, etc. relevant to the sorghum research community. You are always welcome to [Contact us](https://www.sorghumbase.org/contact), if you are interested in hosting a training event, collaborating with the SorghumBase team or would like to contact us with any questions or comments. 

Core funding for the project is provided by the Agricultural Research Service of the [U.S. Department of Agriculture](http://www.usda.gov/) (USDA ARS 8062-21000-041-00D).

## What's New in release 5?

New features include a new Papers tab in the search results listing PubMed-indexed papers associated with curated genes (7184 Arabidopsis, 4006 rice, and 354 maize) from [NCBI's GeneRIF](https://www.ncbi.nlm.nih.gov/gene/about-generif) (Jimeno-Yepes _et al_, 2013) and [RAP-DB](https://rapdb.dna.affrc.go.jp/) (Sakai _et al_, 2013 - PMID: 23299411). This data makes it now possible to search for the Trait Ontology (TO) and Plant Ontology (PO) terms associated with curated genes. In addition, a new link to the Gene Tree Curation Tool has been added in the Homology results tab for sorghum genes in both assemblues, v3 and 5.1. 

Over 6,000 QTL-gene associations in 223 traits (~150 studies) imported from the Sorghum QTL Atlas, are now searchable. Try for example, searching for sorghum genes associated with ["plant height"](https://dev.sorghumbase.org/genes?filters={%22status%22:%22init%22,%22operation%22:%22AND%22,%22negate%22:false,%22marked%22:false,%22leftIdx%22:0,%22rightIdx%22:3,%22children%22:[{%22fq_field%22:%22location%22,%22fq_value%22:%22(map:GCA_000003195.3%20AND%20region:9%20AND%20start:[57327486%20TO%2057634721])%22,%22name%22:%22Height%20(plant%20height)%22,%22category%22:%22QTLs%22,%22leftIdx%22:1,%22rightIdx%22:2,%22negate%22:false,%22showMenu%22:false,%22marked%22:true}],%22showMarked%22:false,%22showMenu%22:false,%22moveCopyMode%22:%22%22,%22searchOffset%22:0,%22rows%22:20}&genomes=)

This release includes version 5.1 of the BTx623 assembly from Phytozome. The v5.1 assembly is a wholly resequenced genome applying PacBio long-read data. The annotation is an update that uses all the v3 resources, but with additional RNA-seq from JGI projects, and full-length transcripts to further improve the completeness of the gene set. For additional details on the methodology for this assembly and gene annotation, see [Phytozome](https://phytozome-next.jgi.doe.gov/info/Sbicolor_v5_1). It is important to note that the v5.1 assembly was released before scientific publication according to the Fort Lauderdale Accord by the Department of Energy's Joint Genome Institute (JGI). The accord restricts publication of articles containing analyses of genes or genomic data on this chromosome-scale assembly prior to publication of a comprehensive genome analysis by JGI and/or its collaborators. Therefore, we are only providing a basic genome browser for the new v5.1 assembly. The assembly has also been integrated in the Compara gene trees together with the 28 sorghum genomes in SorghumBase, including BTx623 v3, and run through the InterProScan pipeline to predict protein domains. New genetic variation data is being released for: 

  1) BTx623 (NCBI_v3): 8.6 million of EMS-induced mutations determined in 890 pools at the USDA Lubbock Texas station, and 
  2) Tx2783: 32.5 million SNPs called in the 400 SAP lines 

Our publications database has also grown by 181 sorghum papers, 13 of which have been highlighted in recent [News](https://www.sorghumbase.org/posts?categories=research-highlights) posts since our December 2022 release.


## Release Information
- [Overall Highlights](#overall-highlights-6)
- [Databases](#databases-6)
- [References](#references-6)

## Overall Highlights

**New Genomes & Gene Annotations**

- [_Sorghum bicolor_ BTx623 v5](https://ensembl-dev.sorghumbase.org/Sorghum_bicolorv5) from JGI's [Phytozome](https://phytozome-next.jgi.doe.gov/info/Sbicolor_v5_1) restricted by the Ft Lauderdale Accord. Genes in this assembly have been included in gene family trees and assigned InterPro domains.

Note: To search for genes in the v5.1 browser, please append ".v5.1" to the corresponding Sobic gene identifier like so: [Sobic.001G121600.v5.1](https://ensembl-dev.sorghumbase.org/Sorghum_bicolorv5/Gene/Summary?db=core;g=Sobic.001G121600.v5.1;r=1:9578725-9579867;t=Sobic.001G121600.1.v5.1).

(_Internal note_: There are 1,958 less protein-coding genes in v5 (32,160 in Phytozome) compared to v3 (34,118 in Gramene/SB/Ensembl, from our BTx623 annotation section & Mart) -- Check diff, transcript not mapped, compare Sbicolor_730_v5.1.synonym.txt Phytozome against Sunitaâ€™s mapping file for v3 gene ids; also Sbicolor_730_v5.1.locus_transcript_name_map.txt shows mappings from v5.1 to v3.1.1, no transcript in a line means that locus is mapped and transcript is not. Q on a diff note, _should we consider providing a mapping file?_)


**New Data**

Comparative genomics:

- New gene trees including BTx623 v5.1
- Putative split genes identified for BTx623 v5.1

Genetic Variation:

- BTx623 v3 (NCBI): 8.6 million EMS-induced mutations in 890 samples
- Tx2783: 32.5M SNPs in 400 SAP lines


**Other Updates and Highlights**

- Updated ENA identifiers for 12 sorghum assemblies: 

1) Grass1	GCA_947241715.1
2) PI329311	GCA_947241655.1
3) PI506069	GCA_947241665.1
4) PI229841	GCA_947241625.1
5) PI510757	GCA_947241685.1
6) Leoti	GCA_947241725.1
7) PI655972	GCA_947241635.1
8) Chinese Amber	GCA_947241645.1
9) PI297155	GCA_947241675.1
10) Rio(PI563295 CPNAMrio)	GCA_947241735.1
11) Sorghum_bicolor-Tx2783-Reference-CSHL-USDA-1.0	GCA_903166285.1
12) Sorghum_bicolor-Tx436-Reference-CSHL-USDA-1.0	GCA_903166325.1

- New features in the search interface: Papers tab with PubMed links curated by GeneRIF and RAP-DB; and newly indexed QTL traits, PO and TO terms associated with curated genes

- New SIFT scores for all BTx623 genetic variants

- New links to the Gene Tree Curation Tool in the Homology tab of all BTx623 genes

- 181 relevant sorghum papers added to our [Publications](https://www.sorghumbase.org/publications) database with 13 of them described in more detailed in news items that include author quotes, images and links to relevant websites 


## Databases
### Comparative Genomics

[**Gene Trees.**](https://ensembl.sorghumbase.org/prot_tree_stats.html) A total of
42,926 protein-coding gene family trees were constructed using the peptide encoded by
the canonical transcript (i.e., a representative transcript for a given gene) for each
of 1,233,135 individual genes (1,275,791 input proteins) from 36 plant genomes.

(_Internal note_: This is less than in R5 for same number of genomes: 43,673 trees - 1,239,742 individual genes (1,283,146 proteins) from 36 plant genomes. Q: _Shouldn't there be 37 genomes in R6 because of the addition of v5.1?_)

[**Synteny**](https://ensembl.sorghumbase.org/compara_analyses.html). No updates in the current release, please see [release notes for version 5.0](https://www.sorghumbase.org/relnotes?section=PRelease%205) for the 83 synteny maps in SorghumBase.

[**Whole-Genome Alignments**](https://ensembl.sorghumbase.org/compara_analyses.html). No updates in the current release, please see [release notes for version 5.0](https://www.sorghumbase.org/relnotes?section=PRelease%205) for the 5 pairwise DNA alignments in SorghumBase.


### Variation

**BTx623 v3 (NCBI)**:

Genetic variation data sets for BTx623 v3 (NCBI) are available as about 70 million sorghum variants including
nearly 51 million SNPs (33 million SNPs plus 5 million indels genotyped by sequencing 400 SAP lines [Boatwright _et al_, 2022](https://www.sorghumbase.org/post/whole-genome-sequencing-of-400-sorghum-association-panel-sap-accessions-establishes-a-crucial-resource-for-dissecting-genomic-diversity-in-sorghum)), almost 13 million SNPs determined in 499 sorghum accessions from the TERRA-MEPP, TERRA-REF population panels, and lines previously genotyped by Emma Mace and collaborators in 2013, from the study by [Roberto Lozano and colleagues (2021)](https://www.sorghumbase.org/post/comparative-analysis-of-deleterious-mutations-in-sorghum-versus-maize);
13.8 EMS-induced mutations (Jiao and colleagues, submitted to The Plant Journal), [(Addo-Quaye _et al_, 2018)](https://www.sorghumbase.org/paper/19942) and [(Jiao _et al_, 2016)](https://sorghumbase.org/paper/a-sorghum-mutant-resource-as-an-efficient-platform-for-gene-discovery-in-grasses), 
and nearly 6,000 QTLs for 220 sorghum traits from the [Sorghum QTL Atlas](https://aussorgm.org.au/).

 **Tx2783**

About 32.5 million SNPs called in the 400 SAP lines SNPs called with the GATK4 pipeline by Yong Zhou from Rod Wing's group at King Abdullah University of Science and Technology, are now available from the Tx2783 genome browser, see for example [4_6047465_C_T](https://ensembl-dev.sorghumbase.org/Sorghum_tx2783pac/Variation/Sample?db=core;r=4:6046965-6047965;v=4_6047465_C_T;vdb=variation;vf=11387812), a SNP predicted to introduce a STOP codon and result in a truncated [SbiRTX2783.04G076100](https://ensembl-dev.sorghumbase.org/Sorghum_tx2783pac/Gene/Summary?db=core;g=SbiRTX2783.04G076100;r=4:6046424-6048133;t=SbiRTX2783.04G076100.1;v=4_6047465_C_T;vdb=variation;vf=11387812) protein product.


### Expression

Gene expression data for the _S. bicolor_ BTx623 genome reference was curated and
processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments). No other updates since release 2.

### Pathways

Orthology-based sorghum pathways for _S. bicolor_ BTx623 projected from curated Japonica rice pathways by [Grameneâ€™s Plant Reactome](https://plantreactome.gramene.org/). No other updates since release 2.

### Curated Publications

Our database of sorghum research papers has grown to 739 publications. The integrated search interface includes matching papers when searching by author, keywords, pubmed ID, or any other text within the title or abstract. 
Publications in the database can be linked to news items and funding sources, for example: [Boatwright _et al_(2021)](https://www.sorghumbase.org/post/whole-genome-sequencing-of-400-sorghum-association-panel-sap-accessions-establishes-a-crucial-resource-for-dissecting-genomic-diversity-in-sorghum).


## References

- Jiao Y, Singh D, Barry K, Daum C, Yoshinaga Y, Lipzen A, Khan A, Parasa S-P, Wei S, Lu Z, Tello-Ruiz Marcela K, Burow G, Hayes C, Chen J, Brandizzi F, Mortimer J, Ware D, Xin Z. _A Large Sequenced Mutant Library - Valuable Reverse Genetic Resource that Covers 98% of Sorghum Genes_. Manuscript submitted.
- Jimeno-Yepes, A. J., Sticco, J. C., Mork, J. G., & Aronson, A. R. (2013). GeneRIF indexing: sentence selection based on machine learning. BMC Bioinformatics, 14, 171. https://doi.org/10.1186/1471-2105-14-171.
- Sakai, H., Lee, S. S., Tanaka, T., Numa, H., Kim, J., Kawahara, Y., Wakimoto, H., Yang, C.-C., Iwamoto, M., Abe, T., Yamada, Y., Muto, A., Inokuchi, H., Ikemura, T., Matsumoto, T., Sasaki, T., & Itoh, T. (2013). Rice Annotation Project Database (RAP-DB): an integrative and interactive database for rice genomics. Plant & Cell Physiology, 54(2), e6. https://doi.org/10.1093/pcp/pcs183.
