## SorghumBase Release 5.0
### Released: December 2022
## Summary

This release includes 10 new sorghum genomes from the Carbon-Partitioning Nested Association Mapping (CP-NAM) panel: Chinese Amber [PI 22913], Grassl [PI 154844], Leoti [PI 586454], PI 229841, PI 297155, PI 329311, PI 506069, PI 510757, PI 655972, and Rio [PI 563295]. 
The CP-NAM population panel was developed and publicly released as a key genetic resource for the characterization and improvement of sorghum (Boatwright et al, 2021 & 2022; Kumar et al, 2022).

This release also includes nearly 44 million genetic variants including ~38 M SNPs and ~5 M indels that were genotyped in the 400 Sorghum Association Panel (SAP) via whole-genome sequencing (WGS) by Lucas Boatwright and collaborators (2022).

In addition, a total of 84 new synteny maps including those between each of the 28 sorghum genomes in SorghumBase and maize B73 v5 or Japonica rice IRGSPv1, and the sorghum BTx623 reference and each of the other 27 sorghum varieties has been added. 
Plus whole-genome alignments between the sorghum BTx623 reference and each of the anchors for orthologous species: Arabidopsis TAIR10, Japonica rice IRGSPv1, and maize B73 v5.

Our publications database has grown to 529 papers, 31 of which have been highlighted in recent [News](https://www.sorghumbase.org/posts?categories=research-highlights) posts, together with 76 relevant [Events](https://www.sorghumbase.org/events) for the sorghum research community.

Core funding for the project is provided by the Agricultural Research Service of the [U.S. Department of Agriculture](http://www.usda.gov/) (USDA ARS 8062-21000-041-00D).

## Release Information
- [Overall Highlights](#overall-highlights)
- [Databases](#databases)
- [References](#references)

## Overall Highlights

**New Genomes**

- [_Populus trichocarpa_](https://ensembl.sorghumbase.org/Populus_trichocarpa) (poplar tree)
- [_Sorghum bicolor CHINESE AMBER_](https://ensembl-dev.sorghumbase.org/Sorghum_chineseamber) (Chinese Amber [PI 22913])
- [_Sorghum bicolor GRASSL_](https://ensembl-dev.sorghumbase.org/Sorghum_chineseamber) (Grassl [PI 154844])
- [_Sorghum bicolor LEOTI_](https://ensembl-dev.sorghumbase.org/Sorghum_chineseamber) (Leoti [PI 586454])
- [_Sorghum bicolor IS2382_](https://ensembl-dev.sorghumbase.org/Sorghum_chineseamber) (IS2382 [PI 229841])
- [_Sorghum bicolor  IS13633_](https://ensembl-dev.sorghumbase.org/Sorghum_chineseamber) (IS13633 [PI 297155])
- [_Sorghum bicolor IS11069_](https://ensembl-dev.sorghumbase.org/Sorghum_chineseamber) (IS11069 [PI 329311])
- [_Sorghum bicolor MBONOU_](https://ensembl-dev.sorghumbase.org/Sorghum_chineseamber) (Mbonou [PI 506069])
- [_Sorghum bicolor AP79-714_](https://ensembl-dev.sorghumbase.org/Sorghum_chineseamber) (AP79-714 [PI 510757])
- [_Sorghum bicolor PINK KAFIR_](https://ensembl-dev.sorghumbase.org/Sorghum_chineseamber) (Pink Kafir [PI 655972])
- [_Sorghum bicolor RIO_](https://ensembl-dev.sorghumbase.org/Sorghum_chineseamber) (Rio [PI 563295])


**New Data**

- Nearly 44 million genetic variants including ~38 M SNPs and ~5 M indels genotyped in 400 SAP accessions (Boatwright et al, 2022)

- Eighty-four new synteny maps including those between each of the 28 sorghum genomes in SorghumBase, and maize B73 v5 or Japonica rice IRGSPv1, and the sorghum BTx623 reference and each of the other 27 sorghum varieties.
- Whole-genome alignments between the sorghum BTx623 reference and Arabidopsis TAIR10, Japonica rice IRGSPv1, or maize B73 v5.

**Other Updates and Highlights**

- An additional 109 relevant sorghum papers in our [Publications](https://www.sorghumbase.org/publications) database. Fifteen of those are described in more detailed in news items that include author quotes, images and links to relevant websites.


## Databases
### Comparative Genomics

[**Gene Trees.**](https://ensembl.sorghumbase.org/prot_tree_stats.html) A total of
43,673 protein-coding gene family trees were constructed using the peptide encoded by
the canonical transcript (i.e., a representative transcript for a given gene) for each
of 1,239,742 individual genes (1,283,146 input proteins) from 36 plant genomes.

Example of synteny: https://ensembl-dev.sorghumbase.org/Sorghum_bicolor/Location/Synteny?db=core&r=4%3A41625307-41663480&otherspecies=Zea_maysb73

Example of WGA: https://ensembl-dev.sorghumbase.org/Sorghum_bicolor/Location/Compara_Alignments/Image?align=1;db=core;otherspecies=Zea_maysb73;r=4:41625307-41663480;time=1668636258419.419


### Variation

Genetic variation data sets are available as about 61 million sorghum variants including
almost 51 million SNPs (44 million of which plus 5 million indels genotyped by sequencing
in the SAP lines [Boatwright _et al_, 2022](https://www.sorghumbase.org/post/whole-genome-sequencing-of-400-sorghum-association-panel-sap-accessions-establishes-a-crucial-resource-for-dissecting-genomic-diversity-in-sorghum)), 
4.3 EMS-induced mutations [(Addo-Quaye _et al_, 2018)](https://www.sorghumbase.org/paper/19942) and [(Jiao _et al_, 2016)](https://sorghumbase.org/paper/a-sorghum-mutant-resource-as-an-efficient-platform-for-gene-discovery-in-grasses), 
and nearly 6,000 QTLs for 220 sorghum traits from the [Sorghum QTL Atlas](https://aussorgm.org.au/).

### Expression

Gene expression data for the _S. bicolor_ BTx623 genome reference was curated and
processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments). No other updates since release 2.

### Pathways

Orthology-based sorghum pathways for _S. bicolor_ BTx623 projected from curated Japonica rice pathways by [Grameneâ€™s Plant Reactome](https://plantreactome.gramene.org/). No other updates since release 2.

### Curated Publications

Our database of sorghum research papers has grown to 529 publications. The integrated search interface includes matching papers when searching by author, keywords, pubmed ID, or any other text within the title or abstract. 
Publications in the database can be linked to news items and funding sources, for example: [Boatwright _et al_, 2021](https://www.sorghumbase.org/post/whole-genome-sequencing-of-400-sorghum-association-panel-sap-accessions-establishes-a-crucial-resource-for-dissecting-genomic-diversity-in-sorghum).


## References

Boatwright JL, Sapkota S, Jin H, Schnable JC, Brenton Z, Boyles R, Kresovich S. 2022. _Sorghum Association Panel whole-genome sequencing establishes cornerstone resource for dissecting genomic diversity_. Plant J. PMID: 35653240. 
doi: [10.1111/tpj.15853](https://doi.org/10.1111/tpj.15853).
