## SorghumBase Release 9.0
### Released: December 2024

SorghumBase (https://www.sorghumbase.org) is a USDA-ARS funded resource released to the public in 2021. The SorghumBase team works closely with the community to support stewardship of sorghum genomics data, establishes best-practices on managing the data, and provides opportunities for networking and capacity building within the community. We maintain a database of scientific publications, genome sequences, gene structure annotations, and comparative genomic analyses integrated with curated data from [EBI's Gene Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments), [Bio-Analytical Resources' eFPseq browser](https://bar.utoronto.ca/), [Gramene's Plant Reactome](https://plantreactome.gramene.org), and [OZ Sorghum QTL Atlas](https://aussorgm.org.au/sorghum-qtl-atlas/).


## Summary

In its ninth release, SorghumBase (SB) features 12 new _Sorghum bicolor_ ssp. _bicolor_ genomes including two telomere-to-telomere (T2T) assemblies of BTx623 ([Li _et al_, 2024](https://www.sorghumbase.org/paper/24530); [Deng _et al_, 2024](https://www.sorghumbase.org/paper/24194)) to a total of 85 sorghum cultivars (88 assemblies).  

We are also providing variation sampled by the Gates-funded **Sorghum Genomics Toolbox** (SGT) Project (also known as the [Global Sorghum Initiative](https://www.globalsorghuminitiative.org/)) remapped to NCBIv3. This set consists of (36.1 - _check # successfully remapped_) million SNPs called in 940 resequenced genomes.

In an effort to harmonize germplasm identifiers within SorghumBase and based on the recommendation of best practices used by the community and curators of major sorghum germplasm repositories such as GRIN and ICRISAT, where available, we provide genotyping data using standard germplasm identifiers (e.g., PI and IS numbers) and standard SNP identifiers (e.g., rsIDs). 
  
The Germplasm tab of the results of a gene query now includes variants that cause the loss of a gene's function (LOF) from 46 ex-PVP breeding lines and Search results are now grouped and tallied by species.

Accessions with genetic variants causing the LOFs are now organized by study or population, type of variant (e.g., stop gained/lost, frameshift, splice acceptor/donor) and homozygous/heterozygous state in the Germplasm tab of the gene.

Besides genomic resources, SorghumBase provides weekly updates of news items, conferences, community events and funded research projects. Our publications database now includes 1,147(UPDATE from 1,026) sorghum papers, 30(UPDATE from 13) of which have been [highlighted in News posts](https://www.sorghumbase.org/posts?categories=research-highlights) since the last SB release in October. News items and project pages are indexed and searchable, and include information on how to access related data on SorghumBase. A recently updated [user guide](https://www.sorghumbase.org/guides) is available from the site and feedback may be provided via a [contact form](https://www.sorghumbase.org/contact). Individual training sessions may be scheduled upon request. Please check the upcoming [events](https://www.sorghumbase.org/events) on the latest webinars, conferences, meetings, etc. relevant to the sorghum research community. You are always welcome to [contact us](https://www.sorghumbase.org/contact), if you are interested in hosting a training event, interested in collaboration with the SorghumBase team or would like to provide feedback or ask questions. 

Funding for the project is provided by the Agricultural Research Service of the [U.S. Department of Agriculture](http://www.usda.gov/) (USDA ARS 8062-21000-051-00D). 


## What's New in release 9?

### New Data

***Genomes & Gene Annotations***

1) We are providing in this release, 12 new genome assemblies:

  - 2 T2T assemblies of Sorghum BTx623 from [Li _et al_ (2024)](https://www.sorghumbase.org/paper/24530) and [Deng _et al_ (2024)](https://www.sorghumbase.org/paper/24194)
  - 1 T2T assembly for Ji2055
  - PI 656068	(SC 170-6-17)
  - PI 576434	(Ex-Mubi)
  - PI 561072	(A/BTxARG-1)
  - PI 561073	(A/BTx635)
  - PI 552861	(A/BTx631)
  - PI 534133	(No. 69 Mashela Tinguish, Warakul Ethiopi)
  - PI 655993	(TX 399(DALHART RES WHTLND)B LN)
  - Grif 16309 (Grif16309)
  - PI 329250	(ADAR)

2) New gene models for the 46 ex-PVP breeding lines with their corresponding Variation Effect Predictions (PEVs).
   
 
***Genetic & Phenotypic Variation***

- Sorghum BTx623 (NCBIv3)

  * 36.1 million SNPs from the Gates-funded [Global Sorghum Initiative](https://www.globalsorghuminitiative.org/) called in 940 resequenced genomes using the JGIv5 assembly (see [release notes for version 8.0](https://www.sorghumbase.org/relnotes?section=Release%208)) were lifted to the NCBIv3 assembly. 

  * 41 million rsIDs were mapped from BTx623 v3.1 to 16 sorghum accessions: ausTRCF317961, IS19953, Ji2731, R93194522, Chinese Amber, IS36143, PI525695, Rio (PI651496), Tx436, 353, IS12661, IS929, PI532566, PI536008, S3691, and Tx2783.


### New Features

* Search results are now grouped and tallied by species. In other words, the lowest common ancestor in the taxonomy tree for all of the genes that are associated with a suggestion is appended after the number of hit genes for a query.


### New Papers & Funded Projects

- 121 sorghum papers were added to the [Publications](https://www.sorghumbase.org/publications) database with 30 of them described in more detail in [research highlights](https://www.sorghumbase.org/posts?categories=research-highlights). Corresponding title, abstract, authors, keywords, PubMed id, and doi are searchable.

- 35 funded projects for Sorghum research were added to the site and integrated into the site-wide search to a total of 340. The [Projects](https://www.sorghumbase.org/projects) page shows the location map of institutions, and a searchable table of awards based on  PI, agency, award dates etc. Each project has its own page with basic information from the funding agency. We invite PIs to work with us to enrich their project pages to include links to publications, blog entries, project web pages, etc.


## Release Contents -- Databases

### Genomes & Gene Annotations

This release includes a total of 85 sorghum assemblies including four versions of BTx623: NCBIv3, JGIv5.1 (restricted under the [Fort Lauderdale Accord](https://www.sanger.ac.uk/wp-content/uploads/fortlauderdalereport.pdf), and two T2T assemblies in addition to seven plant outgroup species. See [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for more details. 


### Variation

Standard rsID identifiers assigned by the EVA were mapped to nearly 41 million SNPs in both BTx623 reference assemblies (NCBIv3 and JGIv5.1), as well as all other sorghum genomes including the 46 ex-PVP breeding lines. In addition, rsIDs were also assigned to Japonica rice and B73 maize SNPs.


**BTx623 (NCBIv3)**

Over XX (78) million sorghum variants including more than XX (65) million naturally occurring SNPs genotyped in over 4,450 [_CHECK_] accessions, and almost 13 million chemically-induced variants (i.e., ethyl methanesulfonate (EMS)-induced mutations) in a mutagenized population BTx623 background. This data was sourced from the following projects:

***Natural variation***

* [Sorghum Genomics Toolbox](https://www.globalsorghuminitiative.org/) - 36.1M SNPs, 940 resequenced genomes
* [Kumar et al (2024)](https://www.sorghumbase.org/paper/24501) - 18.7M SNPs and indels, whole-genome sequencing 365 accessions (including 332 BAP)
* [Lasky et al (2015)](https://www.sorghumbase.org/paper/24502) - 400K SNPs and indels, 1943 georeferenced landraces
* [Boatwright et al (2022)](https://www.sorghumbase.org/paper/20741) - 33M SNPs and 5M indels genotyped by sequencing 400 SAP lines
* [Lozano et al (2021)](https://www.sorghumbase.org/paper/comparative-evolutionary-genetics-of-deleterious-load-in-sorghum-and-maize) - nearly 13M SNPs, 499 sorghum accessions from the TERRA-MEPP, TERRA-REF population panels, and lines previously genotyped by [Mace et al (2013)](https://www.sorghumbase.org/paper/21275)

***Chemically-induced mutagenesis screens***

* [Jiao et al (2023)](https://www.sorghumbase.org/paper/23165) - 8.6 million EMS-induced mutations, 890 lines
* [Addo-Quaye et al (2018)](https://www.sorghumbase.org/paper/19942) - 2.5 million EMS-indiced mutations, 486 M3 lines
* [Jiao et al (2016)](https://www.sorghumbase.org/paper/a-genomic-resource-for-the-development-improvement-and-exploitation-of-sorghum-for-bioenergy) - 1.7 million EMS-induced mutations, 252 M4 mutant pools

_Note_: 41 million rsIDs were mapped from EVA release 5 (July 2023) to the genetic variants in sorghum BTx623 NCBIv3. In the near future, we aim to have rsIDs assigned to all naturally occuring variants in BTx623 in SorghumBase.


**BTx623 (JGI_v5.1)**

* 36.1M SNPs, 940 resequenced sorghum genomes from the [Sorghum Genomics Toolbox](https://www.globalsorghuminitiative.org/) (Global Sorghum Initiative).


***BTx623 rsIDs mapped to other sorghum accessions***

Nearly 41 million standard rsIDs from EVA release 5 (July 2023) were mapped from BTx623 (NCBIv3) to every genome in SorghumBase. [_Confirm this holds true for 74+11, what about multiple versions of BTx623?_]


***Loss of function mutations***

Germplasm with LOF mutations among all sorghum accessions is now available from the Gramene search.


***GWAs***

No updates in the current release, please see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207) for the GWAs hits from the meta-analysis of 234 phenotypic trait datasets of 40 studies (25 qualified for inclusion) on 406 SAP accessions performed by Mural et al (2021).

***QTL data***

No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for the 5,843 sorghum QTLs for 148 sorghum traits imported from the [Sorghum QTL Atlas](https://aussorgm.org.au/).


 **Tx2783**

No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for the ~32.5 million Tx2873 SNPs called in the 400 SAP with the HPC-based genome variant calling GATK4 pipeline [Zhou et al (2023)](https://sorghumbase.org/paper/23243).


### Expression

No updates in the current release, please see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207) for the gene expression data for the _S. bicolor_ BTx623 (NCBIv3) genome reference curated and processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments) consisting of 8 studies with baseline expression and 3 with differential expression. Baseline experiments, as well as expression of paralogs for the reference study and eFP expression visualizations, are available through our Gene Search (Expression panel). [Differential experiments](https://www.ebi.ac.uk/gxa/experiments?kingdom=Plants&species=Sorghum+bicolor&experimentType=Differential) are available at the Expression Atlas. Baseline gene expression is also available from the gene pages of the Genome Browser, see for example [Expression for SORBI_3006G095600](https://ensembl.sorghumbase.org/Sorghum_bicolor/Gene/ExpressionAtlas?g=SORBI_3006G095600;r=6:46566240-46571064). Data for the eFP browser on the Expression tab of the SorghumBase search results projects gene expression from a Developmental and a Stress Atlas by McCormick et al (2018).


### Pathways

No updates in the current release, please see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207) for the 271 orthology-based sorghum pathways for _S. bicolor_ BTx623 (NCBIv3), which were projected from curated Nipponbare rice pathways by [Grameneâ€™s Plant Reactome](https://plantreactome.gramene.org/) Team as of August 2023.

### Regulation

We continue to offer non-coding features (lnc_RNA, ncRNA, ncRNA_gene, pre_miRNA, RNase_MRP_RNA, rRNA, snoRNA, snRNA, SRP_RNA, and tRNA) available as annotations and visualizations for BTx623 (NCBIv3) and lifted to other sorghum accessions, see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207), as well as ATAC-seq and Chip-seq data from [Lou et al (2019)](https://sorghumbase.org/paper/24376) for sorghum, maize, rice and barley.


### Comparative Genomics

[**Gene Trees**](https://ensembl.sorghumbase.org/prot_tree_stats.html). _UPDATE when/if newly annotated ex-PVP genomes and ~11 new genomes have been included in new trees._ There are (42,926) protein-coding gene family trees and (24,055) putative split genes for the (36) plant genomes in SorghumBase. 

[**Synteny**](https://ensembl.sorghumbase.org/compara_analyses.html). No updates in the current release, please see [release notes for version 5.0](https://www.sorghumbase.org/relnotes?section=PRelease%205) for the 83 synteny maps in SorghumBase.

[**Whole-Genome Alignments**](https://ensembl.sorghumbase.org/compara_analyses.html). No updates in the current release, please see [release notes for version 5.0](https://www.sorghumbase.org/relnotes?section=PRelease%205) for the 15 pairwise DNA alignments in SorghumBase.


### Curated Publications

Our database of sorghum research papers has grown to XX (1,127 on Nov 27) publications. The integrated search interface includes matching papers when searching by author, keywords, pubmed ID, or any other text within the title or abstract. Publications in the database can be linked to news items and funding sources, for example: [Boatwright et al (2021)](https://www.sorghumbase.org/post/whole-genome-sequencing-of-400-sorghum-association-panel-sap-accessions-establishes-a-crucial-resource-for-dissecting-genomic-diversity-in-sorghum).

### Funded Projects

With the addition of NNN initiatives funded by the [Sorghum Checkoff](https://www.sorghumcheckoff.com/), our database of funded sorghum research has grown to a total of XX (305) projects.


## References

- [Li M, Chen C, Wang H, Qin H, Hou S, Yang X, Jian J, Gao P, Liu M, Mu Z. 2024.](https://www.sorghumbase.org/paper/24530) "Telomere-to-telomere genome assembly of sorghum." Sci Data. 11(1):835. doi: [10.1038/s41597-024-03664-8](https://doi.org/10.1038/s41597-024-03664-8).

- [Deng Y, Zhou P, Li F, Wang J, Xie K, Liang H, Wang C, Liu B, Zhu Z, Zhou W, Dun B, Lu X, Diao X, He Q. 2024.](https://www.sorghumbase.org/paper/24194) "A complete assembly of the sorghum BTx623 reference genome." Plant Commun. 5(6):100977. doi: [10.1016/j.xplc.2024.100977](https://doi.org/10.1016/j.xplc.2024.100977).

