## SorghumBase Release 3.0
### Released: March 2022
## Summary
Release #3 is out and included updates to the data and usability of the site.  New to this release are over 17 million new genetic variants including naturally occurring SNPs ([Lozano _et al_, 2021](https://www.sorghumbase.org/post/comparative-analysis-of-deleterious-mutations-in-sorghum-versus-maize)) and chemically-induced point mutations ([Addo-Quaye _et al_, 2018](https://www.sorghumbase.org/post/an-improved-high-resolution-method-for-the-in-silico-detection-of-ems-induced-mutations-in-sorghum-mutant-populations-2)). In addition, we have recalled variation in the 252 M4 mutant pools previously described by [Jiao _et al_ (2016)](https://sorghumbase.org/paper/a-sorghum-mutant-resource-as-an-efficient-platform-for-gene-discovery-in-grasses) for a total of over 1.7 million EMS-induced mutations; the increase in the number of variants demonstrates an improvement in the sensitivity of the detection method from the previous release. The release continues to provide access to 18 sorghum genomes and 7 other species used to generate 30,475 protein-coding gene family trees to support phylogenetic analyses and cross-species comparisons. The website and [Events page](https://www.sorghumbase.org/events) were improved. In addition, there are over 250 new publications related to sorghum in our Publications database and 6 news posts describing relevant recent research publications. Core funding for the project is provided by the Agricultural Research Service of the [U.S. Department of Agriculture](http://www.usda.gov/) (USDA ARS 8062-21000-041-00D).

## Release Information
- [Overall Highlights](#overall-highlights)
- [Databases](#databases)
- [References](#references)

## Overall Highlights {#overall-highlights}

**New Data**

   * **[Lozano _et al_ (2021)](https://sorghumbase.org/post/Comparative-Analysis-of-Deleterious-Mutations-in-Sorghum-versus-Maize)** – Nearly 13 million naturally occurring SNPs in 499 sorghum accessions (includes 14 duplicate samples). Included are accessions from the TERRA-MEPP and TERRA-REF population panels, and lines previously genotyped by Mace _et al_ (2013).

- **[Addo-Quaye _et al_ (2018)](https://www.sorghumbase.org/post/an-improved-high-resolution-method-for-the-in-silico-detection-of-ems-induced-mutations-in-sorghum-mutant-populations-2)** – Over 2.6 million point mutations identified in 486 sorghum accessions corresponding to the M3 generation of an EMS-mutagenized sorghum population.


**Updated Data**

- **[Jiao _et al_ (2016)](https://sorghumbase.org/paper/a-sorghum-mutant-resource-as-an-efficient-platform-for-gene-discovery-in-grasses)** – Over 1.7 million EMS-induced mutations recalled in 252 M4 mutant pools.


**Other Updates and Highlights**

- Improved look and feel of the website. For an example, check out our [Events page](https://www.sorghumbase.org/events).

- A significant increase (n=259) in relevant sorghum papers in our [Publications](https://www.sorghumbase.org/publications) database. Six of those are described in more detailed in news items that include author quotes, images and links to relevant websites.


We continue to host 18 sorghum genomes, protein-coding gene trees, plus genetic and phenotypic variation, gene expression and orthology-based pathway projections for the reference genome *S. bicolor* BTx623.


## Databases {#databases}
### Comparative Genomics

[**Gene Trees.**](https://ensembl.sorghumbase.org/prot_tree_stats.html) A total of
30,475 protein-coding gene family trees were constructed using the peptide encoded by
the canonical transcript (i.e., a representative transcript for a given gene) for each
of 796,813 individual genes (836,223 input proteins) from 25 plant genomes. No updates since release 2.

### Variation

Genetic variation data sets for over 17 million sorghum variants including almost 13 million single nucleotide
polymorphisms (SNPs) [(Lozano _et al_, 2020)](https://sorghumbase.org/paper/comparative-evolutionary-genetics-of-deleterious-load-in-sorghum-and-maize) and a combined 4.3 million chemically induced by ethyl methanesulfonate (EMS) point mutations [(Addo-Quaye _et al_, 2018)](https://www.sorghumbase.org/paper/19942) and [(Jiao _et al_, 2016)](https://sorghumbase.org/paper/a-sorghum-mutant-resource-as-an-efficient-platform-for-gene-discovery-in-grasses); and nearly 6,000 QTLs from the [Sorghum QTL Atlas](https://aussorgm.org.au/).

### Expression

Gene expression data for the _S. bicolor_ BTx623 genome reference was curated and
processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments). No other updates since release 2.

### Pathways

Orthology-based sorghum pathways for _S. bicolor_ BTx623 projected from curated Japonica rice pathways by [Gramene’s Plant Reactome](http://gramene.org/). No other updates since release 2.

## References {#references}

Addo-Quaye C, Tuinstra M, Carraro N, Weil C, Dilkes BP. Whole-Genome Sequence Accuracy Is Improved by Replication in a Population of Mutagenized Sorghum. G3 . 2018;8: 1079–1094. 
PMID: 29378822. https://doi.org/10.1534/g3.117.300301

Jiao, Yinping, John J. Burke, Ratan Chopra, Gloria Burow, Junping Chen, Bo Wang, Chad Hayes, Yves Emendack, Doreen Ware, and Zhanguo Xin. 2016.
"A Sorghum Mutant Resource as an Efficient Platform for Gene Discovery in Grasses."
*The Plant Cell*.
PMID: 27354556. https://doi.org/10.1105/tpc.16.00373.

Lozano R, Gazave E, Dos Santos JPR, Stetter MG, Valluru R, Bandillo N, et al. Comparative evolutionary genetics of deleterious load in sorghum and maize. Nat Plants. 2021;7: 17–24. 
PMID: 33452486. https://doi.org10.1038/s41477-020-00834-5.
