## SorghumBase Release 7.0
### Released: March 2024

SorghumBase (https://www.sorghumbase.org) is a USDA-ARS funded resource released to the public in 2021. The Sorghumbase team works closely with the community to support stewardship of sorghum genomics data, establish best-practices on managing the data, and provide opportunities for networking and capacity building within the community. We maintain a database of scientific publications, genome sequences, gene structure annotations, and comparative genomic analyses integrated with curated data from [EBI's Gene Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments), [Bio-Analytical Resources' eFPseq browser](https://bar.utoronto.ca/), [Gramene's Plant Reactome](https://plantreactome.gramene.org), and [OZ Sorghum QTL Atlas](https://aussorgm.org.au/sorghum-qtl-atlas/).

## What's New in release 7?

### New Functionality

- SorghumBase's Search Results enhancements:

  1) Expression tab now features electronic Fluorescent Pictograph (eFP) browsers for sorghum, maize and Arabidopsis. Sorghum gene expression therein is provided as a Developmental Atlas and a Stress Atlas with data from McCormick _et al_ (2018).

  2) Papers tab listing publications describing a gene's function has now a user entry form to suggest gene function given experimental evidence. Currently, there are 15 functionally curated sorghum genes. We invite you to check out your favorite gene and contribute your expertise!

  3) New Sequences tab provides quick access to gene, transcript and protein sequences for gene models.
     
  4) New Germplasm tab lists germplasm bearing a protein-truncating variant (PTVs are putative loss-of-function SNPs) within the canonical transcript of a gene model. The table includes links to GRIN or SorbMutDB (USDA-Lubbock-EMS lines), as well as a way to search for other genes with a PTV in a given germplasm accession. The indexed PTVs include the following predicted functional effects: start lost, stop gained, stop lost, splice site acceptor, and splice site donor.
  
- 214 sorghum papers added to the [Publications](https://www.sorghumbase.org/publications) database with 30 of them described in more detailed in [research highlights](https://www.sorghumbase.org/posts?categories=research-highlights). Corresponding title, abstract, authors, keywords, PubMed id, and doi are searchable.

- 289 Funded projects for Sorghum research have been added to the site and integrated into the site-wide search. A [Projects](https://www.sorghumbase.org/projects) page shows a map of institutions involved, and a searchable table of awards. Each project has its own page with basic information from the funding agency, but we intend to work with PIs to enrich their project pages to include links to publications, blog entries, project web pages, etc.
 
### New Data

***Genetic & Phenotypic Variation***

- Sorghum BTx623 (NCBIv3): 41 million SNPs were assigned standard rsIDs provided by the [European Variation Archive](https://www.ebi.ac.uk/eva/).
  
- Visualization of GWAS hits from a meta-analysis of 234 phenotypic trait datasets on 406 accessions of the Sorghum Association Panel (SAP) extracted from 25 distinct studies and analyzed by [Mural _et al_ (2021)](https://www.sorghumbase.org/post/Inadvertent-Introgression:-Pleiotropy-and-the-Sorghum-Genome). Density plot for the GWAS hits for the traits is available on a karyotype view. Traits will be made searchable by trait ontology (TO) term.

***Gene Regulation***

- Non-coding features such as lnc_RNA, ncRNA, ncRNA_gene, pre_miRNA, RNase_MRP_RNA, rRNA, snoRNA, snRNA, SRP_RNA, and tRNA were extracted from the BTx623 (NCBIv3) annotations, and lifted to other sorghum accessions. These features are now available as tracks in the "Region in Detail" view on all sorghum genome browsers, see for example [Tx2783 at chr1:12,651,008-12,761,009](https://ensembl.sorghumbase.org/Sorghum_tx2783pac/Share/2240e1a028368d8891a625ccc96d972f?redirect=no).


## Summary

In its current release 7, the search interface of SorghumBase features electronic Fluorescent Pictograph (eFP) browsers for sorghum, maize and Arabidopsis via its Expression tab. Sorghum gene expression therein correspond to the Developmental and Stress Atlases by McCormick _et al_ (2018). This is in addition to a new Sequences tab to provide quick access to gene, transcript and protein sequences. Meanwhile, the Papers tab was complemented with an [easy-to-fill form](https://docs.google.com/forms/d/e/1FAIpQLSey-xPyTysdd9c2phXT6kcbjEDeCppG4dLG7LjZCeLpx_KGog/viewform) to _Submit a gene function_. Finally, a new Germplasm tab lists accessions which bear a protein-truncating variant (PTV), the predicted consequence, whether it is a homozygous or heterozygous allele, and the genotyping study from which it was identified. The table includes links to GRIN or SorbMutDB as well as a way to search for other genes will predicted loss-of-function alleles in the germplasm.

In its seventh release, 41 million standard identifiers (rsIDs) provided by the EVA were assigned to SorghuBase's SNP collection. Moreover, visualization of GWAS hits from a meta-analysis of 234 trait datasets on 406 SAP accessions evaluated in 40 distinct studies and analyzed by Mural _et al_ (2021). Non-coding features including lnc_RNA, ncRNA, ncRNA_gene, pre_miRNA, RNase_MRP_RNA, rRNA, snoRNA, snRNA, SRP_RNA, and tRNA were projected from the BTx623 (NCBIv3) to other sorghum accessions and are available as tracks on all sorghum browsers.

Members of the SorghumBase Team contributed to two new publications, one led by Dr. Yinping Jiao and Dr. Zhanguo Xin describing a large sequenced mutant library covering 98% of sorghum genes [(Jiao _et al_, 2023)](https://www.sorghumbase.org/paper/23165), and the other one by Dr. Yong Zhou and Dr. Rod Wing describing the HPC-based GATK4 pipeline used to call 32.5 million SNPs on the Tx2783 genome assembly [(Zhou _et al_, 2024)](https://sorghumbase.org/paper/23243).

In addition to genomic resources, SorghumBase provides weekly updates of news items, conferences, community events and funded research projects. Our publications database includes now 953 sorghum papers, 86 of which have been [highlighted in News posts](https://www.sorghumbase.org/posts?categories=research-highlights). News items and project pages are indexed and searchable, and include information on how to access related data on SorghumBase. A [user guide](https://www.sorghumbase.org/guides) is available from the site and feedback may be provided via a [contact form](https://www.sorghumbase.org/contact). Individual training sessions may be scheduled upon request. Please check the upcoming [events](https://www.sorghumbase.org/events) on the latest webinars, conferences, meetings, etc. relevant to the sorghum research community. You are always welcome to [contact us](https://www.sorghumbase.org/contact), if you are interested in hosting a training event, collaborating with the SorghumBase team, would like to provide feedback or ask questions. 

Funding for the project is provided by the Agricultural Research Service of the [U.S. Department of Agriculture](http://www.usda.gov/) (USDA ARS 8062-21000-051-00D). 


## Release Contents -- Databases

### Genomes & Gene Annotations

This release includes a total of 29 sorghum assemblies including two versions of BTx623 (NCBIv3 and 5.1 from Phytozome, which released under the [Fort Lauderdale Accord](https://www.sanger.ac.uk/wp-content/uploads/fortlauderdalereport.pdf) for its restricted use prior to publication), and seven plant outgroup species. See [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for more details. 

### Variation

**BTx623 (NCBIv3)**

Reference SNP cluster identifiers (rsIDs) assigned by the European Variation Archive (EVA) replaced temporary identifiers in 41 million variants of the total 59 million described in [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206).

***Loss of function mutations***

No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206).

***QTL data***

No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for the 5,843 sorghum QTLs for 148 sorghum traits imported from the [Sorghum QTL Atlas](https://aussorgm.org.au/).

***GWAs***

GWAs hits from a meta-analysis of 234 phenotypic trait datasets of 40 studies (25 qualified for inclusion) on 406 SAP accessions performed by Mural _et al_ (2021).

 **Tx2783**

No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for the ~32.5 million Tx2873 SNPs called in the 400 SAP with the HPC-based genome variant calling GATK4 pipeline [Zhou _et al_ (2023)](https://sorghumbase.org/paper/23243).

### Expression

Gene expression data for the _S. bicolor_ BTx623 (NCBIv3) genome reference was curated and processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments). The set consists of 8 studies with baseline expression and 3 with differential expression. Baseline experiments, as well as expression of paralogs for the reference study and eFP expression visualizations, are available through our Gene Search (Expression panel). [Differential experiments](https://www.ebi.ac.uk/gxa/experiments?kingdom=Plants&species=Sorghum+bicolor&experimentType=Differential) are available at the Expression Atlas. Baseline gene expression is also available from the gene pages of the Genome Browser, see for example [Expression for SORBI_3006G095600](https://ensembl.sorghumbase.org/Sorghum_bicolor/Gene/ExpressionAtlas?g=SORBI_3006G095600;r=6:46566240-46571064). Data for the eFP browser on the Expression tab of the SorghumBase search results projects gene expression from a Developmental and a Stress Atlas by McCormick _et al_ (2018).

### Pathways

There are 271 orthology-based sorghum pathways for _S. bicolor_ BTx623 (NCBIv3), which were projected from curated Nipponbare rice pathways by [Gramene’s Plant Reactome](https://plantreactome.gramene.org/) Team as of August 2023.

### Regulation

Non-coding features such as lnc_RNA, ncRNA, ncRNA_gene, pre_miRNA, RNase_MRP_RNA, rRNA, snoRNA, snRNA, SRP_RNA, and tRNA extracted from the BTx623 (NCBIv3) annotation and visualization were lifted to other sorghum accessions and are available as tracks on all sorghum browsers.


### Comparative Genomics

[**Gene Trees.**](https://ensembl.sorghumbase.org/prot_tree_stats.html). No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for the 42,926 protein-coding gene family trees and 24,055 putative split genes for the 36 plant genomes in SorghumBase.

[**Synteny**](https://ensembl.sorghumbase.org/compara_analyses.html). No updates in the current release, please see [release notes for version 5.0](https://www.sorghumbase.org/relnotes?section=PRelease%205) for the 83 synteny maps in SorghumBase.

[**Whole-Genome Alignments**](https://ensembl.sorghumbase.org/compara_analyses.html). No updates in the current release, please see [release notes for version 5.0](https://www.sorghumbase.org/relnotes?section=PRelease%205) for the 15 pairwise DNA alignments in SorghumBase.


### Curated Publications

Our database of sorghum research papers has grown to 953 publications. The integrated search interface includes matching papers when searching by author, keywords, pubmed ID, or any other text within the title or abstract. Publications in the database can be linked to news items and funding sources, for example: [Boatwright _et al_ (2021)](https://www.sorghumbase.org/post/whole-genome-sequencing-of-400-sorghum-association-panel-sap-accessions-establishes-a-crucial-resource-for-dissecting-genomic-diversity-in-sorghum).


## References

- Jiao Y, Nigam D, Barry K, Daum C, Yoshinaga Y, Lipzen A, Khan A, Parasa SP, Wei S, Lu Z, Tello-Ruiz MK, Dhiman P, Burow G, Hayes C, Chen J, Brandizzi F, Mortimer J, Ware D, Xin Z. 2023. "A large sequenced mutant library – valuable reverse genetic resource that covers 98% of sorghum genes." The Plant Journal 117(5):1543-1557. doi: [10.1111/tpj.16582](https://doi.org/10.1111/tpj.16582).

- McCormick RF, Truong SK, Sreedasyam A, Jenkins J, Shu S, Sims D, Kennedy M, et al. 2018. "The Sorghum Bicolor Reference Genome: Improved Assembly, Gene Annotations, a Transcriptome Atlas, and Signatures of Genome Organization." The Plant Journal 93(2):338–54. doi: [10.1111/tpj.13781](https://doi.org/10.1111/tpj.13781).

- Mural RV, Grzybowski M, Miao C, Damke A, Sapkota S, Boyles RE, Fernandez MGS, Schnable PS, Sigmon B, Kresovich S, Schnable JC. 2021. "Meta-analysis identifies pleiotropic loci controlling phenotypic trade-offs in sorghum." Genetics 218:3. doi: [10.1093/genetics/iyab087](https://doi.org/10.1093/genetics/iyab087).
  
- Zhou Y, Kathiresan N, Yu Z, Rivera LF, Yang Y, Thimma M, Manickam K, Chebotarov D, Mauleon R, Chougule K, Wei S, Gao T, Green CD, Zuccolo A, Xie W, Ware D, Zhang J, McNally KL, Wing RA. 2024. "A high-performance computational workflow to accelerate GATK SNP detection across a 25-genome dataset." BMC biology. doi: [10.1186/s12915-024-01820-5](https://doi.org/10.1186/s12915-024-01820-5).
