## SorghumBase Release 6.0
### Released: June 2023

SorghumBase (https://www.sorghumbase.org) is a USDA-ARS funded resource released to the public in 2021. The Sorghumbase team works closely with the community to support stewardship of sorghum genomics data, establish best-practices on managing the data, and provide opportunities for networking and capacity building within the community. We maintain a database of scientific publications, genome sequences, gene structure annotations, and comparative genomic analyses integrated with curated data from [EBI's Gene Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments), [Gramene's Plant Reactome](https://plantreactome.gramene.org), and [OZ Sorghum QTL Atlas](https://aussorgm.org.au/sorghum-qtl-atlas/).

## What's New in release 6?

### New Functionality

- Updated ENA identifiers for 12 sorghum assemblies, see [species list](https://ensembl.sorghumbase.org/species.html)

- 12,966 QTLs (330 traits) are now searchable in SorghumBase's search interface including 5,843 sorghum QTLs (148 traits) from the [Sorghum QTL Atlas](https://aussorgm.org.au/sorghum-qtl-atlas/), and 7,123 rice QTLs from [Gramene](https://www.gramene.org). Example: [sorghum grain mold](https://sorghumbase.org/genes?filters={%22status%22:%22init%22,%22operation%22:%22AND%22,%22negate%22:false,%22marked%22:false,%22leftIdx%22:0,%22rightIdx%22:3,%22children%22:[{%22fq_field%22:%22QTL_TO__ancestors%22,%22fq_value%22:%2220008%22,%22name%22:%22sorghum%20grain%20mold%22,%22category%22:%22QTLs%22,%22leftIdx%22:1,%22rightIdx%22:2,%22negate%22:false,%22showMenu%22:false,%22marked%22:true}],%22showMarked%22:true,%22showMenu%22:false,%22moveCopyMode%22:%22%22,%22searchOffset%22:0,%22rows%22:20}&genomes=)

- New links to the Gene Tree Curation Tool in the Homology tab of BTx623 genes. Example: [Sobic.006G095600](http://curate.gramene.org/admin/curationUI/sorghum_v6/?since=4479&gene=SORBI_3006G095600)

- SorghumBase's search results includes a new Papers tab listing PubMed articles associated with [curated genes from GeneRIF and RAP-DB](https://sorghumbase.org/genes?filters={%22status%22:%22init%22,%22operation%22:%22AND%22,%22negate%22:false,%22marked%22:false,%22leftIdx%22:0,%22rightIdx%22:3,%22children%22:[{%22fq_field%22:%22capabilities%22,%22fq_value%22:%22pubs%22,%22name%22:%22publication%22,%22category%22:%22Curated%22,%22leftIdx%22:1,%22rightIdx%22:2,%22negate%22:false,%22showMenu%22:false,%22marked%22:true}],%22showMarked%22:true,%22showMenu%22:false,%22moveCopyMode%22:%22%22,%22searchOffset%22:0,%22rows%22:20}&genomes=). Asssociated Plant Ontology (PO) and Trait Ontology (TO) terms are now searchable, example: [PO: Anatomy: root](https://sorghumbase.org/genes?filters={%22status%22:%22init%22,%22operation%22:%22AND%22,%22negate%22:false,%22marked%22:false,%22leftIdx%22:0,%22rightIdx%22:3,%22children%22:[{%22fq_field%22:%22PO__ancestors%22,%22fq_value%22:%229005%22,%22name%22:%22root%22,%22category%22:%22Plant%20Ontology:%20anatomy%22,%22leftIdx%22:1,%22rightIdx%22:2,%22negate%22:false,%22showMenu%22:false,%22marked%22:true}],%22showMarked%22:true,%22showMenu%22:false,%22moveCopyMode%22:%22%22,%22searchOffset%22:0,%22rows%22:20}&genomes=) or [TO: drought tolerance](https://sorghumbase.org/genes?filters={%22status%22:%22init%22,%22operation%22:%22AND%22,%22negate%22:false,%22marked%22:true,%22leftIdx%22:0,%22rightIdx%22:3,%22children%22:[{%22fq_field%22:%22TO__ancestors%22,%22fq_value%22:%22276%22,%22name%22:%22drought%20tolerance%22,%22category%22:%22Trait%20ontology%22,%22leftIdx%22:1,%22rightIdx%22:2,%22negate%22:false,%22showMenu%22:false,%22marked%22:true,%22parentIdx%22:0}],%22showMarked%22:true,%22showMenu%22:false,%22moveCopyMode%22:%22%22,%22searchOffset%22:0,%22rows%22:20}&genomes=)

- 181 sorghum papers added to the [Publications](https://www.sorghumbase.org/publications) database with 13 of them described in more detailed in [research highlights](https://www.sorghumbase.org/posts?categories=research-highlights). Corresponding title, abstract, authors, keywords, PubMed id, and doi are searchable


### New Data

***Genomes & Gene Annotations***

- [_Sorghum bicolor_ BTx623 v5.1](https://ensembl.sorghumbase.org/Sorghum_bicolorv5) from [JGI's Phytozome](https://phytozome-next.jgi.doe.gov/info/Sbicolor_v5_1). This assembly is restricted by the [Ft Lauderdale Accord](https://www.sanger.ac.uk/wp-content/uploads/fortlauderdalereport.pdf); therefore, genes in this assembly have only been included in gene family trees and assigned InterPro domains.

**NOTE**: To search for genes in the v5.1 browser, please append ".v5.1" to the corresponding Sobic gene identifier like so: [Sobic.001G121600.v5.1](https://ensembl.sorghumbase.org/Sorghum_bicolorv5/Gene/Summary?db=core;g=Sobic.001G121600.v5.1;r=1:9578725-9579867;t=Sobic.001G121600.1.v5.1).

With the BTx623 v5.1 assembled genome, there are a total of **37 genomes** in SorghumBase: 29 sorghum genomes and 8 non-sorghum genomes.

***Genetic Variation***

- BTx623 v3 (NCBI): 8.6 million EMS-induced mutations in 890 samples
- Tx2783: 32.5M SNPs in 400 Sorghum Association Panel (SAP) lines

****Loss of function mutations****

- 1.6 million [loss-of-function mutations for BTx623](https://ftp.sorghumbase.org/release-6/LossOfFunction_mutations/)


***Comparative genomics***

- 42,926 new gene trees with BTx623 v5.1. See [protein tree statistics](https://ensembl.sorghumbase.org/prot_tree_stats.html?page=sizes)
- 156 putative split genes in BTx623 v5.1 ([24,055 split genes in all 37 genomes](https://sorghumbase.org/genes?filters={%22status%22:%22init%22,%22operation%22:%22AND%22,%22negate%22:false,%22marked%22:false,%22leftIdx%22:0,%22rightIdx%22:3,%22children%22:[{%22fq_field%22:%22homology__gene_split%22,%22fq_value%22:%22*%22,%22name%22:%22Split%20genes%22,%22category%22:%22Homology%22,%22leftIdx%22:1,%22rightIdx%22:2,%22negate%22:false,%22showMenu%22:false,%22marked%22:true}],%22showMarked%22:true,%22showMenu%22:false,%22moveCopyMode%22:%22%22,%22searchOffset%22:0,%22rows%22:20}&genomes=))



## Summary

In its current release 6, the search interface of SorghumBase features a new Papers tab in the search results, which lists PubMed-indexed papers associated with curated genes (7184 arabidopsis, 4006 rice, 354 maize, 15 sorghum, 3 selaginella, and 1 chlamydomonas) from [NCBI's GeneRIF](https://www.ncbi.nlm.nih.gov/gene/about-generif) (Jimeno-Yepes _et al_, 2013) and [RAP-DB](https://rapdb.dna.affrc.go.jp/) (Sakai _et al_, 2013). This makes it now possible to search for Trait Ontology (TO) and Plant Ontology (PO) terms associated with the curated genes. Moreover, nearly 13K QTLs (330 traits), including 5.8K sorghum QTLs (148 traits) from the [OZ Sorghum QTL Atlas](https://aussorgm.org.au/sorghum-qtl-atlas/) were indexed and are now searchable. In addition, a link to the Gene Tree Curation Tool has been added in the Homology results tab of all sorghum BTx623 genes. 

In its sixth release, SorghumBase has 29 sorghum genomes including 6 Sorghum Association Panel (SAP), 11 Bioenergy Panel (BAP), 10 Sorghum Carbon-Partitioning Nested Association Mapping (CP-NAM) accessions, and two assembly versions of the BTx623 reference: version 3 from the National Center for Biotechonology Information (NCBI_v3), and version 5.1 from Phytozome (JGI_v5), and 8 additional outspecies, all of which are available for BLAST sequence alignments. We host about 59 million genetic variants for BTx623 version 3, including over 46 million naturally occuring SNPs and indels, and nearly 13 million novel EMS-induced point mutations, together with 32.5 million SNPs called in Tx2783. All variants in a given gene can be visualized in a downloadable high-resolution image in the context of the gene's protein domains (InterProScan), and analyzed in a downloadable table with relative position and functional impact on a transcript as assessed with a Variant Effect Predictor (VEP). Gene expression and orthology-based pathway projections are available for BTx623 v3. Over 43K protein-coding gene trees support information on orthology, paralogy, putative split gene annotations, synteny maps, and whole-genome DNA alignments. 

In addition to genomic resources, SorghumBase provides weekly updates of news items, conferences, and community events. Our publications database includes now 739 sorghum papers, 53 of which have been [highlighted in recent News posts](https://www.sorghumbase.org/posts?categories=research-highlights). News items are indexed and searchable, and include information on how to access data related to the articles from the site. A [user guide](https://www.sorghumbase.org/guides) is available from the site and feedback may be provided via a [contact form](https://WWW.sorghumbase.org/contact). Individual training sessions may be scheduled upon request. Please check the upcoming [Events](https://www.sorghumbase.org/events) on the latest webinars, conferences, meetings, etc. relevant to the sorghum research community. You are always welcome to [contact us](https://www.sorghumbase.org/contact), if you are interested in hosting a training event, collaborating with the SorghumBase team, would like to provide feedback or ask questions. 

Funding for the project is provided by the Agricultural Research Service of the [U.S. Department of Agriculture](http://www.usda.gov/) (USDA ARS 8062-21000-041-00D). 




## Release Contents -- Databases

### Genomes & Gene Annotations

This release includes version 5.1 of the BTx623 assembly from Phytozome. The v5.1 assembly is a wholly resequenced genome applying PacBio long-read data. The annotation is an update that uses all the v3 resources, but with additional RNA-seq from JGI projects, and full-length transcripts to further improve the completeness of the gene set. For additional details on the methodology for this assembly and gene annotation, see [Phytozome](https://phytozome-next.jgi.doe.gov/info/Sbicolor_v5_1). It is important to note that the v5.1 assembly was released before scientific publication according to the [Fort Lauderdale Accord](https://www.sanger.ac.uk/wp-content/uploads/fortlauderdalereport.pdf) by the Department of Energy's Joint Genome Institute (JGI). The accord restricts publication of articles containing analyses of genes or genomic data on this chromosome-scale assembly prior to publication of a comprehensive genome analysis by JGI and/or its collaborators. Therefore, we are only providing a basic genome browser for the new v5.1 assembly. The assembly has been integrated in the Compara gene trees together with the 28 sorghum genomes in SorghumBase, including BTx623 v3, and run through the InterProScan pipeline to predict protein domains. 


### Variation

**BTx623 (NCBI_v3)**

Genetic variation data sets for BTx623 v3 (NCBI) are available as about 70 million sorghum variants including
nearly 51 million SNPs (33 million SNPs plus 5 million indels genotyped by sequencing 400 SAP lines ([Boatwright _et al_, 2022](https://www.sorghumbase.org/post/whole-genome-sequencing-of-400-sorghum-association-panel-sap-accessions-establishes-a-crucial-resource-for-dissecting-genomic-diversity-in-sorghum)), almost 13 million SNPs determined in 499 sorghum accessions from the TERRA-MEPP, TERRA-REF population panels, and lines previously genotyped by Emma Mace and collaborators in 2013, from the study by [Roberto Lozano and colleagues (2021)](https://www.sorghumbase.org/post/comparative-analysis-of-deleterious-mutations-in-sorghum-versus-maize)); 13.8 EMS-induced mutations (Dr. Zhanguo Xin, manuscript in preparation; [Addo-Quaye _et al_ (2018)](https://www.sorghumbase.org/post/an-improved-high-resolution-method-for-the-in-silico-detection-of-ems-induced-mutations-in-sorghum-mutant-populations-2) and [Jiao _et al_ (2016)](https://www.sorghumbase.org/paper/a-genomic-resource-for-the-development-improvement-and-exploitation-of-sorghum-for-bioenergy); and 5,843 sorghum QTLs for 148 sorghum traits from the [Sorghum QTL Atlas](https://aussorgm.org.au/).

***Loss of function mutations***

Loss-of-function (LOF) mutations are those in which the altered gene product lacks the molecular function of the wild type gene.  In SorghumBase,  the following seven functional consequences in a genetic variant are predicted to result in LOF of a protein coding gene: splice acceptor variant, splice donor variant, stop gained, frameshift variant, stop gained, start lost, and missense variant. In the present release, for each of the above mentioned BTx623 genetic variation studies ([Boatwright _et al_ (2022)](https://www.sorghumbase.org/post/whole-genome-sequencing-of-400-sorghum-association-panel-sap-accessions-establishes-a-crucial-resource-for-dissecting-genomic-diversity-in-sorghum), [Lozano _et al_ (2021)](https://www.sorghumbase.org/post/comparative-analysis-of-deleterious-mutations-in-sorghum-versus-maize); Xin _et al_, manuscript in preparation; [Addo-Quaye _et al_ (2018)](https://www.sorghumbase.org/paper/19942), and [Jiao _et al_ (2016)](https://www.sorghumbase.org/paper/a-genomic-resource-for-the-development-improvement-and-exploitation-of-sorghum-for-bioenergy)), we are making available a list of [LOF mutations](https://ftp.sorghumbase.org/release-6/LossOfFunction_mutations/) including +/- 250 nt flanking sequences for each variant. This data may be downloaded in bulk from [SorghumBase's FTP](https://ftp.sorghumbase.org/release-6/LossOfFunction_mutations/).

 **Tx2783**

About 32.5 million SNPs were called in 400 SAP lines by Rod Wing's group with the HPC-based AFW-GVC/GATK4 pipeline [Yong Zhou, Nagarajan Kathiresan and Rod A. Wing (2023), manuscript in preparation], and are now available from the Tx2783 genome browser. See for example [4_6047465_C_T](https://ensembl.sorghumbase.org/Sorghum_tx2783pac/Variation/Sample?db=core;r=4:6046965-6047965;v=4_6047465_C_T;vdb=variation;vf=11387812), a SNP predicted to introduce a STOP codon and result in a truncated [SbiRTX2783.04G076100](https://ensembl.sorghumbase.org/Sorghum_tx2783pac/Gene/Summary?db=core;g=SbiRTX2783.04G076100;r=4:6046424-6048133;t=SbiRTX2783.04G076100.1;v=4_6047465_C_T;vdb=variation;vf=11387812) protein product.

### Expression

Gene expression data for the _S. bicolor_ BTx623 genome reference was curated and processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments). No other updates since release 2.

### Pathways

There are 268 orthology-based sorghum pathways for _S. bicolor_ BTx623, which were projected from curated Nipponbare rice pathways by [Gramene’s Plant Reactome](https://plantreactome.gramene.org/) Team. No other updates since release 2.


### Comparative Genomics

[**Gene Trees.**](https://ensembl.sorghumbase.org/prot_tree_stats.html) A total of
42,926 protein-coding gene family trees were constructed with the peptide encoded by the canonical transcript (i.e., a representative transcript for a given gene) for each of 1,233,135 individual genes (1,275,791 input proteins) from 36 plant genomes. Putative split genes are available for bulk download from [SorghumBase's FTP](https://ftp.sorghumbase.org/release-6/split_genes).

[**Synteny**](https://ensembl.sorghumbase.org/compara_analyses.html). No updates in the current release, please see [release notes for version 5.0](https://www.sorghumbase.org/relnotes?section=PRelease%205) for the 83 synteny maps in SorghumBase.

[**Whole-Genome Alignments**](https://ensembl.sorghumbase.org/compara_analyses.html). No updates in the current release, please see [release notes for version 5.0](https://www.sorghumbase.org/relnotes?section=PRelease%205) for the 15 pairwise DNA alignments in SorghumBase.


### Curated Publications

Our database of sorghum research papers has grown to 739 publications. The integrated search interface includes matching papers when searching by author, keywords, pubmed ID, or any other text within the title or abstract. 
Publications in the database can be linked to news items and funding sources, for example: [Boatwright _et al_ (2021)](https://www.sorghumbase.org/post/whole-genome-sequencing-of-400-sorghum-association-panel-sap-accessions-establishes-a-crucial-resource-for-dissecting-genomic-diversity-in-sorghum).


## References

- Addo-Quaye C, Tuinstra M, Carraro N, Weil C, Dilkes BP. Whole-Genome Sequence Accuracy Is Improved by Replication in a Population of Mutagenized Sorghum. G3 . 2018;8: 1079–1094. doi: [10.1534/g3.117.300301](https://doi.org/10.1534/g3.117.300301).

- Boatwright JL, Sapkota S, Jin H, Schnable JC, Brenton Z, Boyles R, Kresovich S. 2022. [_Sorghum Association Panel whole-genome sequencing establishes cornerstone resource for dissecting genomic diversity_](https://www.sorghumbase.org/post/whole-genome-sequencing-of-400-sorghum-association-panel-sap-accessions-establishes-a-crucial-resource-for-dissecting-genomic-diversity-in-sorghu). Plant J. PMID: 35653240. doi: [10.1111/tpj.15853](https://doi.org/10.1111/tpj.15853).

- Jiao, Yinping, John J. Burke, Ratan Chopra, Gloria Burow, Junping Chen, Bo Wang, Chad Hayes, Yves Emendack, Doreen Ware, and Zhanguo Xin. 2016. "A Sorghum Mutant Resource as an Efficient Platform for Gene Discovery in Grasses." *The Plant Cell*. doi: [10.1105/tpc.16.00373](https://doi.org/10.1105/tpc.16.00373).

- Jimeno-Yepes, A. J., Sticco, J. C., Mork, J. G., & Aronson, A. R. (2013). GeneRIF indexing: sentence selection based on machine learning. BMC Bioinformatics, 14, 171. doi: [10.1186/1471-2105-14-171](https://doi.org/10.1186/1471-2105-14-171).

- Lozano R, Gazave E, Dos Santos JPR, Stetter MG, Valluru R, Bandillo N, et al. Comparative evolutionary genetics of deleterious load in sorghum and maize. Nat Plants. 2021;7: 17–24. doi: [10.1038/s41477-020-00834-5](https://doi.org/10.1038/s41477-020-00834-5).

- Sakai, H., Lee, S. S., Tanaka, T., Numa, H., Kim, J., Kawahara, Y., Wakimoto, H., Yang, C.-C., Iwamoto, M., Abe, T., Yamada, Y., Muto, A., Inokuchi, H., Ikemura, T., Matsumoto, T., Sasaki, T., & Itoh, T. (2013). Rice Annotation Project Database (RAP-DB): an integrative and interactive database for rice genomics. Plant & Cell Physiology, 54(2), e6. https://doi.org/10.1093/pcp/pcs183.
