## SorghumBase Release 6.0
### Released: May 2023

SorghumBase (https://www.sorghumbase.org) is a USDA-ARS funded resource released to the public in 2021. The Sorghumbase team works closely with the community to support stewardship of sorghum genomics data, establish best-practices on managing the data, and provide opportunities for networking and capacity building within the community. We maintain a database of scientific publications, genome sequences, gene structure annotations, and comparative genomic analyses integrated with curated data from [EBI's Gene Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments), [Gramene's Plant Reactome](https://plantreactome.gramene.org), and [OZ Sorghum QTL Atlas](https://aussorgm.org.au/sorghum-qtl-atlas/).

## What's New in release 6?

**New Functionality**

- SorghumBase's search results includes a new Papers tab listing PubMed articles associated with curated genes from GeneRIF and RAP-DB. Asssociated PO and TO terms are now searchable

- 12,966 QTLs (330 traits) are now searchable in SorghumBase's search interface including 5,843 sorghum QTLs (148 traits) from the [Sorghum QTL Atlas](https://aussorgm.org.au/sorghum-qtl-atlas/), and 7,123 rice QTLs from [Gramene](https://www.gramene.org)

- 181 sorghum papers added to the [Publications](https://www.sorghumbase.org/publications) database with 13 of them described in more detailed in [research highlights](https://www.sorghumbase.org/posts?categories=research-highlights). Corresponding title, abstract, authors, keywords, PubMed id, and doi are searchable

- New links to the Gene Tree Curation Tool in the Homology tab of BTx623 genes

- Updated ENA identifiers for 12 sorghum assemblies


**New Data**


***Genomes & Gene Annotations***

- [_Sorghum bicolor_ BTx623 v5](https://ensembl-dev.sorghumbase.org/Sorghum_bicolorv5) from JGI's [Phytozome](https://phytozome-next.jgi.doe.gov/info/Sbicolor_v5_1) restricted by the Ft Lauderdale Accord. Genes in this assembly have been included in gene family trees and assigned InterPro domains.

**NOTE**: To search for genes in the v5.1 browser, please append ".v5.1" to the corresponding Sobic gene identifier like so: [Sobic.001G121600.v5.1](https://ensembl-dev.sorghumbase.org/Sorghum_bicolorv5/Gene/Summary?db=core;g=Sobic.001G121600.v5.1;r=1:9578725-9579867;t=Sobic.001G121600.1.v5.1).

(_Internal note_: There are 1,958 less protein-coding genes in v5 (32,160 in Phytozome) compared to v3 (34,118 in Gramene/SB/Ensembl, from our BTx623 annotation section & Mart) -- Check diff, transcript not mapped, compare Sbicolor_730_v5.1.synonym.txt Phytozome against Sunita’s mapping file for v3 gene ids; also Sbicolor_730_v5.1.locus_transcript_name_map.txt shows mappings from v5.1 to v3.1.1, no transcript in a line means that locus is mapped and transcript is not. Q on a diff note, _should we consider providing a mapping file?_)


***Comparative genomics***

- New gene trees with BTx623 v5.1
- Putative split genes identified for BTx623 v5.1


***Genetic Variation***

- BTx623 v3 (NCBI): 8.6 million EMS-induced mutations in 890 samples
- Tx2783: 32.5M SNPs in 400 SAP lines



## Summary

In its current release 6, the search interface of SorghumBase features a new Papers tab in the search results, which lists PubMed-indexed papers associated with curated genes (7184 arabidopsis, 4006 rice, 354 maize, 15 sorghum, 3 selaginella, and 1 chlamydomonas) from [NCBI's GeneRIF](https://www.ncbi.nlm.nih.gov/gene/about-generif) (Jimeno-Yepes _et al_, 2013) and [RAP-DB](https://rapdb.dna.affrc.go.jp/) (Sakai _et al_, 2013). This data makes it now possible to search for Trait Ontology (TO) and Plant Ontology (PO) terms associated with the curated genes. Moreover, over 6000 QTL-gene associations in 223 traits (~150 studies) imported from the Sorghum QTL Atlas, were indexed and are now searchable. In addition, a link to the Gene Tree Curation Tool has been added in the Homology results tab of all sorghum genes. 

In its sixth release, SorghumBase has 29 sorghum genomes including 10 CP-NAM, 11 BAP, 6 SAP, and two assembly versions of the BTx623 reference: version 3 from the National Center for Biotechonology Information (NCBI_v3), and version 5.1 from Phytozome (JGI_v5), and 8 additional outspecies, all of which are available for BLAST sequence alignments. Over 43K protein-coding gene trees support information on orthology, paralogy, putative split gene annotations, synteny maps, and whole-genome DNA alignments. We host about 59 million genetic variants for BTx623 version 3, including over 46 million naturally occuring SNPs and indels, and nearly 13 million novel EMS-induced point mutations, together with 32.5 million SNPs called in Tx2783. All variants can be visualized in the context of the gene's protein domains (InterProScan), and their functional impact on a transcript is assessed with a Variant Effect Predictor (VEP). Gene expression and orthology-based pathway projections are available for BTx623 v3

In addition to genomic resources, SorghumBase provides weekly updates of news items, conferences, and community events. Our publications database includes now 739 sorghum papers, 53 of which have been [highlighted in recent News posts](https://www.sorghumbase.org/posts?categories=research-highlights). News items are indexed and searchable, and include information on how to access data related to the articles from the site. A [user guide](https://www.sorghumbase.org/guides) is available from the site and feedback may be provided via a [contact form](https://WWW.sorghumbase.org/contact). Individual training sessions may be scheduled upon request. Please check the upcoming [Events](https://www.sorghumbase.org/events) on the latest webinars, conferences, meetings, etc. relevant to the sorghum research community. You are always welcome to [contact us](https://www.sorghumbase.org/contact), if you are interested in hosting a training event, collaborating with the SorghumBase team, would like to provide feedback or ask questions. 

Funding for the project is provided by the Agricultural Research Service of the [U.S. Department of Agriculture](http://www.usda.gov/) (USDA ARS 8062-21000-041-00D). 




## Release Contents -- Databases

### Genomes & Gene Annotations

Working on this


### Comparative Genomics

[**Gene Trees.**](https://ensembl.sorghumbase.org/prot_tree_stats.html) A total of
42,926 protein-coding gene family trees were constructed using the peptide encoded by
the canonical transcript (i.e., a representative transcript for a given gene) for each
of 1,233,135 individual genes (1,275,791 input proteins) from 36 plant genomes.

(_Internal note_: This is less than in R5 for same number of genomes: 43,673 trees - 1,239,742 individual genes (1,283,146 proteins) from 36 plant genomes. Q: _Shouldn't there be 37 genomes in R6 because of the addition of v5.1? Was one of the outgroups dropped?)

[**Synteny**](https://ensembl.sorghumbase.org/compara_analyses.html). No updates in the current release, please see [release notes for version 5.0](https://www.sorghumbase.org/relnotes?section=PRelease%205) for the 83 synteny maps in SorghumBase.

[**Whole-Genome Alignments**](https://ensembl.sorghumbase.org/compara_analyses.html). No updates in the current release, please see [release notes for version 5.0](https://www.sorghumbase.org/relnotes?section=PRelease%205) for the 5 pairwise DNA alignments in SorghumBase.


### Variation

**BTx623 (NCBI_v3)**:

Genetic variation data sets for BTx623 v3 (NCBI) are available as about 70 million sorghum variants including
nearly 51 million SNPs (33 million SNPs plus 5 million indels genotyped by sequencing 400 SAP lines [Boatwright _et al_, 2022](https://www.sorghumbase.org/post/whole-genome-sequencing-of-400-sorghum-association-panel-sap-accessions-establishes-a-crucial-resource-for-dissecting-genomic-diversity-in-sorghum)), almost 13 million SNPs determined in 499 sorghum accessions from the TERRA-MEPP, TERRA-REF population panels, and lines previously genotyped by Emma Mace and collaborators in 2013, from the study by [Roberto Lozano and colleagues (2021)](https://www.sorghumbase.org/post/comparative-analysis-of-deleterious-mutations-in-sorghum-versus-maize);
13.8 EMS-induced mutations (REACH OUT TO ZHANGUO), [(Addo-Quaye _et al_, 2018)](https://www.sorghumbase.org/paper/19942) and [(Jiao _et al_, 2016)](https://sorghumbase.org/paper/a-sorghum-mutant-resource-as-an-efficient-platform-for-gene-discovery-in-grasses), 
and nearly 6,000 QTLs for 220 sorghum traits from the [Sorghum QTL Atlas](https://aussorgm.org.au/).

 **Tx2783**

About 32.5 million SNPs called in the 400 SAP lines SNPs called with the GATK4 pipeline by Rod Wing's group at King Abdullah University of Science and Technology [REACH OUT TO ROD FOR SENTENCE], are now available from the Tx2783 genome browser, see for example [4_6047465_C_T](https://ensembl-dev.sorghumbase.org/Sorghum_tx2783pac/Variation/Sample?db=core;r=4:6046965-6047965;v=4_6047465_C_T;vdb=variation;vf=11387812), a SNP predicted to introduce a STOP codon and result in a truncated [SbiRTX2783.04G076100](https://ensembl-dev.sorghumbase.org/Sorghum_tx2783pac/Gene/Summary?db=core;g=SbiRTX2783.04G076100;r=4:6046424-6048133;t=SbiRTX2783.04G076100.1;v=4_6047465_C_T;vdb=variation;vf=11387812) protein product.


### Expression

Gene expression data for the _S. bicolor_ BTx623 genome reference was curated and processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments). No other updates since release 2.

### Pathways

There are 278 orthology-based sorghum pathways for _S. bicolor_ BTx623, which were projected from curated Nipponbare rice pathways by [Gramene’s Plant Reactome](https://plantreactome.gramene.org/) Team. No other updates since release 2.

### Curated Publications

Our database of sorghum research papers has grown to 739 publications. The integrated search interface includes matching papers when searching by author, keywords, pubmed ID, or any other text within the title or abstract. 
Publications in the database can be linked to news items and funding sources, for example: [Boatwright _et al_(2021)](https://www.sorghumbase.org/post/whole-genome-sequencing-of-400-sorghum-association-panel-sap-accessions-establishes-a-crucial-resource-for-dissecting-genomic-diversity-in-sorghum).


## References

- Jimeno-Yepes, A. J., Sticco, J. C., Mork, J. G., & Aronson, A. R. (2013). GeneRIF indexing: sentence selection based on machine learning. BMC Bioinformatics, 14, 171. https://doi.org/10.1186/1471-2105-14-171.
- Sakai, H., Lee, S. S., Tanaka, T., Numa, H., Kim, J., Kawahara, Y., Wakimoto, H., Yang, C.-C., Iwamoto, M., Abe, T., Yamada, Y., Muto, A., Inokuchi, H., Ikemura, T., Matsumoto, T., Sasaki, T., & Itoh, T. (2013). Rice Annotation Project Database (RAP-DB): an integrative and interactive database for rice genomics. Plant & Cell Physiology, 54(2), e6. https://doi.org/10.1093/pcp/pcs183.
