## SorghumBase Release 3.0
### Released: March 2022
## Summary
[SorghumBase](https://www.sorghumbase.org/), a web portal for comparative plant genomics focused on sorghum crop varieties, has now released its third version. Release #3 focus on genetic variation. New to this release are over 17 million new genetic variants, including naturally occurring SNPs ([Lozano _et al_, 2021](https://www.sorghumbase.org/post/comparative-analysis-of-deleterious-mutations-in-sorghum-versus-maize) and chemically induced point mutations ([Addo-Quaye _et al_, 2018](https://www.sorghumbase.org/post/an-improved-high-resolution-method-for-the-in-silico-detection-of-ems-induced-mutations-in-sorghum-mutant-populations-2)). In addition, over 1.7 million EMS-induced mutations in the 252 M4 mutant pools previously described by [Jiao _et al_ (2016)](https://sorghumbase.org/paper/a-sorghum-mutant-resource-as-an-efficient-platform-for-gene-discovery-in-grasses) were recalled for this set. In addition, this release continues to include nearly 6,000 QTLs from the Sorghum QTL Atlas. The release continues to provide access to a total of 18 sorghum genomes and 7 other species used to generate 30,475 protein-coding gene family trees to support phylogenetic analyses and cross-species comparisons. The genome and genetic variation databases were built in direct collaboration with the [Gramene](http://gramene.org) and [Ensembl Plants](http://plants.ensembl.org) projects. Other data sets were facilitated via collaborations with the [Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments), the [Sorghum QTL Atlas](https://aussorgm.org.au/), and the [Plant Reactome](https://plantreactome.gramene.org/) databases. Core funding for the project is provided by the Agricultural Research Service of the [U.S. Department of Agriculture](http://www.usda.gov/) (USDA ARS 8062-21000-041-00D).

## Release Information
- [Overall Highlights](#overall-highlights)
- [Databases](#databases)
- [References](#references)

## Overall Highlights {#overall-highlights}

**New Data**

   * **[Lozano _et al_ (2021)](https://sorghumbase.org/post/Comparative-Analysis-of-Deleterious-Mutations-in-Sorghum-versus-Maize)** – Nearly 13 million naturally occurring SNPs in 499 sorghum accessions (includes 14 duplicate samples). Included are accessions from the TERRA-MEPP and TERRA-REF population panels, and lines previously genotyped by Mace _et al_ (2013).

- **[Addo-Quaye _et al_ (2018)](https://www.sorghumbase.org/post/an-improved-high-resolution-method-for-the-in-silico-detection-of-ems-induced-mutations-in-sorghum-mutant-populations-2)** – Over 2.6 million point mutations identified in 486 sorghum accessions corresponding to the M3 generation of an EMS-mutagenized sorghum population.


**Updated Data**

- **[Jiao _et al_ (2016)](https://sorghumbase.org/paper/a-sorghum-mutant-resource-as-an-efficient-platform-for-gene-discovery-in-grasses)** – Over 1.7 million EMS-induced mutations recalled in 252 M4 mutant pools.


**Other Updates and Highlights**

- Improved look and feel of the website. Check out our [Events page](https://www.sorghumbase.org/events).
- A modest increase in relevant sorghum papers in our [Publications](https://www.sorghumbase.org/publications) database.


We continue to host 18 sorghum genomes, protein-coding gene trees, plus genetic and phenotypic variation, gene expression and orthology-based pathway projections for the reference genome *S. bicolor* BTx623.


## Databases {#databases}
### Comparative Genomics

[**Gene Trees.**](https://ensembl.sorghumbase.org/prot_tree_stats.html) A total of
30,475 protein-coding gene family trees were constructed using the peptide encoded by
the canonical transcript (i.e., a representative transcript for a given gene) for each
of 796,813 individual genes (836,223 input proteins) from 25 plant genomes.

### Variation

Genetic variation data sets for over 17 million sorghum variants including almost 13 million single nucleotide
polymorphisms (SNPs) [(Lozano _et al_, 2020)](https://sorghumbase.org/paper/comparative-evolutionary-genetics-of-deleterious-load-in-sorghum-and-maize) and a combined 4.3 million chemically induced by ethyl methanesulfonate (EMS) point mutations [(Addo-Quaye _et al_, 2018)](https://www.sorghumbase.org/paper/19942) and [(Jiao _et al_, 2016)](https://sorghumbase.org/paper/a-sorghum-mutant-resource-as-an-efficient-platform-for-gene-discovery-in-grasses); and nearly 6,000 QTLs from the [Sorghum QTL Atlas](https://aussorgm.org.au/).

### Expression

Gene expression data for the S. bicolor BTx623 genome reference was curated and
processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments).
The set consists of seven studies with baseline expression in BTx623 (Davidson _et al_,
2012; Makita _et al_, 2014; Wang _et al_, 2018; Emms _et al_, 2016; Olson _et al_, 2014; Turco _et al_, 2017; and
one unpublished study with SRA project SRP062564). At the Expression
Atlas site, baseline expression is also available for a bioenergy sorghum genotype
R.07020 (Kebrom _et al_, 2017), and differential expression for three studies, but only one (Dugas _et al_, 2011) in the BTx623 reference.

For the baseline studies in SorghumBase, sampling was done in a single site
(e.g., stem internode), a discrete cell type (e.g., bundle sheath, leaf mesophyll),
organism parts (including seed, spikelet, stem, leaf, flag leaf, inflorescence, pistil,
embryo, endosperm, anther, pericarp, pollen, root, shoot, floral meristem, flower,
vegetative meristem, vascular/non-vascular system) or the whole organism, at one or
multiple developmental stages. For example, Wang and collaborators (Wang _et al_, 2018)
examined 11 matched tissues of maize B73 and sorghum BTx623 at different developmental
stages for gene expression profiling totalling 165 experimental assays.

### Pathways

269 orthology-based sorghum pathways associated with 1,248 S. bicolor BTx623 genes.
These were projected from curated Japonica rice pathways and are linked to [Gramene’s Plant Reactome](http://gramene.org/).

## References {#references}

Addo-Quaye C, Tuinstra M, Carraro N, Weil C, Dilkes BP. Whole-Genome Sequence Accuracy Is Improved by Replication in a Population of Mutagenized Sorghum. G3 . 2018;8: 1079–1094. 
PMID: 29378822. https://doi.org/10.1534/g3.117.300301

Aken, Bronwen L., Sarah Ayling, Daniel Barrell, Laura Clarke, Valery Curwen, Susan Fairley, Julio Fernandez Banet, et al. 2016.
"The Ensembl Gene Annotation System."
*Database: The Journal of Biological Databases and Curation*.
PMID: 27337980. https://doi.org/10.1093/database/baw093.

Brenton, Zachary W., Elizabeth A. Cooper, Mathew T. Myers, Richard E. Boyles, Nadia Shakoor, Kelsey J. Zielinski, Bradley L. Rauh, William C. Bridges, Geoffrey P. Morris, and Stephen Kresovich. 2016.
"A Genomic Resource for the Development, Improvement, and Exploitation of Sorghum for Bioenergy."
*Genetics* 204 (1): 21–33.
PMID: 27356613. https://doi.org/10.1534/genetics.115.183947.

Davidson, Rebecca M., Malali Gowda, Gaurav Moghe, Haining Lin, Brieanne Vaillancourt, Shin-Han Shiu, Ning Jiang, and C. Robin Buell. 2012.
"Comparative Transcriptomics of Three Poaceae Species Reveals Patterns of Gene Expression Evolution."
*The Plant Journal: For Cell and Molecular Biology* 71 (3): 492–502.
PMID: 22443345. https://doi.org/10.1111/j.1365-313X.2012.05005.x.

Emms, David M., Sarah Covshoff, Julian M. Hibberd, and Steven Kelly. 2016.
"Independent and Parallel Evolution of New Genes by Gene Duplication in Two Origins of C4 Photosynthesis Provides New Insight into the Mechanism of Phloem Loading in C4 Species."
*Molecular Biology and Evolution* 33 (7): 1796–1806.
PMID: 27016024. https://doi.org/10.1093/molbev/msw057.

Goodstein, David M., Shengqiang Shu, Russell Howson, Rochak Neupane, Richard D. Hayes, Joni Fazo, Therese Mitros, et al. 2012.
"Phytozome: A Comparative Platform for Green Plant Genomics."
*Nucleic Acids Research* 40 (Database issue): D1178–86.
PMID: 22110026. https://doi.org/10.1093/nar/gkr944.

Jiao, Yinping, John J. Burke, Ratan Chopra, Gloria Burow, Junping Chen, Bo Wang, Chad Hayes, Yves Emendack, Doreen Ware, and Zhanguo Xin. 2016.
"A Sorghum Mutant Resource as an Efficient Platform for Gene Discovery in Grasses."
*The Plant Cell*.
PMID: 27354556. https://doi.org/10.1105/tpc.16.00373.

Lozano R, Gazave E, Dos Santos JPR, Stetter MG, Valluru R, Bandillo N, et al. Comparative evolutionary genetics of deleterious load in sorghum and maize. Nat Plants. 2021;7: 17–24. 
PMID: 33452486. https://doi.org10.1038/s41477-020-00834-5

Makita, Yuko, Setsuko Shimada, Mika Kawashima, Tomoko Kondou-Kuriyama, Tetsuro Toyoda, and Minami Matsui. 2015.
"MOROKOSHI: Transcriptome Database in Sorghum Bicolor."
*Plant & Cell Physiology* 56 (1): e6.
PMID: 25505007. https://doi.org/10.1093/pcp/pcu187.

Olson, Andrew, Robert R. Klein, Diana V. Dugas, Zhenyuan Lu, Michael Regulski, Patricia E. Klein, and Doreen Ware. 2014.
"Expanding and Vetting Sorghum Bicolor Gene Annotations through Transcriptome and Methylome Sequencing."
*The Plant Genome* 7 (2): plantgenome2013.08.0025. https://doi.org/10.3835/plantgenome2013.08.0025.

Turco, Gina M., Kaisa Kajala, Govindarajan Kunde-Ramamoorthy, Chew-Yee Ngan, Andrew Olson, Shweta Deshphande, Denis Tolkunov, et al. 2017.
"DNA Methylation and Gene Expression Regulation Associated with Vascularization in Sorghum Bicolor."
*The New Phytologist* 214 (3): 1213–29.
PMID: 28186631. https://doi.org/10.1111/nph.14448.

Wang, Bo, Michael Regulski, Elizabeth Tseng, Andrew Olson, Sara Goodwin, W. Richard McCombie, and Doreen Ware. 2018.
"A Comparative Transcriptional Landscape of Maize and Sorghum Obtained by Single-Molecule Sequencing."
*Genome Research* 28 (6): 921–32.
PMID: 29712755 https://doi.org/10.1101/gr.227462.117.

