## SorghumBase Release 10
### Released: January 2026

SorghumBase (https://www.sorghumbase.org) is a USDA-ARS funded resource released to the public in 2021. The SorghumBase team works closely with the community to support stewardship of sorghum genomics data, establishes best-practices on managing the data, and provides opportunities for networking and capacity building within the community. We maintain a database of scientific publications, genome sequences, gene structure annotations, and comparative genomic analyses integrated with curated data from [EBI's Gene Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments), [Bio-Analytical Resources' eFPseq browser](https://bar.utoronto.ca/), [Gramene's Plant Reactome](https://plantreactome.gramene.org), and [OZ Sorghum QTL Atlas](https://aussorgm.org.au/sorghum-qtl-atlas/).


## Summary

In its tenth release, SorghumBase (SB) features 31 new _Sorghum bicolor_ ssp. _bicolor_ genomes from [JGI](https://phytozome-next.jgi.doe.gov/). The site now hosts a total of 116 sorghum genome assemblies. Protein-coding gene family analysis constructed 49,455 gene trees from 2,487,060 input gene models representing 68 sorghum and 8 outgroup genomes. Standard identifiers for genetic variation (rsIDs) assigned by the European Variation Archive (EVA) to variants called on BTx623 v3 have been projected to 100 other sorghum assemblies. This resource allows traits to be anchored to stable identifiers and accessed in a pan-genome context.

In an effort to harmonize germplasm identifiers within SorghumBase and based on the recommendation of best practices used by the community and curators of major sorghum germplasm repositories such as GRIN and ICRISAT, where available, we provide genotyping data using standard germplasm identifiers (e.g., PI and IS numbers) and standard SNP identifiers (e.g., rsIDs). 

In the Germplasm tab of the gene search page, accessions with genetic variants causing the Loss of Functions (LOFs) are now organized by study or population, type of variant (e.g., stop gained/lost, frameshift, splice acceptor/donor) and homozygous/heterozygous state.

Besides genomic resources, SorghumBase provides weekly updates of news items, conferences, community events and funded research projects. Our publications database now includes over 1,500 sorghum papers, a subset of which have been [highlighted in News posts](https://www.sorghumbase.org/posts?categories=research-highlights). News items and project pages are indexed and searchable, and include information on how to access related data on SorghumBase. A recently updated [user guide](https://www.sorghumbase.org/guides) is available from the site and feedback may be provided via a [contact form](https://www.sorghumbase.org/contact). Individual training sessions may be scheduled upon request. Please check the upcoming [events](https://www.sorghumbase.org/events) on the latest webinars, conferences, meetings, etc. relevant to the sorghum research community. You are always welcome to [contact us](https://www.sorghumbase.org/contact), if you are interested in hosting a training event, interested in collaboration with the SorghumBase team or would like to provide feedback or ask questions. 

Funding for the project is provided by the Agricultural Research Service of the [U.S. Department of Agriculture](http://www.usda.gov/) (USDA ARS 8062-21000-051-00D). 


## What's New in release 10?

### New Data

***Genomes & Gene Annotations***

We are providing in this release, 31 new genome assemblies:
 
 
***Genetic & Phenotypic Variation***

- Sorghum BTx623 (NCBIv3)

  * 41 million rsIDs were mapped from BTx623 v3.1 to ??? sorghum accessions


### New Features



### New Papers & Funded Projects

- 121 sorghum papers were added to the [Publications](https://www.sorghumbase.org/publications) database with 30 of them described in more detail in [research highlights](https://www.sorghumbase.org/posts?categories=research-highlights). Corresponding title, abstract, authors, keywords, PubMed id, and doi are searchable.

- 35 funded projects for Sorghum research were added to the site and integrated into the site-wide search to a total of 340. The [Projects](https://www.sorghumbase.org/projects) page shows the location map of institutions, and a searchable table of awards based on  PI, agency, award dates etc. Each project has its own page with basic information from the funding agency. We invite PIs to work with us to enrich their project pages to include links to publications, blog entries, project web pages, etc.


## Release Contents -- Databases

### Genomes & Gene Annotations

This release includes a total of 88 sorghum assemblies including four versions of BTx623: NCBIv3, JGIv5.1 (restricted under the [Fort Lauderdale Accord](https://www.sanger.ac.uk/wp-content/uploads/fortlauderdalereport.pdf), and two T2T assemblies in addition to seven plant outgroup species. See [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for more details. 


### Variation

Standard rsID identifiers assigned by the EVA were mapped to nearly 41 million SNPs in both BTx623 reference assemblies (NCBIv3 and JGIv5.1), as well as 46 ex-PVP breeding lines, and 25 other sorghum genomes (see Variation column in the [Species Table](https://ensembl.sorghumbase.org/species.html) for a complete list. In addition, rsIDs were also assigned to Arabidopsis, Japonica rice and B73 maize SNPs.


**BTx623 (NCBIv3)**

Over 78 million sorghum variants including more than 65 million naturally occurring SNPs genotyped in over 4,450 accessions, and almost 13 million chemically-induced variants (i.e., ethyl methanesulfonate (EMS)-induced mutations) in a mutagenized population BTx623 background. This data was sourced from the following projects:

***Natural variation***

* [Sorghum Genomics Toolbox](https://www.globalsorghuminitiative.org/) - 36.1M SNPs, 940 resequenced genomes
* [Kumar et al (2024)](https://www.sorghumbase.org/paper/24501) - 18.7M SNPs and indels, whole-genome sequencing 365 accessions (including 332 BAP)
* [Lasky et al (2015)](https://www.sorghumbase.org/paper/24502) - 400K SNPs and indels, 1943 georeferenced landraces
* [Boatwright et al (2022)](https://www.sorghumbase.org/paper/20741) - 33M SNPs and 5M indels genotyped by sequencing 400 SAP lines
* [Lozano et al (2021)](https://www.sorghumbase.org/paper/comparative-evolutionary-genetics-of-deleterious-load-in-sorghum-and-maize) - nearly 13M SNPs, 499 sorghum accessions from the TERRA-MEPP, TERRA-REF population panels, and lines previously genotyped by [Mace et al (2013)](https://www.sorghumbase.org/paper/21275)

***Chemically-induced mutagenesis screens***

* [Jiao et al (2023)](https://www.sorghumbase.org/paper/23165) - 8.6 million EMS-induced mutations, 890 lines
* [Addo-Quaye et al (2018)](https://www.sorghumbase.org/paper/19942) - 2.5 million EMS-indiced mutations, 486 M3 lines
* [Jiao et al (2016)](https://www.sorghumbase.org/paper/a-sorghum-mutant-resource-as-an-efficient-platform-for-gene-discovery-in-grasses) - 1.7 million EMS-induced mutations, 252 M4 mutant pools


***BTx623 v3 rsIDs remapped to other sorghum accessions***

Nearly 41 million rsIDs mapped from EVA release 5 (July 2023) to the genetic variants in sorghum BTx623 NCBIv3 have been remapped to BTx623 JGIv5, 46 ex-PVP sorghum breeding accessions, and 25 additional sorghum genomes in SorghumBase (including nine newly released in the current release 9).

In the near future, we aim to have rsIDs assigned to all naturally occuring variants in BTx623 in SorghumBase. 


***Loss of function mutations***

Germplasm with LOF mutations mapped to BTx623 v3 (and Japonica rice) is available from the Gramene search.


***GWAs***

No updates in the current release, please see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207) for the GWAs hits from the meta-analysis of 234 phenotypic trait datasets of 40 studies (25 qualified for inclusion) on 406 SAP accessions performed by Mural et al (2021).


***QTL data***

No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for the 5,843 sorghum QTLs for 148 sorghum traits imported from the [Sorghum QTL Atlas](https://aussorgm.org.au/).


**BTx623 (JGI_v5.1)**

* 36.1M SNPs, 940 resequenced sorghum genomes from the [Sorghum Genomics Toolbox](https://www.globalsorghuminitiative.org/) (Global Sorghum Initiative).
  

 **Tx2783**

No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for the ~32.5 million Tx2873 SNPs called in the 400 SAP with the HPC-based genome variant calling GATK4 pipeline [Zhou et al (2023)](https://sorghumbase.org/paper/23243).


### Expression

There are updates on the gene expression data in the curent release. Now the SorghumBase has total 16 bulk expression studies, consisting of 11 studies with baseline expression and 5 with differential expression gene expression data for the _S. bicolor_ BTx623 (NCBIv3) genome reference curated and processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments). The Paralogs tab allows the users to select the baseline or differential study to provide the expression of paralogs for the selected reference study and generates the heatmap. These are available through our Gene Search (Expression panel).  
Data for the eFP browser on the Expression tab of the SorghumBase search results have also been updated form one studies to 10 expression studies. It provides maps based on tissue, developmental stages, and stress conditions. 


### Pathways

No updates in the current release, please see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207) for the 271 orthology-based sorghum pathways for _S. bicolor_ BTx623 (NCBIv3), which were projected from curated Nipponbare rice pathways by [Grameneâ€™s Plant Reactome](https://plantreactome.gramene.org/) Team as of August 2023.

### Regulation

We continue to offer non-coding features (lnc_RNA, ncRNA, ncRNA_gene, pre_miRNA, RNase_MRP_RNA, rRNA, snoRNA, snRNA, SRP_RNA, and tRNA) available as annotations and visualizations for BTx623 (NCBIv3) and lifted to other sorghum accessions, see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207), as well as ATAC-seq and Chip-seq data from [Lou et al (2019)](https://sorghumbase.org/paper/24376) for sorghum, maize, rice and barley.


### Comparative Genomics

[**Gene Trees**](https://ensembl.sorghumbase.org/prot_tree_stats.html). There are 44,841 protein-coding gene family trees and 12,258 putative split genes for 48 plant genomes in SorghumBase (ex-PVP breeding lines and T2T assemblies excluded). 

[**Synteny**](https://ensembl.sorghumbase.org/compara_analyses.html). No updates in the current release, please see [release notes for version 5.0](https://www.sorghumbase.org/relnotes?section=PRelease%205) for the 83 synteny maps in SorghumBase.

[**Whole-Genome Alignments**](https://ensembl.sorghumbase.org/compara_analyses.html). No updates in the current release, please see [release notes for version 5.0](https://www.sorghumbase.org/relnotes?section=PRelease%205) for the 15 pairwise DNA alignments in SorghumBase.


### Curated Publications

Our database of sorghum research papers has grown to 1,127 publications. The integrated search interface includes matching papers when searching by author, keywords, pubmed ID, or any other text within the title or abstract. Publications in the database can be linked to news items and funding sources, for example: [Boatwright et al (2021)](https://www.sorghumbase.org/post/whole-genome-sequencing-of-400-sorghum-association-panel-sap-accessions-establishes-a-crucial-resource-for-dissecting-genomic-diversity-in-sorghum).


### Funded Projects

With the addition of 35 initiatives, our database of funded sorghum research has grown to a total of 340 projects.


## References

- [Li M, Chen C, Wang H, Qin H, Hou S, Yang X, Jian J, Gao P, Liu M, Mu Z. 2024.](https://www.sorghumbase.org/paper/24530) "Telomere-to-telomere genome assembly of sorghum." Sci Data. 11(1):835. doi: [10.1038/s41597-024-03664-8](https://doi.org/10.1038/s41597-024-03664-8).

- [Deng Y, Zhou P, Li F, Wang J, Xie K, Liang H, Wang C, Liu B, Zhu Z, Zhou W, Dun B, Lu X, Diao X, He Q. 2024.](https://www.sorghumbase.org/paper/24194) "A complete assembly of the sorghum BTx623 reference genome." Plant Commun. 5(6):100977. doi: [10.1016/j.xplc.2024.100977](https://doi.org/10.1016/j.xplc.2024.100977).

