## SorghumBase Release 8.0
### Released: August 2024

SorghumBase (https://www.sorghumbase.org) is a USDA-ARS funded resource released to the public in 2021. The SorghumBase team works closely with the community to support stewardship of sorghum genomics data, establish best-practices on managing the data, and provide opportunities for networking and capacity building within the community. We maintain a database of scientific publications, genome sequences, gene structure annotations, and comparative genomic analyses integrated with curated data from [EBI's Gene Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments), [Bio-Analytical Resources' eFPseq browser](https://bar.utoronto.ca/), [Gramene's Plant Reactome](https://plantreactome.gramene.org), and [OZ Sorghum QTL Atlas](https://aussorgm.org.au/sorghum-qtl-atlas/).


## Summary

In its eight release, SorghumBase (SB) features 46 new _Sorghum bicolor_ ssp. _bicolor_ breeding lines with an expired Plant Variety Protection (ex-PVP) Act certificate) sequenced by a team led by Dr. Michael Todd at the Salk Institute to a total of 75 sorghum genomes and XXX sorghum genes. Moreover, a total of 66,928 sorghum pan-genes created using annotations from 28 sorghum genomes in SB release 7 were mapped to each of the 46 ex-PVP accessions by a team led by Dr. Doreen Ware at CSHL. In addition, we are providing 36.1 million SNPs called in 940 resequenced genomes from the Gates-funded [Global Sorghum Initiative](https://www.globalsorghuminitiative.org/) in an effort led by Dr. Nadia Shakoor at the Donald Danforth Plant Science Center, Dr. John Lovell at the HudsonAlpha Institute for Biotechnology, and Dr. Geoff Morris at Colorado State University. This genotyping data was provided in JGI version 5.1 (JGI_V5.1), which is subject to the [Fort Lauderdale accord](https://www.sanger.ac.uk/wp-content/uploads/fortlauderdalereport.pdf) restrictions. In an effort to harmonize germplasm identifiers within SorghumBase and based on the recommendation of best practices used by the community and curators of major sorghum germplasm repositories such as GRIN and ICRISAT, we provide this data using standard germplasm identifiers (e.g., PI and IS numbers) and standard SNP identifiers (e.g., rsIDs).

Besides genomic resources, SorghumBase provides weekly updates of news items, conferences, community events and funded research projects. Our publications database now includes 1,026 sorghum papers, 13 of which have been [highlighted in News posts](https://www.sorghumbase.org/posts?categories=research-highlights). News items and project pages are indexed and searchable, and include information on how to access related data on SorghumBase. A recently updated [user guide](https://www.sorghumbase.org/guides) is available from the site and feedback may be provided via a [contact form](https://www.sorghumbase.org/contact). Individual training sessions may be scheduled upon request. Please check the upcoming [events](https://www.sorghumbase.org/events) on the latest webinars, conferences, meetings, etc. relevant to the sorghum research community. You are always welcome to [contact us](https://www.sorghumbase.org/contact), if you are interested in hosting a training event, collaborating with the SorghumBase team, would like to provide feedback or ask questions. 

Funding for the project is provided by the Agricultural Research Service of the [U.S. Department of Agriculture](http://www.usda.gov/) (USDA ARS 8062-21000-051-00D). 


## What's New in release 8?

### New Data

***Genomes & Gene Annotations***

1) We are providing in this release, the genomes of 46 ex-PVP breeding lines (44 bicolor and 2 drummondii). Ex-PVP accessions are breeding lines with an expired Plant Variety Protection (PVP) Act certificate. These accessions were sequenced and assembled by the groups of Dr. Michael Todd of the Salk Institute and Dr. Nadia Shakoor of the Donald Danforth Plant Science Center. 

  - _Sorghum bicolor (L.) Moench subsp. bicolor_: PI543243 (PH256), PI543246 (PHB279), PI543247 (PHB349), PI544069 (PH352), PI554646 (PH288), PI554647 (PH355), PI554648 (PH356), PI554649 (PH410), PI554650 (PH449), PI554652 (PHB429), PI554654 (PHB432), PI555457 (PHB331), PI561926 (R160), PI562621 (PH538), PI562622 (PHB122), PI562623 (PHB118), PI562624 (PHA118), PI562625 (PHA122), PI574398 (PHA651), PI574406 (PHA151), PI574407 (PHB151), PI594354 (PHWIABQE), PI594355 (PHB1CGQF), PI596332 (PH387LM), PI596567 (PH4KNYE), PI601264 (PH227), PI601415 (PH232), PI601552 (A8087C), PI601553 (PH257), PI601554 (PH289), PI601555 (PH308), PI601556 (PH309), PI601557 (PH310), PI601716 (PH328), PI601717 (PH333), PI601718 (PHA82), PI601719 (PHA86), PI601720 (PHB82), PI601721 (PHB86), PI601743 (HP150), PI601744 (R159), PI601756 (R145), PI602599 (PHBCWNL), and PI602600 (PHBEPOYJ).

  - _S. bicolor (L.) Moench nothosubsp. drummondii (Steud.) de Wet ex Davidse_: PI564085 (WKMIII) and PI595221 (EXCELS235).


2) 66,928 sorghum pan-genes created using annotations from the 28 sorghum genomes in SB release 7 were mapped to each of the 46 ex-PVP accessions and are provided as a track on the corresponding ex-PVP browser pages.


***Genetic & Phenotypic Variation***

- Sorghum BTx623 (JGI_V5.1)

  1) Approximately 41 million rsIDs from the European Variation Archive (EVA) were mapped to SNPs lifted from the NCBIv3 to the JGI_V5.1 assembly.

  2) 36.1 million SNPs were called in 940 resequenced genomes from the Gates-funded [Global Sorghum Initiative](https://www.globalsorghuminitiative.org/). 
     
_Disclaimer_: The chromosome-scale assembly of Sorghum bicolor BTx623 known as Phytozome JGI version 5.1 (JGI_V5.1) is subject to the [Fort Lauderdale accord](https://www.sanger.ac.uk/wp-content/uploads/fortlauderdalereport.pdf) restrictions for publication of articles containing analyses of genomic data or genes prior to publication of a comprehensive genome analysis by JGI and/or its collaborators.

- Ex-PVP breeding lines

  * 41 million rsIDs were mapped from BTx623 v3.1 to the 46 ex-PVP breeding lines by the SorghumBase Team. 


***Regulation***

- Sorghum BTx623 (NCBIv3): ATAC-seq and Chip-seq data from [Lou _et al_ (2019)](https://sorghumbase.org/paper/24376) for sorghum, maize, rice and barley.


### New Papers & Funded Projects

- 73 sorghum papers were added to the [Publications](https://www.sorghumbase.org/publications) database with 13 of them described in more detail in [research highlights](https://www.sorghumbase.org/posts?categories=research-highlights). Corresponding title, abstract, authors, keywords, PubMed id, and doi are searchable.

- 16 funded projects for Sorghum research have been added to the site and integrated into the site-wide search to a total of 305. A [Projects](https://www.sorghumbase.org/projects) page shows a map of institutions involved, and a searchable table of awards. Each project has its own page with basic information from the funding agency; we invite PIs to work with us to enrich their project pages to include links to publications, blog entries, project web pages, etc.


## Release Contents -- Databases

### Genomes & Gene Annotations

This release includes a total of 75 sorghum assemblies including two versions of BTx623 (NCBIv3 and 5.1 from Phytozome, which was released under the [Fort Lauderdale Accord](https://www.sanger.ac.uk/wp-content/uploads/fortlauderdalereport.pdf) for restricted use prior to publication), and seven plant outgroup species. See [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for more details. 

--> _What were the "GenomeGeneFeature" tracks for the five original SB genomes in R7 (e.g., Tx623, Tx436, Tx439 and Rio Gene Feature tracks in the Location view of the Tx2783 genome browser)? Also the "lifted the non coding features from Btx623 v3.1 annotation" - all of them disappeared in the dev R8 site  -- KAPEEL, SHARON_

### Variation

**BTx623 (NCBIv3)**

Over 59 million sorghum variants including more than 46 million naturally occurring Single Nucleotide Polymorphisms (SNPs) genotyped in almost 900 accessions, and almost 13 million chemically-induced variants (i.e., ethyl methanesulfonate (EMS)-induced mutations) in a mutagenized population BTx623 background. 

Standard rsID identifiers assigned by the EVA were mapped to approximately 41 million SNPs in both assemblies, NCBIv3 and JGIv5.1.

**BTx623 (JGI_V5.1)**

Approximately 41 million rsIDs from the European Variation Archive were mapped to SNPs lifted from NCBIv3 to JGI_V5.1.

940 newly resequenced sorghum genomes of the [Global Sorghum Initiative](https://www.globalsorghuminitiative.org/) founded by the Gates Foundation. This data is relative to [S. bicolor JGI_v5.1](https://phytozome-next.jgi.doe.gov/info/Sbicolor_v5_1).

Libraries were resequenced using Illumina HiSeq X10 and Illumina NovaSeq 6000 paired-end sequencing (2 × 150 bp) at HudsonAlpha Institute for Biotechnology and aligned to the Btx623 reference with BWA-mem. The resulting .bam file was filtered for duplicates using Picard (http://broadinstitute.github.io/picard) and realigned around indels using GATK 3.056. Multi-sample SNP calling was done using SAMtools mpileup and Varscan V2.4.089 with a minimum coverage of eight, a minimum alternate allele count of four, and a minimum minor allele frequency of 0.001. The full 2144-library VCF was subsetted to the Gates Foundation samples using bcftools 1.9.

***BTx623 rsIDs mapped to other sorghum accessions***

Standard rsIDs were mapped from BTx623 (NCBIv3) to the 46 ex-PVP breeding lines resequenced by the team lead by Dr. Michael Todd of the Salk Institute.

***Loss of function mutations***

No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206).

***QTL data***

No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for the 5,843 sorghum QTLs for 148 sorghum traits imported from the [Sorghum QTL Atlas](https://aussorgm.org.au/).

***GWAs***

No updates in the current release, please see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207) for the GWAs hits from the meta-analysis of 234 phenotypic trait datasets of 40 studies (25 qualified for inclusion) on 406 SAP accessions performed by Mural _et al_ (2021).


 **Tx2783**

No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for the ~32.5 million Tx2873 SNPs called in the 400 SAP with the HPC-based genome variant calling GATK4 pipeline [Zhou _et al_ (2023)](https://sorghumbase.org/paper/23243).


### Expression

No updates in the current release, please see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207) for the gene expression data for the _S. bicolor_ BTx623 (NCBIv3) genome reference curated and processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments) consisting of 8 studies with baseline expression and 3 with differential expression. Baseline experiments, as well as expression of paralogs for the reference study and eFP expression visualizations, are available through our Gene Search (Expression panel). [Differential experiments](https://www.ebi.ac.uk/gxa/experiments?kingdom=Plants&species=Sorghum+bicolor&experimentType=Differential) are available at the Expression Atlas. Baseline gene expression is also available from the gene pages of the Genome Browser, see for example [Expression for SORBI_3006G095600](https://ensembl.sorghumbase.org/Sorghum_bicolor/Gene/ExpressionAtlas?g=SORBI_3006G095600;r=6:46566240-46571064). Data for the eFP browser on the Expression tab of the SorghumBase search results projects gene expression from a Developmental and a Stress Atlas by McCormick _et al_ (2018).

### Pathways

No updates in the current release, please see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207) for the 271 orthology-based sorghum pathways for _S. bicolor_ BTx623 (NCBIv3), which were projected from curated Nipponbare rice pathways by [Gramene’s Plant Reactome](https://plantreactome.gramene.org/) Team as of August 2023.

### Regulation

In addition to the non-coding features (lnc_RNA, ncRNA, ncRNA_gene, pre_miRNA, RNase_MRP_RNA, rRNA, snoRNA, snRNA, SRP_RNA, and tRNA) available as annotations and visualizations for BTx623 (NCBIv3) and lifted to other sorghum accessions, see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207), we are offering in this release, ATAC-seq and Chip-seq data from [Lou _et al_ (2019)](https://sorghumbase.org/paper/24376) for sorghum.


### Comparative Genomics

[**Gene Trees**](https://ensembl.sorghumbase.org/prot_tree_stats.html). No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=PRelease%206) for the 42,926 protein-coding gene family trees and 24,055 putative split genes for the 36 plant genomes in SorghumBase.

[**Synteny**](https://ensembl.sorghumbase.org/compara_analyses.html). No updates in the current release, please see [release notes for version 5.0](https://www.sorghumbase.org/relnotes?section=PRelease%205) for the 83 synteny maps in SorghumBase.

[**Whole-Genome Alignments**](https://ensembl.sorghumbase.org/compara_analyses.html). No updates in the current release, please see [release notes for version 5.0](https://www.sorghumbase.org/relnotes?section=PRelease%205) for the 15 pairwise DNA alignments in SorghumBase.

--> _Do the mRNA and protein alignments (Rfam Models (LCA) and tRNA Models) have data?  They have not been described in the user guide; I will need more details to include them... -- KAPEEL, SHARON_


### Curated Publications

Our database of sorghum research papers has grown to 1,026 publications. The integrated search interface includes matching papers when searching by author, keywords, pubmed ID, or any other text within the title or abstract. Publications in the database can be linked to news items and funding sources, for example: [Boatwright _et al_ (2021)](https://www.sorghumbase.org/post/whole-genome-sequencing-of-400-sorghum-association-panel-sap-accessions-establishes-a-crucial-resource-for-dissecting-genomic-diversity-in-sorghum).

### Funded Projects

Our database of funded sorghum research includes 305 projects.


## References

- Lu Z, Marand AP, Ricci WA, Ethridge CL, Zhang X, Schmitz RJ. 2019. "The prevalence, evolution and chromatin signatures of plant regulatory elements." Nature Plants. doi: [10.1038/s41477-019-0548-z](https://doi.org/10.1038/s41477-019-0548-z).
- Shumate A and Salzberg SL. 2020. “Liftoff: Accurate Mapping of Gene Annotations.” Bioinformatics 37(12):1639–1643. doi: [10.1093/bioinformatics/btaa1016](https://doi.org/10.1093/bioinformatics/btaa1016).
-  Wang B, Chougule K, Jiao  Y, Olson A, Kumar V, Gladman N, Huang J, Llaca V, Fengler K, Wei X, Wang L, Wang  X, Regulski M, Drenkow J, Gingeras T, Hayes C, Armstrong JS, Huang Y, Xin Z, Ware D. 2024. "High-quality chromosome scale genome assemblies of two important Sorghum inbred lines, Tx2783 and RTx436". NAR Genom Bioinform. 6(3):lqae097. doi: [10.1093/nargab/lqae097](https://doi.org/10.1093/nargab/lqae097).
