## SorghumBase Release 8.0
### Released: August 2024

SorghumBase (https://www.sorghumbase.org) is a USDA-ARS funded resource released to the public in 2021. The Sorghumbase team works closely with the community to support stewardship of sorghum genomics data, establish best-practices on managing the data, and provide opportunities for networking and capacity building within the community. We maintain a database of scientific publications, genome sequences, gene structure annotations, and comparative genomic analyses integrated with curated data from [EBI's Gene Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments), [Bio-Analytical Resources' eFPseq browser](https://bar.utoronto.ca/), [Gramene's Plant Reactome](https://plantreactome.gramene.org), and [OZ Sorghum QTL Atlas](https://aussorgm.org.au/sorghum-qtl-atlas/).

## What's New in release 8?

### New Functionality

- SorghumBase's Search Results enhancements:
  
- N sorghum papers added to the [Publications](https://www.sorghumbase.org/publications) database with n of them described in more detailed in [research highlights](https://www.sorghumbase.org/posts?categories=research-highlights). Corresponding title, abstract, authors, keywords, PubMed id, and doi are searchable.

- N funded projects for Sorghum research have been added to the site and integrated into the site-wide search. A [Projects](https://www.sorghumbase.org/projects) page shows a map of institutions involved, and a searchable table of awards. Each project has its own page with basic information from the funding agency; we invite PIs to work with us to enrich their project pages to include links to publications, blog entries, project web pages, etc.
 
### New Data

***Genomes & Gene Annotations***

A total of 46 new _Sorghum bicolor_ ssp. _bicolor_ genomes are being made available in this release including:

- 46 sorghum ex-PVP accessions (i.e., lines with an expired Plant Variety Protection (PVP) Act certificate) sequenced and assembled by the groups of Dr. Michael Todd of the Salk Institute (and Dr. Nadia Shakoor of the Donald Danforth Plant Science Center?): PI543243 (PH256), PI543246 (PHB279), PI543247 (PHB349), PI544069 (PH352), PI554646 (PH288), PI554647 (PH355), PI554648 (PH356), PI554649 (PH410), PI554650 (PH449), PI554652 (PHB429), PI554654 (PHB432), PI555457 (PHB331), PI561926 (R160), PI562621 (PH538), PI562622 (PHB122), PI562623 (PHB118), PI562624 (PHA118), PI562625 (PHA122), PI564085 (WKMIII), PI574398 (PHA651), PI574406 (PHA151), PI574407 (PHB151), PI594354 (PHWIABQE), PI594355 (PHB1CGQF), PI595221 (EXCELS235), PI596332 (PH387LM), PI596567 (PH4KNYE), PI601264 (PH227), PI601415 (PH232), PI601552 (A8087C), PI601553 (PH257), PI601554 (PH289), PI601555 (PH308), PI601556 (PH309), PI601557 (PH310), PI601716 (PH328), PI601717 (PH333), PI601718 (PHA82), PI601719 (PHA86), PI601720 (PHB82), PI601721 (PHB86), PI601743 (HP150), PI601744 (R159), PI601756 (R145), PI602599 (PHBCWNL), and PI602600 (PHBEPOYJ).

- A total of 66,928 pan-genes developed at CSHL labs using annotations 28 Sorghum genomes was lifted to each ex-PVP accession using Liftoff tool. The lifted pan-gene annotation is provided as track on the browser pages. _Sharon needs to configure the tracks on the browser._

***Genetic & Phenotypic Variation***

- Sorghum BTx623 (JGI_V5.1)

1) 36.1M SNPs called in 940 resequenced genomes from the Gates-funded [Global Sorghum Initiative](https://www.globalsorghuminitiative.org/) - Sorghum Genomics Toolbox. This data was the result of an effort led by Dr. Nadia Shakoor at the Donald Danforth Plant Science Center, Dr. John Lovell at the Hudson Alpha Institute for Biotechnology, and Dr. Geoff Morris at Colorado State University. This genotyping data was provided in JGI version 5.1 (JGI_V5.1) genomic coordinates. JGI_V5.1 is subject to the [Fort Lauderdale accord](https://www.sanger.ac.uk/wp-content/uploads/fortlauderdalereport.pdf) restrictions by which publication of articles containing analyses of genes or genomic data on this chromosome-scale assembly prior to publication of a comprehensive genome analysis by JGI and/or its collaborators. 

**Important note**: The approximately 41 million SNP variants (without their respective genotypes) with standard rsIDs in the Sorghum BTx623 NCBIv3 assembly, have also been mapped to JGI_V5.1. All genotyping data and corresponding frequencies for those variants continues to be available from the [Sorghum BTx623 NCBIv3 assembly browser](https://ensembl.sorghumbase.org/Sorghum_bicolor/Info/Index).

KAPPEEL, SHARON --> 2) SNPs called in the 46 ex-PVP lines in an effort led by Dr. Michael Todd of the Salk Institute and Dr. Nadia Shakoor at the Donald Danforth Plant Science Center? (Then rsIDs mapped to the 30 (of the 46 ex-PVP breeding lines?). Same as rsIDs mapped to the pangennomes or QTLs mapped from BTx623 via rsIDs???


## Summary

In its current release 8, the search interface of SorghumBase features ... ANDREW

In its eigth release, ...

Members of the SorghumBase Team contributed to n publications... 

In addition to genomic resources, SorghumBase provides weekly updates of news items, conferences, community events and funded research projects. Our publications database includes now N sorghum papers, n of which have been [highlighted in News posts](https://www.sorghumbase.org/posts?categories=research-highlights). News items and project pages are indexed and searchable, and include information on how to access related data on SorghumBase. A recently updated [user guide](https://www.sorghumbase.org/guides) is available from the site and feedback may be provided via a [contact form](https://www.sorghumbase.org/contact). Individual training sessions may be scheduled upon request. Please check the upcoming [events](https://www.sorghumbase.org/events) on the latest webinars, conferences, meetings, etc. relevant to the sorghum research community. You are always welcome to [contact us](https://www.sorghumbase.org/contact), if you are interested in hosting a training event, collaborating with the SorghumBase team, would like to provide feedback or ask questions. 

Funding for the project is provided by the Agricultural Research Service of the [U.S. Department of Agriculture](http://www.usda.gov/) (USDA ARS 8062-21000-051-00D). 


## Release Contents -- Databases

### Genomes & Gene Annotations

This release includes a total of 89 sorghum assemblies including two versions of BTx623 (NCBIv3 and 5.1 from Phytozome, which released under the [Fort Lauderdale Accord](https://www.sanger.ac.uk/wp-content/uploads/fortlauderdalereport.pdf) for its restricted use prior to publication), and seven plant outgroup species. See [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for more details. 

### Variation

**BTx623 (NCBIv3)**

Update R7 contents as needed: 

Over 59 million sorghum variants including more than 46 million naturally ocurring Single Nucleotide Polymorphisms (SNPs) genotyped in almost 900 accessions, and almost 13 million chemically-induced variants (i.e., ethyl methanesulfonate (EMS)-induced mutations) in a mutagenized ppopulation BTx623 background.

New: 940 resequenced sorghum genomes [Global Sorghum Initiative](https://www.globalsorghuminitiative.org/) founded by the Gates Foundation. Data relative to https://phytozome-next.jgi.doe.gov/info/Sbicolor_v5_1.

Sequencing methods: Libraries were resequenced using Illumina HiSeq X10 and Illumina NovaSeq 6000 paired-end sequencing (2 × 150 bp) at HudsonAlpha Institute for Biotechnology and aligned to the Btx623 reference with BWA-mem. The resulting .bam file was filtered for duplicates using Picard (http://broadinstitute.github.io/picard) and realigned around indels using GATK 3.056. Multi-sample SNP calling was done using SAMtools mpileup and Varscan V2.4.089 with a minimum coverage of eight, a minimum alternate allele count of four, and a minimum minor allele frequency of 0.001. The full 2144-library VCF was subsetted to the Gates Foundation samples using bcftools 1.9.

Note: Acc samples are from the Ethiopian IBC collection from Taye Tadesse at EIAR. Apparently those genotypes are not currently publicly available.


***BTx623 rsIDs mapped to other sorgum accessions***


***Loss of function mutations***

No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206).

***QTL data***

No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for the 5,843 sorghum QTLs for 148 sorghum traits imported from the [Sorghum QTL Atlas](https://aussorgm.org.au/).

***GWAs***

No updates in the current release, please see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207) for the GWAs hits from the meta-analysis of 234 phenotypic trait datasets of 40 studies (25 qualified for inclusion) on 406 SAP accessions performed by Mural _et al_ (2021).

 **Tx2783**

No updates in the current release, please see [release notes for version 6.0](https://www.sorghumbase.org/relnotes?section=Release%206) for the ~32.5 million Tx2873 SNPs called in the 400 SAP with the HPC-based genome variant calling GATK4 pipeline [Zhou _et al_ (2023)](https://sorghumbase.org/paper/23243).

### Expression

No updates in the current release, please see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207) for the gene expression data for the _S. bicolor_ BTx623 (NCBIv3) genome reference curated and processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments) consisting of 8 studies with baseline expression and 3 with differential expression. Baseline experiments, as well as expression of paralogs for the reference study and eFP expression visualizations, are available through our Gene Search (Expression panel). [Differential experiments](https://www.ebi.ac.uk/gxa/experiments?kingdom=Plants&species=Sorghum+bicolor&experimentType=Differential) are available at the Expression Atlas. Baseline gene expression is also available from the gene pages of the Genome Browser, see for example [Expression for SORBI_3006G095600](https://ensembl.sorghumbase.org/Sorghum_bicolor/Gene/ExpressionAtlas?g=SORBI_3006G095600;r=6:46566240-46571064). Data for the eFP browser on the Expression tab of the SorghumBase search results projects gene expression from a Developmental and a Stress Atlas by McCormick _et al_ (2018).

### Pathways

No updates in the current release, please see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207) for the 271 orthology-based sorghum pathways for _S. bicolor_ BTx623 (NCBIv3), which were projected from curated Nipponbare rice pathways by [Gramene’s Plant Reactome](https://plantreactome.gramene.org/) Team as of August 2023.

### Regulation

No updates in the current release, please see [release notes for version 7.0](https://www.sorghumbase.org/relnotes?section=Release%207) for the non-coding features (lnc_RNA, ncRNA, ncRNA_gene, pre_miRNA, RNase_MRP_RNA, rRNA, snoRNA, snRNA, SRP_RNA, and tRNA) available as annotations and visualizations for BTx623 (NCBIv3), and lifte to other sorghum accessions.


### Comparative Genomics

[**Gene Trees.**](https://ensembl.sorghumbase.org/prot_tree_stats.html). NN protein-coding gene family trees and nn putative split genes for the n plant genomes in SorghumBase.

[**Synteny**](https://ensembl.sorghumbase.org/compara_analyses.html). No updates in the current release, please see [release notes for version 5.0](https://www.sorghumbase.org/relnotes?section=PRelease%205) for the 83 synteny maps in SorghumBase.

[**Whole-Genome Alignments**](https://ensembl.sorghumbase.org/compara_analyses.html). No updates in the current release, please see [release notes for version 5.0](https://www.sorghumbase.org/relnotes?section=PRelease%205) for the 15 pairwise DNA alignments in SorghumBase.


### Curated Publications

Our database of sorghum research papers has grown to N publications. The integrated search interface includes matching papers when searching by author, keywords, pubmed ID, or any other text within the title or abstract. Publications in the database can be linked to news items and funding sources, for example: [Boatwright _et al_ (2021)](https://www.sorghumbase.org/post/whole-genome-sequencing-of-400-sorghum-association-panel-sap-accessions-establishes-a-crucial-resource-for-dissecting-genomic-diversity-in-sorghum).


## References


- McCormick RF, Truong SK, Sreedasyam A, Jenkins J, Shu S, Sims D, Kennedy M, et al. 2018. "The Sorghum Bicolor Reference Genome: Improved Assembly, Gene Annotations, a Transcriptome Atlas, and Signatures of Genome Organization." The Plant Journal 93(2):338–54. doi: [10.1111/tpj.13781](https://doi.org/10.1111/tpj.13781).

- Mural RV, Grzybowski M, Miao C, Damke A, Sapkota S, Boyles RE, Fernandez MGS, Schnable PS, Sigmon B, Kresovich S, Schnable JC. 2021. "Meta-analysis identifies pleiotropic loci controlling phenotypic trade-offs in sorghum." Genetics 218:3. doi: [10.1093/genetics/iyab087](https://doi.org/10.1093/genetics/iyab087).

- Wei C, Gao L, Xiao R, Wang Y, Chen B, Zou W, Li J, Mace E, Jordan D, Tao Y. 2024. Complete telomere-to-telomere assemblies of two sorghum genomes to guide biological discovery." iMeta 3(2):e193. doi: [10.1002/imt2.193](https://doi.org/10.1002/imt2.193)

- Zhou Y, Kathiresan N, Yu Z, Rivera LF, Yang Y, Thimma M, Manickam K, Chebotarov D, Mauleon R, Chougule K, Wei S, Gao T, Green CD, Zuccolo A, Xie W, Ware D, Zhang J, McNally KL, Wing RA. 2024. "A high-performance computational workflow to accelerate GATK SNP detection across a 25-genome dataset." BMC biology. doi: [10.1186/s12915-024-01820-5](https://doi.org/10.1186/s12915-024-01820-5).

