## Gramene Release 65
### Released: April 2022
## Summary

## Release Information
- [Overall Highlights](#overall-highlights)
- [Databases](#databases)
- [References](#references)

## Overall Highlights {#overall-highlights}

**CLIMtools portal**

- The new [CLIMtools](https://gramene.org/CLIMtools) portal brings interactive web-based views of environment x genome associations,
  RiboSNitch prediction, and correlations between the local environment and a pool of curated phenotypes.

**Updated software**

- Improved look and feel of the [homepage](https://gramene.org) and gene search interface.
  - Links to our crop specific pangenome sites for [maize](https://maize-pangenome.gramene.org/), [rice](https://oryza.gramene.org/), [grapevine](https://vitis.gramene.org/) and [sorghum](https://sorghumbase.org).
  - Integrated [@GrameneDatabase](https://twitter.com/GrameneDatabase) twitter feed.
  - The React web application has been rewritten to use [Redux-Bundler](https://reduxbundler.com/) for state management.
- Genome browser and underlying database schema and API updated to [Ensembl 105](https://www.ensembl.info/2021/12/10/ensembl-105-has-been-released/).

**New genomes**

- [_Brassica rapa_ R-o-18](http://ensembl.gramene.org/Brassica_rapa_ro18) replaces cultivar Chiifu-401-42.
- [_Corylus avellana_](http://ensembl.gramene.org/Corylus_avellana) (hazelnut).
- [_Ficus carica_](http://ensembl.gramene.org/Ficus_carica) (common fig).
- [_Lactuca sativa_](http://ensembl.gramene.org/Lactuca_sativa) (lettuce).
- [_Olea europaea_](http://ensembl.gramene.org/Olea_europaea) (European olive tree). Olea europaea sylvestris can still be found [here](http://ensembl.gramene.org/Olea_europaea/Info/Strains?db=core).

**Updated genomes and gene annotation**

- [_Helianthus annuus_](http://ensembl.gramene.org/Helianthus_annuus) (sunflower) assembly updated to HanXRQr2.0-SUNRISE.
- [_Hordeum vulgare_](http://ensembl.gramene.org/Hordeum_vulgare) (barley) assembly updated to Morex V3 with IPK and BaRT gene models. The GoldenPromise and TRITEX varieties can be viewed via the [cultivar view](http://ensembl.gramene.org/Hordeum_vulgare/Info/Strains?db=core).
- [_Chlamydomonas reinhardtii_](http://ensembl.gramene.org/Chlamydomonas_reinhardtii) (green algae) updated with gene name synonyms.
- [_Triticum aestivum_ cultivars](https://ensembl.gramene.org/Triticum_aestivum/Info/Strains) (wheat) updated with de novo genes from the [wheat 10+ genome project](http://www.10wheatgenomes.com/).

**New & updated data**

- [_Triticum aestivum_](https://ensembl.gramene.org/Triticum_aestivum) (wheat) updated with centromere data.
- [_Zea mays_](https://ensembl.gramene.org/Zea_mays) (maize) updated with Plant Reactome cross-references.
- Masking of repeated sequences in 20 angiosperms (including _Arabidopsis thaliana_, _Brachypodium distachyon_, _Brassica rapa_, _Oryza sativa_, _Vitis vinifera_, and _Z. mays_) using k-mer based Repeat Detector (Red).
- SIFT predictions added to VEP output data for several plant species with variation data.


Check which features are supported on each species [here](http://ensembl.gramene.org/species.html); if you would like to request a new species please email [Gramene’s helpdesk](https://www.gramene.org/feedback).

Note that a maximum of 100 genomes is included in the Compara Gene Family Trees analysis, which will change depending on data available for each release. You can find the list of included genomes in [Ensembl’s Compara github repository](https://github.com/Ensembl/ensembl-compara/blob/release/104/conf/plants/allowed_species.json). Click below for a summary of currently available Compara analyses:

- [Protein gene tree analysis](https://ensembl.gramene.org/info/genome/compara/prot_tree_stats.html)
- [Whole-genome alignments summary](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html)



## Databases {#databases}
### Comparative Genomics

[**Gene Trees**](https://ensembl.gramene.org/info/genome/compara/prot_tree_stats.html)
A total of 124,010 protein-coding gene family trees were constructed using the peptide encoded by
the canonical transcript (i.e., a representative transcript for a given gene) for each
of 2,934,710 individual genes (3,247,550 input proteins) from 89 plant genomes.

[**Whole-Genome Alignments**](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html)
There are 320 pairwise genomic alignments. The majority of species in Gramene were aligned with _Oryza sativa_ Japonica (reference monocot), and _Arabidopsis thaliana_ and _Vitis vinifera_ (reference dicots).

[**Synteny**](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html)
Eighty paired synteny maps are available. 

[**Putative Split Genes**](FTP link)
Updated Split Genes: Updated split genes for the current release are available on the FTP site.  These are putative gene annotation artifacts (also known as contiguous gene split models) and are based on the latest Ensembl Compara Gene Tree database. Overall statistics for each species is given in the [README](Vivek/Sunita LINK).


### Protein Annotation, GO, Xref Protein domain information 

These were generated for the new and updated genomes.


### Variation

No new and updated genetic/structural variation data since release 54. Genetic variation data is available for the following 15 species in Gramene:

* Arabidopsis thaliana
* brachypodium
* sunflower
* barley
* apple
* O. glaberrima
* O. glumaepatula
* India rice
* O. sativa japonica
* tomato
* sorghum
* wheat
* durum wheat
* Vitis vinifeera
* Zea mays


### Expression

Baseline gene expression data for 23 plant genomes was curated and processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments) and is available via Gramene's Search, Gramene's Genome Browser, and Gramene's pathways interface, the Plant Reactome. Differential expression for most of those species is available directly in the Atlas website. 


### Pathways

320 manually curated Japonica rice pathways and orthology-based pathways for 106 plant species cross-linked with [Gramene’s Plant Reactome](https://plantreactome.gramene.org/) entries.


### Gramene Mart

New Plant Gene Mart generated to incorporte the new genome data.

## Infrastructure

**Web Services: Gramene's web services page documents many ways to directly connect to and analyze our databases.**

- Public MySQL Server: 

Our partner Ensembl Genomes offers a public, read-only MySQL server with copies of the species-specific and comparative genomic databases that we use. To use this with the mysql command-line client:

  $ mysql -hmysql-eg-publicsql.ebi.ac.uk -P4157 -uanonymous

Please note that the versioning scheme used at this public database differs from ours; Gramene's release 65_105 is set as version 52_105 at Ensembl.

- Website and coding updates

The latest Plant Reactome pathway data has been re-indexed and made available via Gramene search.


## Citing Gramene {#references}

Tello-Ruiz MK, Naithani S, Gupta P, Olson A, Wei S, Preece S, Jiao Y, Wang B, Chougule K, Garg P, Elser J, Kumari S, Kumar V, Contreras-Moreira B, Naamati G, George N, Cook J, Bolser D, D’Eustachio P, Stein LD, Gupta A, Xu W, Regala J, Papatheodorou I, Kersey PJ, Flicek P, Taylor C, Jaiswal P, and Ware D. (2021). "Gramene 2021: Harnessing the power of comparative genomics and pathways for plant research." *Nucleic Acids Research* 49(D1): D1452–D1463. PMID: 33170273. DOI: [10.1093/nar/gkaa979](https://doi.org/10.1093/nar/gkaa979).


