## Gramene Release 69
### Released: July 2025
## Summary

Release #69 includes a total of 200 plant reference genomes. This release features 164,454 gene family trees built from 4,373,089 protein coding genes spanning 119 of the plant genomes. The remaining genomes that were not included in the Compara gene tree build come from pan-genome projects in Hordeum vulgare, Triticum aestivum and Oryza sativa. Each of these projects has crop specific pan-gene trees in the Ensembl browser.

Functional annotations were updated based on the NCBI GeneRIF database and the RAPDB curated gene list (Japonica specific).

The Plant Reactome was updated in April, 2025 and includes 10 new and 5 updated pathways involved in flowering under drought stress, root and shoot gravitropism, and salinity response for a total of 351 manually curated rice pathways projected via orthology to 138 genomes.

One new feature in the search interface is in the Expression tab. Users can now compare baseline and differential expression across paralogs for any RNAseq study in EBI Atlas.

## Release Information
- [New data & features](#new-this-release)
- [Databases](#databases)
- [References](#references)

## New data & features {#new-this-release}

### Genomes
#### New genomes (N):

#### Updated genomes (M):
  
#### Updated gene annotation:

### New features:

## Databases {#databases}
### Comparative Genomics

[**Gene Trees**](https://ensembl.gramene.org/info/genome/compara/prot_tree_stats.html)
A total of 164,454 protein-coding gene family trees were constructed using the peptide encoded by
the canonical transcript (i.e., a representative transcript for a given gene) for each
of 4,373,089 individual genes from 119 plant genomes.

===================NOT UPDATED BELOW THIS LINE==============
[**Whole-Genome Alignments**](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html)
There are 279 pairwise genomic alignments. The majority of species in Gramene were aligned with _Oryza sativa_ Japonica
(reference monocot), and _Arabidopsis thaliana_ and _Vitis vinifera_ (reference dicots).

[**Synteny**](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html)
80 paired synteny maps are available. 

[**Putative Split Genes**](http://ftp.gramene.org/CURRENT_RELEASE/splitgenes/)
Updated split genes for the current release are available on the FTP site. These are putative gene annotation artifacts (also known as contiguous gene split models) and are based on the latest Ensembl Compara Gene Tree database. Overall statistics for each species is given in the [README](https://ftp.gramene.org/CURRENT_RELEASE/split_genes/1_ReadMe_SplitGenesBySpecies.txt).

[**Plant Reactome**](https://plantreactome.gramene.org)
No updates in this release.

Plant Reactome is the pathway knowledgebase of Gramene. We utilize the Reactome pathway data model to represent plant metabolic, transport and signaling pathways, developmental processes, organ differentiation, and transcriptional regulatory networks. Manual biocuration is conducted in the reference species rice (_O. sativa_) and pathways are then projected via gene orthology to other species including single-cell photoautotrophs, lower plants, and higher plants. This release is a maintenance release. Plant Reactome continues to host pathway projections for 130 plant species (99 of which are in Gramene). Additional details and statistics can be found in the Plant Reactome's site [release notes](https://plantreactome.gramene.org/index.php?option=com_content&view=article&id=111&Itemid=360&lang=en).

### Protein Annotation, GO, Xref Protein domain information 

No updates in this release.

### Variation

No updates in this release. Genetic variation data continues to be available for 19 plant species, including _Arabidopsis thaliana_, maize,  rice (Japonica, Indica, _O. glaberrima_, and _O. glumaepatula_), brachypodium, sunflower, barley, apple, tomato, sorghum, grapevine, bread and durum wheat, cucumber, black bean, lettuce, and cocoa. Variation for the last four species is not yet available via BioMart.

### Expression

In this release, we have included embedded eFP Browser views from the [BAR](https://bar.utoronto.ca) API for several studies in Arabidopsis, Soybean, Maize and Sorghum. These views are accessible from the Expression tab in Gramene's search interface. Gramene continues to display Expression Atlas data with a total of 1007 plant experiments. Baseline gene expression data for 23 plant genomes has been curated and processed with the [EMBL-EBI's Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments), and is available via Gramene's Search, Gramene's Genome Browser, and Gramene's pathways interface, the Plant Reactome. Differential expression for most of those species is available directly in the Expression Atlas website. 

### Pathways

No updates in this release. The release continues to host 339 manually curated Japonica rice pathways, and orthology-based pathways for 99 plant species cross-linked with [Grameneâ€™s Plant Reactome](https://plantreactome.gramene.org/) entries.

### Gramene Mart

No updates in this release.

## Infrastructure

**Web Services:** Gramene's [web services page](https://gramene.org/web-services) documents ways to directly connect to and analyze our databases.

- Public MySQL Server: 

Our partner Ensembl Genomes offers a public, read-only MySQL server with copies of the species-specific and comparative genomic databases that we use. To use this with the mysql command-line client:

  $ mysql -hmysql-eg-publicsql.ebi.ac.uk -P4157 -uanonymous

Please note that the versioning scheme used at this public database differs from ours; Gramene's release 68_110 is set as version 57_110 at Ensembl.

## Funding

The Gramene project is currently funded by the United States Department of Agriculture - Agricultural Research Service, USDA ARS grant #8062-21000-041-00D. 
