## Gramene Release 69
### Released: July 2025
## Summary

Release #69 includes a total of 200 plant reference genomes. This release features 164,454 gene family trees built from 4,373,089 protein coding genes spanning 119 of the plant genomes. The remaining genomes that were not included in the Compara gene tree build come from pan-genome projects in Hordeum vulgare, Triticum aestivum and Oryza sativa. Each of these projects has crop specific pan-gene trees in the Ensembl browser.

Functional annotations were updated based on the NCBI GeneRIF database and the RAPDB curated gene list (Japonica specific).

The Plant Reactome was updated in April, 2025 and includes 10 new and 5 updated pathways involved in flowering under drought stress, root and shoot gravitropism, and salinity response for a total of 351 manually curated rice pathways projected via orthology to 138 genomes.

One new feature in the search interface is in the Expression tab. Users can now compare baseline and differential expression across paralogs for any RNAseq study in EBI Atlas.

## Release Information
- [New data & features](#new-this-release)
- [Databases](#databases)
- [References](#references)

## New data & features {#new-this-release}

### Genomes
#### New genomes (83):
- Aegilops umbellulata (Umbel goatgrass; GCA_032464435.1)
- Arachis hypogaea (Peanut; GCA_003086295.3)
- Hordeum vulgare pangenome (75 Barley genomes)
- Lathyrus sativus (Grass pea; GCA_963859935.3)
- Sphenostylis stenocarpa  (African yam bean; GCA_963425845.1)
- Triticum aestivum Alchemy (wheat cultivar; GCA_951799155.1)
- Triticum timopheevii (Sanduri wheat; GCA_963921465.1)
- Vicia faba (Broad bean; GCA_948472305.1)
- Wild soybean (Glycine soja; GCA_004193775.2)

#### Updated genomes (7):
- Brassica rapa (Field mustard; GCA_900412535.3)
- Cork oak (Quercus suber; GCA_002906115.4) 
- Marchantia polymorpha (Common liverwort; GCA_039105155.1)
- Manihot esculenta (Cassava; GCA_001659605.2)
- Medicago truncatula (Barrel Medic; GCA_003473485.2)
- Triticum aestivum (Common wheat; Paragon v2; GCA_949126075.1)
- Vitis vinifera PN40024 T2T assembly (Grapes, GCA_030704535.1)

### New features:
Explore functional conservation or specialization in a gene family within the Expression tab of the gene search interface. The default view loads the EBI Expression Atlas widget for expressed paralogs of the gene of interest. It is now possible to choose any baseline or differential expression RNA-seq study indexed by EBI Atlas to compare expression profiles across the gene family. The set of paralogs may span multiple gene trees if they belong to a large gene family (supertree).

Within the Homology tab, if a large gene family supertree was split into multiple trees, the Search Filters section includes a button which invokes a search for all genes in the supertree.

## Databases {#databases}
### Comparative Genomics

[**Gene Trees**](https://ensembl.gramene.org/info/genome/compara/prot_tree_stats.html)
A total of 164,454 protein-coding gene family trees were constructed using the peptide encoded by
the canonical transcript (i.e., a representative transcript for a given gene) for each
of 4,373,089 individual genes from 119 plant genomes.

[**Whole-Genome Alignments**](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html)
There are 598 pairwise genomic alignments among 144 species. The majority of species in Gramene were aligned with _Oryza sativa_ Japonica
(reference monocot), and _Arabidopsis thaliana_ and _Vitis vinifera_ (reference dicots).

[**Synteny**](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html)
204 paired synteny maps are available. 

[**Putative Split Genes**](http://ftp.gramene.org/CURRENT_RELEASE/splitgenes/)
Updated split genes for the current release are available on the FTP site. These are putative gene annotation artifacts (also known as contiguous gene split models) and are based on the latest Ensembl Compara Gene Tree database. Overall statistics for each species is given in the [README](https://ftp.gramene.org/CURRENT_RELEASE/split_genes/1_ReadMe_SplitGenesBySpecies.txt).

### Protein Annotation, GO, Xref Protein domain information 

No updates in this release.

### Variation

Additional Watkins Wheat variation data has been added for Triticum aestivum (Common wheat; GCA_900519105.1). The Watkins Wheat variation collection encompasses the genetic diversity found in the A.E. Watkins Landrace Collection of bread wheat, which includes 827 landraces from 32 countries. 

Genetic variation data continues to be available for 19 plant species, including _Arabidopsis thaliana_, maize,  rice (Japonica, Indica, _O. glaberrima_, and _O. glumaepatula_), brachypodium, sunflower, barley, apple, tomato, sorghum, grapevine, bread and durum wheat, cucumber, black bean, lettuce, and cocoa.

### Expression

In this release, we have included embedded eFP Browser views from the [BAR](https://bar.utoronto.ca) API for several studies in Arabidopsis, Soybean, Maize and Sorghum. These views are accessible from the Expression tab in Gramene's search interface. Gramene continues to display Expression Atlas data with a total of 460 plant RNA-seq experiments. Baseline gene expression data for 23 plant genomes has been curated and processed with the [EMBL-EBI's Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments), and is available via Gramene's Search, Gramene's Genome Browser, and Gramene's pathways interface, the Plant Reactome. Differential expression for most of those species is available within the Gramene Search interface and directly in the Expression Atlas website. 

New studies in this release include three baseline expression studies and two differential expression (DE) studies in Sorghum bicolor, one maize DE study, two rice DE studies, and two arabidopsis DE studies. 

| Type         | Species     | Study id                                                                                  | Description                                                                                      |
|--------------|-------------|---------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|
| Baseline     | sorghum     | [E-GEOD-141035](https://www.ebi.ac.uk/gxa/experiments/E-GEOD-141035)                 | Transcriptional trajectories of anther development provide candidates for engineering male fertility in sorghum. |
| Baseline     | sorghum     | [E-GEOD-151860](https://www.ebi.ac.uk/gxa/experiments/E-GEOD-151860)                 | Variation in gene expression between two Sorghum bicolor lines differing in innate immunity response |
| Baseline     | sorghum     | [E-GEOD-167101](https://www.ebi.ac.uk/gxa/experiments/E-GEOD-167101)                 | Characterization of growth and development of sorghum genotypes with differential susceptibility to Striga hermonthica |
| Differential | arabidopsis | [E-MTAB-10374](https://www.ebi.ac.uk/gxa/experiments/E-MTAB-10374)                   | Distinct Cellular Strategies Determine Sensitivity to Mild Drought of Arabidopsis Natural Accessions |
| Differential | arabidopsis | [E-MTAB-10791](https://www.ebi.ac.uk/gxa/experiments/E-MTAB-10791)                   | RNA-seq in Arabidopsis thaliana Col and hcr2 mutant                                              |
| Differential | arabidopsis | [E-MTAB-10920](https://www.ebi.ac.uk/gxa/experiments/E-MTAB-10920)                   | RNAseq of wild type Col-0 Arabidopsis plants and piezo/feronia/camta3 mutants before and 22 minutes after touch treatment |
| Differential | maize       | [E-CURD-147](https://www.ebi.ac.uk/gxa/experiments/E-CURD-147)                       | Comparative transcriptome analysis reveals conserved defence responses in maize and sorghum to Exserohilum turcicum - maize data |
| Differential | rice        | [E-MTAB-10412](https://www.ebi.ac.uk/gxa/experiments/E-MTAB-10412)                   | RNA-Seq of wild type and osmtd2 mutant rice anthers                                              |
| Differential | rice        | [E-MTAB-10439](https://www.ebi.ac.uk/gxa/experiments/E-MTAB-10439)                   | RNA-Seq of wild type and tape-1 mutant rice anthers                                              |
| Differential | sorghum     | [E-CURD-148](https://www.ebi.ac.uk/gxa/experiments/E-CURD-148)                       | Comparative transcriptome analysis reveals conserved defence responses in maize and sorghum to Exserohilum turcicum - sorghum data |
| Differential | sorghum     | [E-GEOD-140928](https://www.ebi.ac.uk/gxa/experiments/E-GEOD-140928)                 | Genome-wide analysis of abscisic acid biosynthesis, catabolism, and signaling in Sorghum bicolor under saline-alkali stress |


### Pathways

The Plant Reactome was updated in April, 2025 and includes 10 new and 5 updated pathways involved in flowering under drought stress, root and shoot gravitropism, and salinity response for a total of 351 manually curated rice pathways projected via orthology to 138 genomes. For more details see the [release summary](https://plantreactome.gramene.org/index.php?option=com_content&view=article&id=113&Itemid=360&lang=en).

### Gramene Mart

Gramene Mart will no longer be hosting a mirror of [Ensembl Plants BioMart](https://plants.ensembl.org/biomart/martview).

## Infrastructure

**Web Services:** Gramene's [web services page](https://gramene.org/web-services) documents ways to directly connect to and analyze our databases.

- Public MySQL Server: 

Our partner Ensembl Genomes offers a public, read-only MySQL server with copies of the species-specific and comparative genomic databases that we use. To use this with the mysql command-line client:

  $ mysql -hmysql-eg-publicsql.ebi.ac.uk -P4157 -uanonymous

Please note that the versioning scheme used at this public database differs from ours; Gramene's release 69_114 is set as version 61_114 at Ensembl.

## Funding

The Gramene project is currently funded by the United States Department of Agriculture - Agricultural Research Service, USDA ARS grant #8062-21000-041-00D. 
