## Gramene Release 65
### Released: May 2022
## Summary

## Release Information
- [Overall Highlights](#overall-highlights)
- [Databases](#databases)
- [References](#references)

## Overall Highlights {#overall-highlights}

**CLIMtools portal**

- The new [CLIMtools](https://gramene.org/CLIMtools) portal brings interactive web-based views of environment x genome associations,
  RiboSNitch prediction, and correlations between the local environment and a pool of curated phenotypes.

**Updated software**

- Improved look and feel of the [homepage](https://gramene.org) and gene search interface.
  - Links to our crop specific pangenome sites for [maize](https://maize-pangenome.gramene.org/), [rice](https://oryza.gramene.org/), [grapevine](https://vitis.gramene.org/) and [sorghum](https://sorghumbase.org).
  - Integrated [@GrameneDatabase](https://twitter.com/GrameneDatabase) twitter feed.
  - The React web application has been rewritten to use [Redux-Bundler](https://reduxbundler.com/) for state management.
- Genome browser and underlying database schema and API updated to [Ensembl 105](https://www.ensembl.info/2021/12/10/ensembl-105-has-been-released/).

**New genomes**

- [_Brassica rapa_ R-o-18](http://ensembl.gramene.org/Brassica_rapa_ro18) replaces cultivar Chiifu-401-42.
- [_Corylus avellana_](http://ensembl.gramene.org/Corylus_avellana) (hazelnut).
- [_Ficus carica_](http://ensembl.gramene.org/Ficus_carica) (common fig).
- [_Lactuca sativa_](http://ensembl.gramene.org/Lactuca_sativa) (lettuce).
- [_Olea europaea_](http://ensembl.gramene.org/Olea_europaea) (European olive tree). Olea europaea sylvestris can still be found [here](http://ensembl.gramene.org/Olea_europaea/Info/Strains?db=core).

**Updated genomes and gene annotation**

- [_Helianthus annuus_](http://ensembl.gramene.org/Helianthus_annuus) (sunflower) assembly updated to HanXRQr2.0-SUNRISE.
- [_Hordeum vulgare_](http://ensembl.gramene.org/Hordeum_vulgare) (barley) assembly updated to Morex V3 with IPK and BaRT gene models. The GoldenPromise and TRITEX varieties can be viewed via the [cultivar view](http://ensembl.gramene.org/Hordeum_vulgare/Info/Strains?db=core).
- [_Chlamydomonas reinhardtii_](http://ensembl.gramene.org/Chlamydomonas_reinhardtii) (green algae) updated with gene name synonyms.
- [_Triticum aestivum_ cultivars](https://ensembl.gramene.org/Triticum_aestivum/Info/Strains) (wheat) updated with de novo genes from the [wheat 10+ genome project](http://www.10wheatgenomes.com/).

**New & updated data**

- [_Triticum aestivum_](https://ensembl.gramene.org/Triticum_aestivum) (wheat) updated with centromere data.
- [_Zea mays_](https://ensembl.gramene.org/Zea_mays) (maize) updated with Plant Reactome cross-references.
- Genetic variants for barley and sunflower mapped to the newer assemblies.
- Masking of repeated sequences in 20 angiosperms (including _Arabidopsis thaliana_, _Brachypodium distachyon_, _Brassica rapa_, _Oryza sativa_, _Vitis vinifera_, and _Z. mays_) using k-mer based Repeat Detector (Red).
- SIFT predictions added to VEP output data for several plant species with variation data.


Check which features are supported on each species [here](http://ensembl.gramene.org/species.html); if you would like to request a new species please email [Gramene’s helpdesk](https://www.gramene.org/feedback).

Note that a maximum of 100 genomes is included in the Compara Gene Family Trees analysis, which will change depending on data available for each release. You can find the list of included genomes in [Ensembl’s Compara github repository](https://github.com/Ensembl/ensembl-compara/blob/release/104/conf/plants/allowed_species.json). Click below for a summary of currently available Compara analyses:

- [Protein gene tree analysis](https://ensembl.gramene.org/info/genome/compara/prot_tree_stats.html)
- [Whole-genome alignments summary](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html)



## Databases {#databases}
### Comparative Genomics

[**Gene Trees**](https://ensembl.gramene.org/info/genome/compara/prot_tree_stats.html)
A total of 124,010 protein-coding gene family trees were constructed using the peptide encoded by
the canonical transcript (i.e., a representative transcript for a given gene) for each
of 2,934,710 individual genes (3,247,550 input proteins) from 89 plant genomes.

[**Whole-Genome Alignments**](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html)
There are 320 pairwise genomic alignments. The majority of species in Gramene were aligned with _Oryza sativa_ Japonica (reference monocot), and _Arabidopsis thaliana_ and _Vitis vinifera_ (reference dicots).

[**Synteny**](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html)
Eighty paired synteny maps are available. 

[**Putative Split Genes**](http://ftp.gramene.org/CURRENT_RELEASE/splitgenes/)
Updated Split Genes: Updated split genes for the current release are available on the FTP site.  These are putative gene annotation artifacts (also known as contiguous gene split models) and are based on the latest Ensembl Compara Gene Tree database. Overall statistics for each species is given in the [README](http://ftp.gramene.org/CURRENT_RELEASE/split_genes/1_ReadMe_SplitGenesBySpecies.txt).


### Protein Annotation, GO, Xref Protein domain information 

These were generated for the new and updated genomes.


### Variation

Genetic variants for barley and sunflower mapped to the newer assemblies. No new genetic/structural variation data since release 54. Genetic variation data is available for 15 plant species in Gramene, including _Arabidopsis thaliana_, brachypodium, sunflower, barley, apple, rice (Japonica, Indica, _O. glaberrima_, and _O. glumaepatula_), tomato, sorghum, bread and durum wheat, graprfruit, and maize.


### Expression

Baseline gene expression data for 23 plant genomes was curated and processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments) and is available via Gramene's Search, Gramene's Genome Browser, and Gramene's pathways interface, the Plant Reactome. Differential expression for most of those species is available directly in the Expression Atlas website. 


### Pathways

320 manually curated Japonica rice pathways and orthology-based pathways for 120 plant species cross-linked with [Gramene’s Plant Reactome](https://plantreactome.gramene.org/) entries.


### Gramene Mart

New Plant Gene and Variation Marts generated to incorporte the new genome data.

## Infrastructure

**Web Services:** Gramene's [web services page](https://gramene.org/web-services) documents ways to directly connect to and analyze our databases.

- Public MySQL Server: 

Our partner Ensembl Genomes offers a public, read-only MySQL server with copies of the species-specific and comparative genomic databases that we use. To use this with the mysql command-line client:

  $ mysql -hmysql-eg-publicsql.ebi.ac.uk -P4157 -uanonymous

Please note that the versioning scheme used at this public database differs from ours; Gramene's release 65_105 is set as version 52_105 at Ensembl.


## Citing Gramene {#references}

Tello-Ruiz MK, Naithani S, Gupta P, Olson A, Wei S, Preece S, Jiao Y, Wang B, Chougule K, Garg P, Elser J, Kumari S, Kumar V, Contreras-Moreira B, Naamati G, George N, Cook J, Bolser D, D’Eustachio P, Stein LD, Gupta A, Xu W, Regala J, Papatheodorou I, Kersey PJ, Flicek P, Taylor C, Jaiswal P, and Ware D. (2021). "Gramene 2021: Harnessing the power of comparative genomics and pathways for plant research." *Nucleic Acids Research* 49(D1): D1452–D1463. PMID: 33170273. DOI: [10.1093/nar/gkaa979](https://doi.org/10.1093/nar/gkaa979).

Yates AD, Allen J, Amode RM, Azov AG, Barba M, Becerra A, et al. "Ensembl Genomes 2022: an expanding genome resource for non-vertebrates." Nucleic Acids Res. 2022;50: D996–D1003. PMID: 34791415. DOI: [10.1093/nar/gkab1007](https://doi.org/10.1093/nar/gkab1007).

Tello-Ruiz MK, Jaiswal P, Ware D. "Gramene: A Resource for Comparative Analysis of Plants Genomes and Pathways." In: Edwards D, editor. *Plant Bioinformatics: Methods and Protocols.* New York, NY: Springer US; 2022. pp. 101–131. PMID: 35037202. DOI: [10.1007/978-1-0716-2067-0_5](https://doi.org/10.1007/978-1-0716-2067-0_5).

Gupta P, Naithani S, Preece J, Kim S, Cheng T, D’Eustachio P, et al. "Plant Reactome and PubChem: The Plant Pathway and (Bio)Chemical Entity Knowledgebases." In: Edwards D, editor. *Plant Bioinformatics: Methods and Protocols.* New York, NY: Springer US; 2022. pp. 511–525. PMID: 35037224. DOI: [10.1007/978-1-0716-2067-0_27](https://doi.org/10.1007/978-1-0716-2067-0_27).

## Funding

The Gramene project is currently funded by the United States Department of Agriculture - Agricultural Research Service, USDA ARS grant #8062-21000-041-00D. 

Gramene was initially supported (2001-2004) by the USDA Initiative for Future Agriculture and Food Systems (IFAFS) (grant #00-52100-9622) and a Cooperative State Research and Education Service (CSREES) agreement through the USDA Agricultural Research Service (grant #1907-21000-030: "Identification of Functional Sequence in Plant Genomes Through Bioinformatic, Genomic, and Genetic Approaches").

The National Science Foundation (NSF) supported this work through NSF Plant Genome Initiative grant #0321685 during the years 2004-2007; from 2007-2012, through NSF Plant Genome Research Resource grant #0703908; and NSF award #0851652 (REU Bioinformatics and Computational Biology Summer Undergraduate Program) in 2009-2012. From 2012-2021, work was supported by NSF Improving Plant Genome Annotation grant award #1127112 (Gramene - Exploring Function through Comparative Genomics and Network Analysis).
