## Gramene Release 67
### Released: August 2023
## Summary

Release #67 features the new [Oryza CLIMtools](https://www.gramene.org/CLIMtools/oryza_v1.0/) portal to bring interactive web-based views of environment x genome associations, RiboSNitch prediction, and correlations between the local environment and a pool of curated phenotypes in rice, in addition to Arabidopsis (see the [CLIMtools portal](https://www.gramene.org/CLIMtools/)).

In addition, we have added homology-based pathway projections for 10 new species, adding up to a total of 130 species with Plant Reactome pathways:

- _Brassica juncea_ (brown mustard)
- _Corymbia citriodora_ (lemon-scented gum)
- _Digitaria exilis_ (white fonio, fonio millet, hungry rice)
- _Echinochloa crus-gallis_ (barnyard grass)
- _Lolium perenne_ (rye grass)
- _Pisum sativum_ (pea)
- _Quercus lobata_ (valley oak)
- _Quercus suber_ (cork oak)
- _Secale cereale_ (rye)
- _Vigna unguiculata_ (cowpea or black-eyed pea)

This release also features two newly curated pathways and one "container" pathway for response to gravitopism stimuli. This adds to a total of 309 reference rice pathways. 

- Top level pathway: Response to stimuli: abiotic stresses and stimuli
- Container pathway: Gravitropism under normal or artificial gravity environments
- Newly curated: Regulation of seed germination and coleoptile growth under submergence and normal gravity environment
- Newly curated: Regulation of lemma joints development and leaf angle by cytokinin

The new pathways have been indexed and are searchable from Gramene's homepage, see for example: [Os06g0670300](https://www.gramene.org/?idList=Os06g0670300) (click on the Pathway tab), a gravitropism gene expressed during the R"regulation of seed germination and coleoptile growth under submergence and normal gravity environment".  

The homepage continues to include links to our crop specific pangenome sites for [maize](https://maize-pangenome.gramene.org/), [rice](https://oryza.gramene.org/), [grapevine](https://vitis.gramene.org/), and [sorghum](https://sorghumbase.org). These pangenome sites provide access to additional genomes from the respective taxonomic clade for comparative analysis
against the primary annotated reference genome representative.


We continue to have an integrated Twitter (X) feed on the homepage. Users and readers are encouraged to follow us @GrameneDatabase to get updates, highlights and news.

Users and readers are encouraged to follow us @GrameneDatabase to get updates, highlights and news, through our integrated Twitter (X) feed on Gramene’s homepage.

The Gramene Knowledgebase is a curated, open-source, integrated data resource for comparative functional genomics in crops and model plant species.

The genome databases are a mirror of EMBL-EBI Ensembl Plants, and built in collaboration with Expression Atlas, and UniProt.

The Plant Reactome database is produced in collaboration with the NIH-funded human Reactome project.

Core funding for the project is provided by the Agricultural Research Service of the U.S. Department of Agriculture (USDA ARS 8062-21000-041-00D).


## Release Information
- [Overall Highlights](#overall-highlights)
- [Databases](#databases)
- [References](#references)

## Overall Highlights {#overall-highlights}

**CLIMtools portal**

- The new [Oryza CLIMtools](https://www.gramene.org/CLIMtools/oryza_v1.0/) portal brings interactive web-based views of environment x genome associations,
  RiboSNitch prediction, and correlations between the local environment and a pool of curated rice phenotypes.
  
**New data & features**

- New Plant Reactome pathway data is now searchable in the Gramene Search Interface, see for example: [EXAMPLE].

**New Reactome Pathways**

- [_Brassica juncea_](https://ensembl.gramene.org/Brassica_juncea) (brown mustard)
- [_Corymbia citriodora_](https://ensembl.gramene.org/Corymbia_citriodora) (lemon-scented gum)
- [_Digitaria exilis_](https://ensembl.gramene.org/Digitaria_exilis) (white fonio, fonio millet, hungry rice)
- [_Echinochloa crus-galli_](https://ensembl.gramene.org/Echinochloa_crusgalli) (barnyard grass)
- [_Lolium perenne_](https://ensembl.gramene.org/Lolium_perenne) (rye grass)
- [_Pisum sativum_](https://ensembl.gramene.org/Pisum_sativum) (pea)
- [_Quercus suber_](https://ensembl.gramene.org/Quercus_suber) (cork oak)
- [_Secale cereale_](https://ensembl.gramene.org/Secale_cereale) (rye)
- [_Vigna unguiculata_](https://ensembl.gramene.org/Vigna_unguiculata) (cowpea or black-eyed pea)



## Databases {#databases}
### Comparative Genomics

[**Gene Trees**](https://ensembl.gramene.org/info/genome/compara/prot_tree_stats.html)
A total of 152,102 protein-coding gene family trees were constructed using the peptide encoded by
the canonical transcript (i.e., a representative transcript for a given gene) for each
of 3,624,840 individual genes (5,140,091 input proteins) from 106 plant genomes.

[**Whole-Genome Alignments**](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html)
There are 278 pairwise genomic alignments. The majority of species in Gramene were aligned with _Oryza sativa_ Japonica (reference monocot), and _Arabidopsis thaliana_ and _Vitis vinifera_ (reference dicots).

[**Synteny**](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html)
Seventy-nine paired synteny maps are available. 

[**Putative Split Genes**](http://ftp.gramene.org/CURRENT_RELEASE/splitgenes/)
Updated split genes for the current release are available on the FTP site.  These are putative gene annotation artifacts (also known as contiguous gene split models) and are based on the latest Ensembl Compara Gene Tree database. Overall statistics for each species is given in the [README](https://ftp.gramene.org/CURRENT_RELEASE/split_genes/1_ReadMe_SplitGenesBySpecies.txt).

[**Plant Reactome**](https://plantreactome.gramene.org)
Plant Reactome is the pathway knowledgebase of Gramene. We utilize the Reactome pathway data model to represent plant metabolic, transport and signaling pathways, developmental processes, organ differentiation, and transcriptional regulatory networks. Manual biocuration is conducted in the reference species rice (_O. sativa_) and pathways are then projected via gene orthology to other species including single-cell photoautotrophs, lower plants, and higher plants. This release is a maintenance release. Plant Reactome continues to host pathway projections for 120 plant species. Additional details and statistics can be found in the Plant Reactome's site [release notes](https://plantreactome.gramene.org/index.php?option=com_content&view=article&id=111&Itemid=360&lang=en).

### Protein Annotation, GO, Xref Protein domain information 

No updates in this release.


### Variation

Genetic variation data continues to be available for 17 plant species, including _Arabidopsis thaliana_, maize,  rice (Japonica, Indica, _O. glaberrima_, and _O. glumaepatula_), brachypodium, sunflower, barley, apple, tomato, sorghum, bread and durum wheat, black bean, lettuce, and cocoa. Variation for the last three species is not yet available via BioMart.

### Expression

Baseline gene expression data for 23 plant genomes was curated and processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments) and is available via Gramene's Search, Gramene's Genome Browser, and Gramene's pathways interface, the Plant Reactome. Differential expression for most of those species is available directly in the Expression Atlas website. 


### Pathways

309 manually curated Japonica rice pathways and orthology-based pathways for 130 plant species cross-linked with [Gramene’s Plant Reactome](https://plantreactome.gramene.org/) entries.


### Gramene Mart

No updates in this release.

## Infrastructure

**Web Services:** Gramene's [web services page](https://gramene.org/web-services) documents ways to directly connect to and analyze our databases.

- Public MySQL Server: 

Our partner Ensembl Genomes offers a public, read-only MySQL server with copies of the species-specific and comparative genomic databases that we use. To use this with the mysql command-line client:

  $ mysql -hmysql-eg-publicsql.ebi.ac.uk -P4157 -uanonymous

Please note that the versioning scheme used at this public database differs from ours; Gramene's release 67_108 is set as version 55_108 at Ensembl.

## References {#references}


Naithani S, Mohanty B, Elser J, D'Eustachio P, Jaiswal P. (2023). "Biocuration of a Transcription Factors Network Involved in Submergence Tolerance during Seed Germination and Coleoptile Elongation in Rice (_Oryza sativa_)".  2023. _Plants _(Basel) 12(11):2146. DOI: [10.3390/plants12112146](https://doi.org/10.3390/plants12112146). 


## Citing Gramene 

Tello-Ruiz MK, Naithani S, Gupta P, Olson A, Wei S, Preece S, Jiao Y, Wang B, Chougule K, Garg P, Elser J, Kumari S, Kumar V, Contreras-Moreira B, Naamati G, George N, Cook J, Bolser D, D’Eustachio P, Stein LD, Gupta A, Xu W, Regala J, Papatheodorou I, Kersey PJ, Flicek P, Taylor C, Jaiswal P, and Ware D. (2021). "Gramene 2021: Harnessing the power of comparative genomics and pathways for plant research." *Nucleic Acids Research* 49(D1): D1452–D1463. PMID: 33170273. DOI: [10.1093/nar/gkaa979](https://doi.org/10.1093/nar/gkaa979).

Yates AD, Allen J, Amode RM, Azov AG, Barba M, Becerra A, et al. "Ensembl Genomes 2022: an expanding genome resource for non-vertebrates." Nucleic Acids Res. 2022;50: D996–D1003. PMID: 34791415. DOI: [10.1093/nar/gkab1007](https://doi.org/10.1093/nar/gkab1007).

Tello-Ruiz MK, Jaiswal P, Ware D. "Gramene: A Resource for Comparative Analysis of Plants Genomes and Pathways." In: Edwards D, editor. *Plant Bioinformatics: Methods and Protocols.* New York, NY: Springer US; 2022. pp. 101–131. PMID: 35037202. DOI: [10.1007/978-1-0716-2067-0_5](https://doi.org/10.1007/978-1-0716-2067-0_5).

Gupta P, Naithani S, Preece J, Kim S, Cheng T, D’Eustachio P, et al. "Plant Reactome and PubChem: The Plant Pathway and (Bio)Chemical Entity Knowledgebases." In: Edwards D, editor. *Plant Bioinformatics: Methods and Protocols.* New York, NY: Springer US; 2022. pp. 511–525. PMID: 35037224. DOI: [10.1007/978-1-0716-2067-0_27](https://doi.org/10.1007/978-1-0716-2067-0_27).

## Funding

The Gramene project is currently funded by the United States Department of Agriculture - Agricultural Research Service, USDA ARS grant #8062-21000-041-00D. 

