## Gramene Release 66
### Released: December 2022
## Summary

Release #66 features 128 reference genomes and 152,102 gene family trees. We are adding 10 new reference genomes including a 
tree (cork oak), two kinds of peas (common pea and cowpea or black-eyed pea), rye, two grasses (rye grass and barnyard
grass), brown mustard, lemon-scented gum, and fonio millet or hungry rice. Our comparative genomics collection currently
includes 320 pairwise DNA alignments and 80 synteny maps. All data is available for bulk download from Gramene’s FTP site.

In this release, we integrated gene functional annotations from NCBI's geneRIFs and rap-db's curated genes. Each of these
curated sources provide a mapping between genes and publications. A total of 12,722 gene models now have associated
publications which describe their function.

The homepage continues to include links to our crop specific pangenome sites for maize, rice, grapevine and sorghum.
These pangenome sites provide access to additional genomes from the respective taxonomic clade for comparative analysis
against the primary annotated reference genome representative.

We have an integrated Twitter feed on the homepage. Users and readers are encouraged to follow us @GrameneDatabase to get updates, highlights and news.

The Plant Reactome pathway knowledgebase continues to host curated rice pathways and gene-orthology based projections for 120 species.

The CLIMtools portal brings interactive web-based views of environment x genome associations, RiboSNitch prediction, and correlations between the local environment and a pool of curated phenotypes in Arabidopsis.

The Gramene Knowledgebase is a curated, open-source, integrated data resource for comparative functional genomics in crops and model plant species.
The genome databases are a mirror of EMBL-EBI Ensembl Plants, and built in collaboration with Expression Atlas, and UniProt.
The Plant Reactome database is produced in collaboration with the NIH-funded human Reactome project.
Core funding for the project is provided by the Agricultural Research Service of the U.S. Department of Agriculture (USDA ARS 8062-21000-041-00D).


## Release Information
- [Overall Highlights](#overall-highlights)
- [Databases](#databases)
- [References](#references)

## Overall Highlights {#overall-highlights}

**New genomes**

- [_Brassica juncea_](https://ensembl.gramene.org/Brassica_juncea) (brown mustard)
- [_Corymbia citriodora_](https://ensembl.gramene.org/Corymbia_citriodora) (lemon-scented gum)
- [_Digitaria exilis)_](https://ensembl.gramene.org/Digitaria_exilis) (white fonio, fonio millet, hungry rice)
- [_Echinochloa crus-galli_](https://ensembl.gramene.org/Echinochloa_crusgalli) (barnyard grass)
- [_Lolium perenne_](https://ensembl.gramene.org/Lolium_perenne) (rye grass)
- [_Pisum sativum_](https://ensembl.gramene.org/Pisum_sativum) (pea)
- [_Quercus suber_](https://ensembl.gramene.org/Quercus_suber) (cork oak)
- [_Secale cereale_](https://ensembl.gramene.org/Secale_cereale) (rye)
- [_Vigna unguiculata_](https://ensembl.gramene.org/Vigna_unguiculata) (cowpea or black-eyed pea)


**New & updated data**

- New variation data for lettuce (_Lactuca sativa_) and cocoa (_Theobroma cacao_) from the European Variation Archive. This data is not yet available via Gramene Mart.
- Updated [split gene predictions](https://ftp.gramene.org/pub/gramene/CURRENT_RELEASE/split_genes/) from [peptide comparative genomics]([peptide comparative genomics](http://ensemblgenomes.org/info/data/peptide_compara).


**Updated software**

- Curated genes
  - Publications tab includes a table of PUBMED linked papers associated with the gene
  - Search for genes curated with Trait Ontology (TO) or Plant Ontology (PO) terms
- Genome browser and underlying database schema and API updated to [Ensembl 108]([https://www.ensembl.info/2021/12/10/ensembl-105-has-been-released/](https://www.ensembl.info/2022/10/20/ensembl-108-has-been-released/)).


Check which features are supported on each species [here](http://ensembl.gramene.org/species.html); if you would like to request a new species please email [Gramene’s helpdesk](https://www.gramene.org/feedback).

The list of genomes included in the Ensembl's Compara Gene Family Trees analysis is available in [Ensembl’s Compara github repository](https://github.com/Ensembl/ensembl-compara/blob/release/108/conf/plants/allowed_species.json). Click below for a summary of currently available Compara analyses:

- [Protein gene tree analysis](https://ensembl.gramene.org/info/genome/compara/prot_tree_stats.html)
- [Whole-genome alignments summary](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html)



## Databases {#databases}
### Comparative Genomics

[**Gene Trees**](https://ensembl.gramene.org/info/genome/compara/prot_tree_stats.html)
A total of 152,102 protein-coding gene family trees were constructed using the peptide encoded by
the canonical transcript (i.e., a representative transcript for a given gene) for each
of 3,624,840 individual genes (5,140,091 input proteins) from 106 plant genomes.

[**Whole-Genome Alignments**](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html)
There are 320 pairwise genomic alignments. The majority of species in Gramene were aligned with _Oryza sativa_ Japonica (reference monocot), and _Arabidopsis thaliana_ and _Vitis vinifera_ (reference dicots).

[**Synteny**](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html)
Eighty paired synteny maps are available. 

[**Putative Split Genes**](http://ftp.gramene.org/CURRENT_RELEASE/splitgenes/)
Updated split genes for the current release are available on the FTP site.  These are putative gene annotation artifacts (also known as contiguous gene split models) and are based on the latest Ensembl Compara Gene Tree database. Overall statistics for each species is given in the [README](https://ftp.gramene.org/CURRENT_RELEASE/split_genes/1_ReadMe_SplitGenesBySpecies.txt).

[**Plant Reactome**](https://plantreactome.gramene.org)
Plant Reactome is the pathway knowledgebase of Gramene. We utilize the Reactome pathway data model to represent plant metabolic, transport and signaling pathways, developmental processes, organ differentiation, and transcriptional regulatory networks. Manual biocuration is conducted in the reference species rice (_O. sativa_) and pathways are then projected via gene orthology to other species including single-cell photoautotrophs, lower plants, and higher plants. This release is a maintenance release. Plant Reactome continues to host pathway projections for 120 plant species. Additional details and statistics can be found in the Plant Reactome's site [release notes](https://plantreactome.gramene.org/index.php?option=com_content&view=article&id=111&Itemid=360&lang=en).

### Protein Annotation, GO, Xref Protein domain information 

These were generated for the new and updated genomes.


### Variation

New genetic variation data for _Lactuca sativa_ (lettuce) and _Theobroma cacao_ (cocoa). Genetic variation data continues to be available for 16 additional plant species, including _Arabidopsis thaliana_, brachypodium, sunflower, barley, apple, rice (Japonica, Indica, _O. glaberrima_, and _O. glumaepatula_), tomato, sorghum, bread and durum wheat, grapefruit, black bean, and maize.


### Expression

Baseline gene expression data for 23 plant genomes was curated and processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments) and is available via Gramene's Search, Gramene's Genome Browser, and Gramene's pathways interface, the Plant Reactome. Differential expression for most of those species is available directly in the Expression Atlas website. 


### Pathways

320 manually curated Japonica rice pathways and orthology-based pathways for 120 plant species cross-linked with [Gramene’s Plant Reactome](https://plantreactome.gramene.org/) entries.


### Gramene Mart

New Plant Gene Mart generated to incorporte the new genome data.

## Infrastructure

**Web Services:** Gramene's [web services page](https://gramene.org/web-services) documents ways to directly connect to and analyze our databases.

- Public MySQL Server: 

Our partner Ensembl Genomes offers a public, read-only MySQL server with copies of the species-specific and comparative genomic databases that we use. To use this with the mysql command-line client:

  $ mysql -hmysql-eg-publicsql.ebi.ac.uk -P4157 -uanonymous

Please note that the versioning scheme used at this public database differs from ours; Gramene's release 66_108 is set as version 55_108 at Ensembl.


## Citing Gramene {#references}

Tello-Ruiz MK, Naithani S, Gupta P, Olson A, Wei S, Preece S, Jiao Y, Wang B, Chougule K, Garg P, Elser J, Kumari S, Kumar V, Contreras-Moreira B, Naamati G, George N, Cook J, Bolser D, D’Eustachio P, Stein LD, Gupta A, Xu W, Regala J, Papatheodorou I, Kersey PJ, Flicek P, Taylor C, Jaiswal P, and Ware D. (2021). "Gramene 2021: Harnessing the power of comparative genomics and pathways for plant research." *Nucleic Acids Research* 49(D1): D1452–D1463. PMID: 33170273. DOI: [10.1093/nar/gkaa979](https://doi.org/10.1093/nar/gkaa979).

Yates AD, Allen J, Amode RM, Azov AG, Barba M, Becerra A, et al. "Ensembl Genomes 2022: an expanding genome resource for non-vertebrates." Nucleic Acids Res. 2022;50: D996–D1003. PMID: 34791415. DOI: [10.1093/nar/gkab1007](https://doi.org/10.1093/nar/gkab1007).

Tello-Ruiz MK, Jaiswal P, Ware D. "Gramene: A Resource for Comparative Analysis of Plants Genomes and Pathways." In: Edwards D, editor. *Plant Bioinformatics: Methods and Protocols.* New York, NY: Springer US; 2022. pp. 101–131. PMID: 35037202. DOI: [10.1007/978-1-0716-2067-0_5](https://doi.org/10.1007/978-1-0716-2067-0_5).

Gupta P, Naithani S, Preece J, Kim S, Cheng T, D’Eustachio P, et al. "Plant Reactome and PubChem: The Plant Pathway and (Bio)Chemical Entity Knowledgebases." In: Edwards D, editor. *Plant Bioinformatics: Methods and Protocols.* New York, NY: Springer US; 2022. pp. 511–525. PMID: 35037224. DOI: [10.1007/978-1-0716-2067-0_27](https://doi.org/10.1007/978-1-0716-2067-0_27).

## Funding

The Gramene project is currently funded by the United States Department of Agriculture - Agricultural Research Service, USDA ARS grant #8062-21000-041-00D. 

