## References {#references}

Addo-Quaye C, Tuinstra M, Carraro N, Weil C, Dilkes BP. 2018. "Whole-Genome Sequence Accuracy Is Improved by Replication in a Population of Mutagenized Sorghum." *G3* 8: 1079–1094. PMID: 29378822. doi: [10.1534/g3.117.300301](https://doi.org/10.1534/g3.117.300301).

Aken BL, Ayling S, Barrell D, Clarke L, Curwen V, Fairley S, Fernandez Banet J, et al. 2016. "The Ensembl Gene Annotation System." *Database: The Journal of Biological Databases and Curation* 2016: baw093. PMID: 27337980. doi: [10.1093/database/baw093](https://doi.org/10.1093/database/baw093).

Boatwright JL, Sapkota S, Jin H, Schnable JC, Brenton Z, Boyles R, Kresovich S. 2022. "Sorghum Association Panel whole-genome sequencing establishes cornerstone resource for dissecting genomic diversity". *Plant J* 111: 888–904. PMID: 35653240. doi: [10.1111/tpj.15853](https://doi.org/10.1111/tpj.15853).

Brenton ZW, Cooper EA, Myers MT, Boyles RE, Shakoor N, Zielinski KJ, Rauh BL, Bridges WC, Morris GP, and Kresovich S. 2016. "A Genomic Resource for the Development, Improvement, and Exploitation of Sorghum for Bioenergy." *Genetics* 204 (1): 21–33. PMID: 27356613. doi: [10.1534/genetics.115.183947](https://doi.org/10.1534/genetics.115.183947).

Casa AM, Pressoir G, Brown PJ, Mitchell SE, Rooney WL, Tuinstra MR, Franks CD, and Kresovich S. 2008. "Community Resources and Strategies for Association Mapping in Sorghum." *Crop Science* 48 (1): 30–40. [10.2135/cropsci2007.02.0080](https://doi.org/10.2135/cropsci2007.02.0080).

Davidson RM, Gowda M, Moghe G, Lin H, Vaillancourt B, Shiu S-H, Jiang N, and Buell CR. 2012. "Comparative Transcriptomics of Three Poaceae Species Reveals Patterns of Gene Expression Evolution." *The Plant Journal: For Cell and Molecular Biology* 71 (3): 492–502. PMID: 22443345. [10.1111/j.1365-313X.2012.05005.x](https://doi.org/10.1111/j.1365-313X.2012.05005.x).

Emms DM, Covshoff S, Hibberd JM, and Kelly S. 2016. "Independent and Parallel Evolution of New Genes by Gene Duplication in Two Origins of C4 Photosynthesis Provides New Insight into the Mechanism of Phloem Loading in C4 Species." *Molecular Biology and Evolution* 33 (7): 1796–1806. PMID: 27016024. doi: [10.1093/molbev/msw057](https://doi.org/10.1093/molbev/msw057).

Gladman N, Hufnagel B, Regulski M, Liu Z, Wang X, Chougule K, Kochian L, Magalhães J, Ware D. 2022. "Sorghum root epigenetic landscape during limiting phosphorus conditions." *Plant Direct*. 6(5):e393. PMID: 35600998. doi: [10.1002/pld3.393](https://doi.org/10.1002/pld3.393).

Goodstein DM, Shu S, Howson R, Neupane R, Hayes RD, Fazo J, Mitros T, et al. 2012. "Phytozome: A Comparative Platform for Green Plant Genomics." *Nucleic Acids Research* 40 (Database issue): D1178–86. PMID: 22110026. doi: [10.1093/nar/gkr944](https://doi.org/10.1093/nar/gkr944).

Jiao Y, Burke JJ, Chopra R, Burow G, Chen J, Wang B, Hayes C, Emendack Y, Ware D, and Xin Z. 2016. "A Sorghum Mutant Resource as an Efficient Platform for Gene Discovery in Grasses." *The Plant Cell*. PMID: 27354556. doi: [10.1105/tpc.16.00373](https://doi.org/10.1105/tpc.16.00373).

Jiao Y, Nigam D, Barry K, Daum C, Yoshinaga Y, Lipzen A, Khan A, Parasa SP, Wei S, Lu Z, Tello-Ruiz MK, Dhiman P, Burow G, Hayes C, Chen J, Brandizzi F, Mortimer J, Ware D, Xin Z. 2023. "A large sequenced mutant library – valuable reverse genetic resource that covers 98% of sorghum genes." The Plant Journal 117(5):1543-1557. doi: [10.1111/tpj.16582](https://doi.org/10.1111/tpj.16582).

Lozano R, Gazave E, Dos Santos JPR, Stetter MG, Valluru R, Bandillo N, et al. 2021. "Comparative evolutionary genetics of deleterious load in sorghum and maize." *Nat Plants* 7:17–24. PMID: 33452486. doi: [10.1038/s41477-020-00834-5](https://doi.org/10.1038/s41477-020-00834-5).

McCormick RF, Truong SK, Sreedasyam A, Jenkins J, Shu S, Sims D, Kennedy M, et al. 2018. "The Sorghum Bicolor Reference Genome: Improved Assembly, Gene Annotations, a Transcriptome Atlas, and Signatures of Genome Organization." *The Plant Journal: For Cell and Molecular Biology* 93 (2): 338–54. PMID: 29161754. doi: [10.1111/tpj.13781](https://doi.org/10.1111/tpj.13781).

Mace ES, Tai S, Gilding EK, Li Y, Prentis PJ, Bian L, Campbell BC, et al. 2013. "Whole-Genome Sequencing Reveals Untapped Genetic Potential in Africa’s Indigenous Cereal Crop Sorghum." *Nature Communications* 4: 2320. PMID: 23982223. doi: [10.1038/ncomms3320](http://doi.org/10.1038/ncomms3320).

Makita Y, Shimada S, Kawashima M, Kondou-Kuriyama T, Toyoda T, and Matsui M. 2015. "MOROKOSHI: Transcriptome Database in Sorghum Bicolor."
*Plant & Cell Physiology* 56 (1): e6. PMID: 25505007. doi: [10.1093/pcp/pcu187](https://doi.org/10.1093/pcp/pcu187).

Morris GP, Ramu P, Deshpande SP, Hash CT, Shah T, Upadhyaya HD, Riera-Lizarazu O, et al. 2013. "Population Genomic and Genome-Wide Association Studies of Agroclimatic Traits in Sorghum." *Proceedings of the National Academy of Sciences of the United States of America* 110 (2): 453–58. PMID: 23267105. [10.1073/pnas.1215985110](https://doi.org/10.1073/pnas.1215985110).

Mural RV, Grzybowski M, Miao C, Damke A, Sapkota S, Boyles RE, Fernandez MGS, Schnable PS, Sigmon B, Kresovich S, Schnable JC. 2021. "Meta-analysis identifies pleiotropic loci controlling phenotypic trade-offs in sorghum." Genetics 218:3. doi: [10.1093/genetics/iyab087](https://doi.org/10.1093/genetics/iyab087).

Olson A, Klein RR, Dugas DV, Lu Z, Regulski M, Klein PE, and Ware D. 2014. "Expanding and Vetting Sorghum Bicolor Gene Annotations through Transcriptome and Methylome Sequencing." *The Plant Genome* 7 (2): plantgenome2013.08.0025. doi: [10.3835/plantgenome2013.08.0025](https://doi.org/10.3835/plantgenome2013.08.0025).

Paterson AH, Bowers JE, Bruggmann R, Dubchak I, Grimwood J, Gundlach H, Haberer G, et al. 2009. "The Sorghum Bicolor Genome and the Diversification of Grasses."
*Nature* 457 (7229): 551–56. PMID: 19189423. doi: [10.1038/nature07723](https://doi.org/10.1038/nature07723).

Turco GM, Kajala K, Kunde-Ramamoorthy G, Ngan C-Y, Olson A, Deshphande S, Tolkunov D, et al. 2017. "DNA Methylation and Gene Expression Regulation Associated with Vascularization in Sorghum Bicolor." *The New Phytologist* 214 (3): 1213–29. PMID: 28186631. doi: [10.1111/nph.14448](https://doi.org/10.1111/nph.14448).

Xin Z, Wang ML, Barkley NA, Burow G, Franks C, Pederson G, and Burke J. 2008. "Applying Genotyping (TILLING) and Phenotyping Analyses to Elucidate Gene Function in a Chemically Induced Sorghum Mutant Population." *BMC Plant Biology*. PMID: 18854043. doi: [10.1186/1471-2229-8-103](https://doi.org/10.1186/1471-2229-8-103).

Wang B, Regulski M, Tseng E, Olson A, Goodwin S, McCombie WR, and Ware D. 2018. "A Comparative Transcriptional Landscape of Maize and Sorghum Obtained by Single-Molecule Sequencing." *Genome Research* 28 (6): 921–32. PMID: 29712755. doi: [10.1101/gr.227462.117](https://doi.org/10.1101/gr.227462.117).

Zheng L-Y, Guo X-S, He B, Sun L-J, Peng Y, Dong S-S, Liu T-F, et al. 2011. "Genome-Wide Patterns of Genetic Variation in Sweet and Grain Sorghum (Sorghum Bicolor)."
*Genome Biology* 12 (11): R114. PMID: 22104744. doi: [10.1186/gb-2011-12-11-r114](http://dx.doi.org/10.1186/gb-2011-12-11-r114).

Zhou Y, Kathiresan N, Yu Z, Rivera LF, Yang Y, Thimma M, Manickam K, Chebotarov D, Mauleon R, Chougule K, Wei S, Gao T, Green CD, Zuccolo A, Xie W, Ware D, Zhang J, McNally KL, Wing RA. 2024. "A high-performance computational workflow to accelerate GATK SNP detection across a 25-genome dataset." BMC biology. doi: [10.1186/s12915-024-01820-5](https://doi.org/10.1186/s12915-024-01820-5).

