## Oryza PanGenome Release 5
### Released: October 2022
## Summary

Gramene's Oryza Pan-Genome (https://oryza.gramene.org) is a web portal for comparative plant genomics focused on rice varieties. In its fifth release, we are providing synteny maps between rice varieties and across closely related crops, whole-genome alignments between the rice reference assembly, sorghum and maize, new links to the Oryza Search page from the Ensembl Gene pages, and updated display names for the rice genomes. 

## Release Information
- [Overall Highlights](#overall-highlights)
- [Databases](#databases)
- [Statistics](#statistics)
- [References](#references)
- [Funding](#funding)

## Overall Highlights

New to this release are 27 new synteny maps between the IRGSP1 Nipponbare assembly and all rice accessions, as well as new cross-species synteny maps between the Nipponbare assembly and Leersia, sorghum and maize. In addition, whole-genome alignments between Nipponbare, sorghum and maize are being provided. Usability of the site has been improved through direct links from the Ensembl Browser Gene pages to the Oryza Search pages. Nomenclature for naming of reference sequence assemblies was updated based on standard recommendations from the NSF-DBI (#2029854): CIBR-BBSRC: PanOryza: Globally coordinated genomes, proteomes and pathways for rice.


## Databases 
### Comparative Genomics

[**Whole-Genome Alignments**](https://oryza-ensembl.gramene.org/compara_analyses.html). New pairwise alignments between the Nipponbare, and the sorghum and maize genomes. See [example](https://oryza-ensembl.gramene.org/Oryza_sativa/Share/79cd0d59180d32b16913170be0f0ca3b?redirect=no;mobileredirect=no).

[**Synteny**](https://oryza-ensembl.gramene.org/compara_analyses.html). Twenty-seven synteny maps between the IRGSP1 Nipponbare assembly and all rice accessions (for example, see [Nipponbare vs KitaakeX](https://oryza-ensembl.gramene.org/Oryza_sativa/Location/Synteny?db=core;otherspecies=Oryza_sativakitaake;r=1:8950777-9114776), and three new synteny maps between the Nipponbare and each of Leersia, sorghum and maize. See the same region of [Nipponbare compared to sorghum](https://oryza-ensembl.gramene.org/Oryza_sativa/Location/Synteny?db=core&r=1%3A8950777-9114776&otherspecies=Sorghum_bicolor). 

[**Gene Trees**](https://oryza-ensembl.gramene.org/prot_tree_stats.html). No updates in the curent release, please see [release notes for version 4.0](https://oryza.gramene.org/news).

### Variation

No updates in the current release, please see [release notes for version 3.0](https://oryza.gramene.org/news). 

[Possibly] Genetic variation for _O. sativa_ Japonica Nipponbare IRGSP1 was imported from the Euorpean Variation Archive (EVA). It consists of the following data sets:

1) Duitama _et al_ (2015): 25.8 million SNPs
2) BGI (2004): 3 million SNPs
3) OMAP (2007): 1.6 million SNPs
4) [3K rice genome project](https://doi.org/10.1186/2047-217x-3-7): 366K SNPs
5) McNally _et al_ (2009): 157K SNPs
6) Zhao et al (2010): 1.3K SNPs

### Expression

No updates in the current release, please see [release notes for version 3.0](https://oryza.gramene.org/news).

### Pathways

No updates in the current release, please see [release notes for version 3.0](https://oryza.gramene.org/news).

## References

- 3,000 rice genomes project. The 3,000 rice genomes project. Gigascience. 2014;3: 7. doi:10.1186/2047-217X-3-7

- Alsantely A, Gutaker R, Navarrete Rodríguez ME, Arrieta-Espinoza G, Fuchs EJ, Costa de Oliveira A, et al. The International Oryza Map Alignment Project (I-OMAP): the Americas-past achievements and future directions. J Exp Bot. 2023;74: 1331–1342. doi:10.1093/jxb/erac490

- Duitama J, Silva A, Sanabria Y, Cruz DF, Quintero C, Ballen C, et al. Whole genome sequencing of elite rice cultivars as a comprehensive information resource for marker assisted selection. PLoS One. 2015;10: e0124617. doi:10.1371/journal.pone.0124617

- McNally KL, Childs KL, Bohnert R, Davidson RM, Zhao K, Ulat VJ, et al. Genomewide SNP variation reveals relationships among landraces and modern varieties of rice. Proc Natl Acad Sci U S A. 2009;106: 12273–12278. doi:10.1073/pnas.0900992106

- Yu J, Wang J, Lin W, Li S, Li H, Zhou J, et al. The Genomes of Oryza sativa: a history of duplications. PLoS Biol. 2005;3: e38. doi:10.1371/journal.pbio.0030038

- Zhao K, Wright M, Kimball J, Eizenga G, McClung A, Kovach M, et al. Genomic diversity and introgression in O. sativa reveal the impact of domestication and breeding on the rice genome. PLoS One. 2010;5: e10780. doi:10.1371/journal.pone.0010780

- Zhou Y, Chebotarov D, Kudrna D, Llaca V, Lee S, Rajasekar S, et al. "A platinum standard pan-genome resource that represents the population structure of Asian rice." Sci Data. 2020;7: 113. doi: [10.1038/s41597-020-0438-2](https://doi.org/10.1038/s41597-020-0438-2).
  
- Zhou _et al_ (2022). Pan-genome inversion index reveals evolutionary insights into the subpopulation structure of Asian rice (Oryza sativa). bioRxiv. doi: [10.1101/2022.06.11.49568](https://doi.org/10.1101/2022.06.11.495682).

## Funding

Core funding for the project was provided by the National Science Foundation (NSF IOS-1127112) and the Agricultural Research Service of the U.S. Department of Agriculture (USDA ARS 8062-21000-041-00D).  

