## Maize PanGenome Release 4.0
### Released: February 2024
## Summary
[Gramene's Maize PanGenome](http://maize-pangenome.gramene.org), a web portal for comparative plant genomics focused on Maize crop varieties,
has now released a new version featuring the genome sequence and gene annotation of the teosinte inbred line **Til11**, also featured in the [MaizeCODE project](http://www.maizecode.org/data/). The site now hosts a total of 52 genomes including three assembly versions of the B73 _Zea mays_ reference (RefGen_v3, RefGen_v4, and RefGen_v5), 25 NAM maize founder genomes, 4 European flint genomes, 4 additional maize genomes (Mo17, W22, PH207, and A188), and seven outgroups (_Arabidopsis thaliana_, _Oryza sativa_ Japonica, _Sorghum bicolor_, _Vitis vinifera_ ssp. vinifera PN40024, _Selaginella moellendorffii_, _Chlamydomonas reinhardtii_, and _Drosophila melanogaster_).

The maize gene models, together with the seven outgroups were used to build 35,360 protein-coding gene family trees. In addition, 143 pairwise synteny maps were build with version 5 of the _Z. mays_ B73 genome, Arabidopsis, rice, and sorghum. Data from 32 baseline gene expression studies from the EBI Expression Atlas, and 270 orthology-based pathway projections from the Plant Reactome are provided for B73 (v5).

The outgroup genome databases were built in direct collaboration with the [Ensembl Plants](http://plants.ensembl.org) project.
Other data sets were facilitated via collaborations with the [Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments),
and the [Plant Reactome](https://plantreactome.gramene.org/) databases.

## Release Information
- [Overall Highlights](#overall-highlights)
- [Comparative Genomics](#comparative-genomics)
- [Variation](#variation)
- [Gene Expression](#gene-expression)
- [Pathways](#pathways)
- [Funding](#funding)

## Overall Highlights
- One new maize reference genome: Til11
- 35,360 protein-coding gene family trees from 1,674,759 protein sequences
- 143 pairwise synteny maps built to compare all maize genomes with _Z. mays_ B73 v5, Arabdopsis, rice, and sorghum
- Additional gene expression and orthology-based pathway projections for the reference 
  genome *Zea mays* B73 v4 and v5
- MaizeCODE tracks: RNA (RNA-seq, RAMPAGE and shRNA-seq) and chIP-seq (H3K27ac, H3Kme3, H3K4me1) for the following four genomes: Til11, B73, NC350 and W22 in tissues including immature ears, endosperm, root tips, coleoptilar tips and pollen
- MaizeCODE tracks with TF chIP-seq data (GT1, HDZIV6, TU1A) for B73 in immature ears
- NAM RNAseq tracks across all 26 maize genomes
- Sequences tab providing quick access to gene, transcript and protein sequences for a gene model

## Comparative Genomics

[**Gene Trees.**](http://maize-pangenome-ensembl.gramene.org/prot_tree_stats.html) A total of
35,360 protein-coding gene family trees were constructed using the peptide encoded by
the canonical transcript (i.e., a representative transcript for a given gene) for each
of 1,628,071 individual genes (1,674,759 input proteins) from 42 plant genomes.

[**Synteny**](https://maize-pangenome-ensembl.gramene.org/compara_analyses.html). 143 pairwise synteny maps between all maize genomes, and _Z. mays_ B73 v5, Arabdopsis, rice or sorghum.

## Variation

The following variation datasets were lifted from assembly B73_RefGen_v4 to Zm-B73-REFERENCE-NAM-5.0

**HapMap2 dataset**

A variation set which comprises the maize HapMap2 data. This dataset incorporates approximately 55 million SNPs and InDels identified in a collection of 103 pre-domesticated and domesticated Zea mays varieties, including a representative from the sister genus, Tripsacum dactyloides (Eastern gamagrass). Each line was sequenced to an average of 4.5-fold coverage using the Illumina GAIIx platform. The reads can be accessed from the SRA, with accession ID: SRA051245. Reads were initially mapped to the B73 RefGen_v3 reference genome using a combination of Bowtie, Novoalign and SOAP, then remapped to the most recent B73 RefGen_v4 reference genome. The variations were scored by taking into account identity-by-descent blocks that are shared among the lines.

**The Panzea 2.7 genotyped-by-sequencing (GBS) dataset**

This variation data set consists of 719,472 SNPs (excluding 332 SNPs that were removed for mapping to scaffolds) typed in 16,718 maize and teosinte lines, and grouped in 14 overlapping populations according to the germplasm set in the corresponding metadata table.

## Gene Expression

Gene expression data for the Zea mays B73 genome reference was curated and
processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments).
The set consists of 32 studies with baseline expression and 57 with differential expression. Baseline experiments
are available through our Gene Search (Expression panel) and Genome Browser; [differential experiments](https://www.ebi.ac.uk/gxa/experiments?kingdom=Plants&species=Zea+mays&experimentType=Differential) are available at the Expression Atlas.

## Pathways

In this fourth release, we also feature 270 in v5 (271 in v4) orthology-based maize pathways associated with 1,769 genes in v5 (1,817 in v4) in the [Plant Reactome](https://plantreactome.gramene.org/), the pathway knowledgebase. We utilize the Reactome pathway data model to represent plant metabolic, transport and signaling pathways, developmental processes, organ differentiation, and transcriptional regulatory networks. Manual biocuration is conducted in the reference _Oryza sativa_ Japonica species, and pathways are then projected via gene orthology to the B73 _Zea mays_ (pior reference), and other crops and model plants, lower plants and single-cell photoautotrophs.


## Funding

Core funding for the project is provided by the Agricultural Research Service of the U.S. Department of Agriculture (USDA ARS 8062-21000-041-00D). The MaizeCODE project was funded via a National Science Foundation award #1445025.
