## Maize PanGenome Release 5
### Released: February 2025
## Summary
[Gramene's Maize PanGenome](http://maize-pangenome.gramene.org), a web portal for comparative plant genomics focused on Maize crop varieties,
has now released its fifth version featuring the genome sequence and gene annotation of 18 maize genomes (10 CAAS genomes ([Wang et al, 2023](https://doi.org/10.1038/s41588-022-01283-w)) and 8 [PanAnd](https://mcstitzer.github.io/panand_assemblies/) genomes), and a genetic variation data set with over 228 million SNPs generated by MaizeGDB from 1,525 public resequenced lines including inbred lines, landraces, and teosintes. In total, the site now hosts 63 genomes with 56 of them being maize. In addition to the 10 CAAS and 8 PanAnd genomes, the the site hosts three assembly versions of the B73 _Zea mays_ reference (RefGen_v3, RefGen_v4, and RefGen_v5), 25 [NAM maize founder genomes](https://nam-genomes.github.io/), 4 European flint genomes (DK105, EP1, F7 and PE0075), and 4 additional maize genomes (Mo17, W22, PH207, and A188), plus 7 outgroups (_Arabidopsis thaliana_, _Oryza sativa_ Japonica Nipponbare, _Sorghum bicolor_ BTx623, _Vitis vinifera_ ssp. vinifera PN40024, _Selaginella moellendorffii_, _Chlamydomonas reinhardtii_, and _Drosophila melanogaster_). The maize gene models, together with the seven outgroups were used to build XX protein-coding gene family trees. In addition, pangene sets were imported from MaizeGDB. New to this release are conserved sequence (CNS) data from the [PanAnd project](https://mcstitzer.github.io/panand_assemblies/). 

Data from the EBI Expression Atlas amounts to 33 baseline gene expression studies on the B73 (v5) referenc, including RNA-seq samples for 10 tissues from each of the 25 maize NAM genomes. This data together withn baseline expression of paralogs for a reference study per gene, and eFP expression visualization from 12 studies, are available through the Expresion tab of the Gene Search results. Differential experiments are available at the Expression Atlas site. The Pathways tab continues to display 270 orthology-based pathway projections from the Plant Reactome, and the Papers tab is periodically updated with curated scientific literature associated with a gene function from GeneRIF, as well as gene name and symbols submitted by members of the Maize community through the [Submit a gene function form](https://docs.google.com/forms/d/e/1FAIpQLSey-xPyTysdd9c2phXT6kcbjEDeCppG4dLG7LjZCeLpx_KGog/viewform). 

The outgroup genome databases were built in direct collaboration with the [Ensembl Plants](http://plants.ensembl.org) project. Other data sets were facilitated via collaborations with the [Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments), the [Plant Reactome](https://plantreactome.gramene.org/), and the [Bio-Analytic Resource for Plant Biology or BAR](https://bar.utoronto.ca/) databases.

## Release Information
- [Overall Highlights](#overall-highlights)
- [Comparative Genomics](#comparative-genomics)
- [Variation](#variation)
- [Gene Expression](#gene-expression)
- [Pathways](#pathways)
- [Publications](#publications)
- [Funding](#funding)

## Overall Highlights
**New genomes**
- 18 new maize reference genomes:
  - 

**New data**
- XX protein-coding gene family trees from XX protein sequences
- Pangene sets from [MaizeGDB](https://www.maizegdb.org/)
- Conserved sequence (CNS) data from the [PanAnd project](https://mcstitzer.github.io/panand_assemblies/)
- Gene expression profiles for [25 NAM maize genomes via the Expression Atlas](https://www.ebi.ac.uk/gxa/experiments/E-MTAB-8633/Experiment%20Design)

**New features**

- Papers tab lists publications describing gene function and has now a user entry form to suggest gene function given experimental evidence. Search for [357 curated maize genes](https://maize-pangenome.gramene.org/?filters={%22status%22:%22init%22,%22operation%22:%22AND%22,%22negate%22:false,%22marked%22:false,%22leftIdx%22:0,%22rightIdx%22:3,%22children%22:[{%22fq_field%22:%22capabilities%22,%22fq_value%22:%22pubs%22,%22name%22:%22publication%22,%22category%22:%22Curated%22,%22leftIdx%22:1,%22rightIdx%22:2,%22negate%22:false,%22showMenu%22:false,%22marked%22:true}],%22showMarked%22:false,%22showMenu%22:false,%22moveCopyMode%22:%22%22,%22searchOffset%22:0,%22rows%22:20}&genomes=4577)

## Comparative Genomics

[**Gene Trees.**](http://maize-pangenome-ensembl.gramene.org/prot_tree_stats.html) A total of
XX protein-coding gene family trees were constructed using the peptide encoded by
the canonical transcript (i.e., a representative transcript for a given gene) for each
of XX individual genes (XX input proteins) from 4XX plant genomes.

[**Synteny**](https://maize-pangenome-ensembl.gramene.org/compara_analyses.html). 143 pairwise synteny maps between all maize genomes, and _Z. mays_ B73 v5, Arabdopsis, rice or sorghum.

[**Whole-Genome Alignments**](https://maize-pangenome-ensembl.gramene.org/info/genome/compara/compara_analyses.html). Pairwise genomic alignments for B73 v5 with sorghum, Japonica rice, wheat, and Brachypodium are available in the Gramene site and will be made available here across the maize genomes in future releases.

## Variation

The following variation datasets are mapped to the _Z. mays_ B73 v5 reference assembly:




**HapMap2 dataset**

A variation set which comprises the maize HapMap2 data. This dataset incorporates approximately 55 million SNPs and InDels identified in a collection of 103 pre-domesticated and domesticated Zea mays varieties, including a representative from the sister genus, Tripsacum dactyloides (Eastern gamagrass). Each line was sequenced to an average of 4.5-fold coverage using the Illumina GAIIx platform. The reads can be accessed from the SRA, with accession ID: SRA051245. Reads were initially mapped to the B73 RefGen_v3 reference genome using a combination of Bowtie, Novoalign and SOAP, then remapped to the most recent B73 RefGen_v4 and subsequently RefGen_v5 reference assemblies. The variations were scored by taking into account identity-by-descent blocks that are shared among the lines.

**The Panzea 2.7 genotyped-by-sequencing (GBS) dataset**

This variation data set consists of 719,472 SNPs (excluding 332 SNPs that were removed for mapping to scaffolds) typed in 16,718 maize and teosinte lines, and grouped in 14 overlapping populations according to the germplasm set in the corresponding metadata table.

*Structural Variation*

Structural variants are available as data tracks for all 26 NAM genomes on the B73 v5 genome browser, see for example [this region](https://maize-pangenome-ensembl.gramene.org/Zea_maysb73/Location/View?r=1:109000-145001): Note: If you don't see the data, click on the configure button (spider gear icon) and select the genome(s) that you want to see.


## Gene Expression

Gene expression data for the Zea mays B73 genome reference was curated and
processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments).
The current set consists of 33 studies with baseline expression and 57 with differential expression. Baseline experiments as well as expression of paralogs for the reference study and electronic Fluorescent Pictograp (eFP) expression visualizations are available through our Gene Search (Expression panel) and Genome Browser; [differential experiments](https://www.ebi.ac.uk/gxa/experiments?kingdom=Plants&species=Zea+mays&experimentType=Differential) are available at the Expression Atlas.

Additional expression data is also available as data tracks on the corresponding maize genome browsers fron the NAM and MaizeENCODE projects, see for examples [this region](https://maize-pangenome-ensembl.gramene.org/Zea_maysti11/Location/View?r=1:109000-145001) of the Til11 genome (note that you only see MaizeCODE tracks because Til11 is not part of the NAM project). For gene expressiona data from both, NAM and MaizeCODE projects, check out [this region of the B73 v5 genome](https://maize-pangenome-ensembl.gramene.org/Zea_maysb73/Location/View?r=1:109000-145001).


## Pathways

In this fourth release, we also feature 270 in v5 (271 in v4) orthology-based maize pathways associated with 1,769 genes in v5 (1,817 in v4) in the [Plant Reactome](https://plantreactome.gramene.org/), the pathway knowledgebase. We utilize the Reactome pathway data model to represent plant metabolic, transport and signaling pathways, developmental processes, organ differentiation, and transcriptional regulatory networks. Manual biocuration is conducted in the reference _Oryza sativa_ Japonica species, and pathways are then projected via gene orthology to the B73 _Zea mays_ (pior reference), and other crops and model plants, lower plants and single-cell photoautotrophs.


## Funding

Core funding for the project is provided by the Agricultural Research Service of the U.S. Department of Agriculture (USDA ARS 8062-21000-051-00D). The MaizeCODE project was funded via a National Science Foundation award #1445025.

