## Maize PanGenome Release 5
### Released: February 2025
## Summary
[Gramene's Maize PanGenome](http://maize-pangenome.gramene.org), a web portal for comparative plant genomics focused on Maize crop varieties, has now released its fifth version featuring the genome sequence and gene annotation of 18 new maize genomes (10 CAAS genomes ([Wang et al, 2023](https://doi.org/10.1038/s41588-022-01283-w)) and 8 [PanAnd](https://mcstitzer.github.io/panand_assemblies/) genomes including a second assembly of teosinte Ti11, see the list of “New Genomes” below), and a genetic variation data set with over 228 million SNPs generated by MaizeGDB from 1,525 public resequenced lines including inbred lines, landraces, and teosintes. In total, the site now hosts a total of 63 genomes including 56 maize assemblies (54 unique maize genomes). In addition to the new CAAS and PanAnd genomes, the site hosts three assembly versions of the B73 Zea mays reference (RefGen_v3, RefGen_v4, and RefGen_v5), B73 Ab10, 25 [NAM maize founder genomes](https://nam-genomes.github.io/), 4 European flint genomes (DK105, EP1, F7 and PE0075), 4 additional maize genomes (Mo17, W22, PH207, and A188), and teosinte Til11 from the [MaizeCODE project](http://www.maizecode.org/data/til11.html) (Cahn et al, 2024). Plus 7 outgroups (Arabidopsis thaliana, Oryza sativa Japonica Nipponbare, Sorghum bicolor BTx623, Vitis vinifera ssp. vinifera PN40024, Selaginella moellendorffii, Chlamydomonas reinhardtii, and Drosophila melanogaster). The maize gene models, together with the seven outgroups, were used to build XX protein-coding gene family trees. In addition, pangene sets generated by MaizeGDB staff using the Pandagma pipeline (56 maize and 1 sorghum genomes) are being provided. This release also introduces Conserved Non-Coding Sequence (CNS) data from the [PanAnd project](https://mcstitzer.github.io/panand_assemblies/), encompassing 2.3 million elements across Andropogoneae species. Enriched for regulatory regions such as transcription factor binding sites, promoters, and chromatin-accessible regions, this data offers valuable insights into gene expression regulation and polyploid genome evolution.

We are also providing updated documented functional gene associations from [NCBI's GeneRIF](https://www.ncbi.nlm.nih.gov/gene/about-generif). New features within the results of a database query include: 1) a new Germplasm tab listing accessions bearing protein-truncating variants or PTVs for putative loss-of-function SNPs and with direct links to MaizeGDB or GRIN, and 2) gene search suggestions with a taxonomy label following the number of genes containing the matching term to distinguish similar gene names assigned to multiple species.

Data from the EBI Expression Atlas amounts to 33 baseline gene expression studies on the B73 (v5) reference, including RNA-seq samples for 10 tissues from each of the 25 maize NAM genomes. This data together with baseline expression of paralogs for a reference study per gene, and eFP expression visualization from 12 studies, are available through the Expression tab of the Gene Search results. Differential experiments are available at the EBI Expression Atlas site. The Pathways tab continues to display 270 orthology-based pathway projections from the Plant Reactome, and the Papers tab is periodically updated with curated scientific literature associated with a gene function from NCBI’s GeneRIF, as well as gene name and symbols submitted by members of the Maize community through the [Submit a gene function form](https://docs.google.com/forms/d/e/1FAIpQLSey-xPyTysdd9c2phXT6kcbjEDeCppG4dLG7LjZCeLpx_KGog/viewform).

The outgroup genome databases were built in direct collaboration with the [Ensembl Plants](http://plants.ensembl.org) project. Other data sets were facilitated via collaborations with the [Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments), the [Plant Reactome](https://plantreactome.gramene.org/), and the [Bio-Analytic Resource for Plant Biology or BAR](https://bar.utoronto.ca/) databases.


## Release Information
- [Overall Highlights](#overall-highlights)
- [Comparative Genomics](#comparative-genomics)
- [Variation](#variation)
- [Gene Expression](#gene-expression)
- [Pathways](#pathways)
- [Publications](#publications)
- [Funding](#funding)

## Overall Highlights
**New genomes**
- 18 new maize reference genomes:
  * CAAS:
    * _Zea mays_ A632
    * _Zea mays_ Chang
    * _Zea mays_ Dan340
    * _Zea mays_ Huangzaosi
    * _Zea mays_ Jing724
    * _Zea mays_ Jing92
    * _Zea mays_ S37
    * _Zea mays_ Xu178
    * _Zea mays_ Ye478
    * _Zea mays_ Zheng58
  * PanAnd:
    * _Zea diploperennis_ Gigi
    * _Zea diploperennis_ Momo
    * _Zea mays_ huehuetenagensis Rimhu001
    * _Zea nicaraguensis_ PI615697
    * _Zea mays_ parviglumis Til01 (teosinte)
    * _Zea mays_ parviglumis Til11 (teosinte)
    * _Zea mays_ mexicana Til18 (teosinte)
    * _Zea mays_ mexicana Til25 (teosinte)

**New data**
- XX protein-coding gene family trees from XX protein sequences
- Pangene sets from 57 species generated by [MaizeGDB](https://www.maizegdb.org/)
- Genetic variation consisting of ~230 million SNPs from a diverse set of 1,525 public resequenced lines including inbred lines, landraces, and teosintes from [MaizeGDB’s SNPversity](https://wgs.maizegdb.org/).
- Reference cluster identifiers (rsIDs) updated according to the [6th release of the European Variation Archive](https://www.ebi.ac.uk/eva/?RS-Release&releaseVersion=6)
- Conserved sequence (CNS) data from the [PanAnd project](https://mcstitzer.github.io/panand_assemblies/)
- Gene expression profiles for [25 NAM maize genomes via the Expression Atlas](https://www.ebi.ac.uk/gxa/experiments/E-MTAB-8633/Experiment%20Design)

**New features**

- A new genome browser track with a subset of rsIDs within a 1.5 Kb region flanking B73 genes projected to the 25 NAM genomes, see for example [LINK]

These are enhancements to the Gramene Maize's Search Results:

- The Gene search suggestions now include a taxonomy label following the number of genes containing the matching term. This helps users distinguish similar gene names assigned to multiple species. If a search term is associated with genes from multiple genomes, the lowest common ancestor in the taxonomy tree is appended.

- The new Germplasm tab lists germplasm bearing a protein-truncating variant (PTVs are putative loss-of-function SNPs) within the canonical transcript of a gene model. The table includes links to MaizeGDB and/or GRIN, as well as a way to search for other genes with a PTV in a given germplasm accession. The indexed PTVs include the following predicted functional effects: start lost, stop gained, stop lost, splice site acceptor, and splice site donor.



## Comparative Genomics

[**Gene Trees.**](http://maize-pangenome-ensembl.gramene.org/prot_tree_stats.html) A total of
XX protein-coding gene family trees were constructed using the peptide encoded by
the canonical transcript (i.e., a representative transcript for a given gene) for each
of XX individual genes (XX input proteins) from XX plant genomes.

[**Synteny**](https://maize-pangenome-ensembl.gramene.org/compara_analyses.html). 143 pairwise synteny maps between _Z. mays_ B73 v5 and each of 35 maize genomes (including _Z. mays_ v4), as well as all 36 maize genomes in R4 and Arabidopsis, rice or sorghum.

[**Whole-Genome Alignments**](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html). Pairwise genomic alignments for B73 v5 with sorghum, Japonica rice, wheat, and Brachypodium are available in the [Gramene site](https://www.gramene.org) and will be made available here across the maize genomes in future releases.

## Variation

The following variation datasets are mapped to the _Z. mays_ B73 v5 reference assembly:

**MaizeGDB 2024 Dataset**

This dataset was generated using a diverse set of inbred lines, landraces, and teosintes from 1,525 public resequenced lines through a standardized variant-calling pipeline against version 5 of the B73 reference genome (Andorf et al, 2025). The output was filtered for mapping quality, genotypic coverage, and linkage disequilibrium, and annotated based on variant effects relative to the B73 RefGen_v5 gene annotations. Two versions of the dataset are available. A high-coverage dataset consisting of ~230 million loci was filtered on mapping quality and genotypic coverage. The projects used to build the dataset include ten studies: 1) WGS resequencing of 1,604 maize inbred lines (Li et al, 2022), 2) Deep DNA resequencing of the association mapping panel (Yang et al, 2019), 3) Maize landrace whole genome resequencing (Wang et al, 2022), 4) Maize Wisconsin Diversity Panel Resequencing (Grzybowski et al, 2023), 5) _Zea mays_ Genome sequencing (Zhang and Qi, 2021), 6) Whole genome sequencing of maize 282 panel (Bukowski et al, 2018), 7) European maize diversity (Brandenburg et al, 2017), 8) Maize Haplotype Map version 3 (Bukowski et al, 2018), 9) _Zea mays_ Genome sequencing (European maize genomes (Haberer et al, 2020), and 10) Resequencing of three Polish maize inbred lines (Grzybowski et al, 2023).

**HapMap2 dataset**

A variation set which comprises the maize HapMap2 data (Chia et al, 2012). This dataset incorporates approximately 55 million SNPs and InDels identified in a collection of 103 pre-domesticated and domesticated _Zea mays_ varieties, including a representative from the sister genus, _Tripsacum dactyloides_ (Eastern gamagrass). Each line was sequenced to an average of 4.5-fold coverage using the Illumina GAIIx platform. The reads can be accessed from the SRA, with accession ID: SRA051245. Reads were initially mapped to the B73 RefGen_v3 reference genome using a combination of Bowtie, Novoalign and SOAP, then remapped to the most recent B73 RefGen_v4 and subsequently RefGen_v5 reference assemblies. The variations were scored by taking into account identity-by-descent blocks that are shared among the lines.

**The [PanZea](https://www.panzea.org/genotypes) 2.7 genotyped-by-sequencing (GBS) dataset**

This variation data set consists of 719,472 SNPs (excluding 332 SNPs removed for mapping to scaffolds) typed in 16,718 maize and teosinte lines (, and grouped in 14 overlapping populations according to the germplasm set in the corresponding metadata table.

*Structural Variation*

A cumulative total of 791,101 structural variants larger than 100 bp in size determined by comparative analysis among the NAM genotypes to B73 (Hufford et al, 2021) are available as data tracks on the B73 v5 genome browser, see for example [this region](https://maize-pangenome-ensembl.gramene.org/Zea_maysb73/Location/View?r=1:109000-145001): Note: If you don't see the data, click on the configure button (spider gear icon) and select the genome(s) that you want to see.


## Gene Expression

Gene expression data for the Zea mays B73 genome reference was curated and
processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments).
The current set consists of 33 studies with baseline expression and 57 with differential expression. Baseline experiments as well as expression of paralogs for the reference study and electronic Fluorescent Pictograph (eFP) expression visualizations are available through Gramene's Gene Search (Expression panel) and Genome Browser; [differential experiments](https://www.ebi.ac.uk/gxa/experiments?kingdom=Plants&species=Zea+mays&experimentType=Differential) are available at the Expression Atlas.

Additional expression data is also available as data tracks on the corresponding maize genome browsers from the NAM and MaizeENCODE projects, see for examples [this region of the Til11 genome](https://maize-pangenome-ensembl.gramene.org/Zea_maysti11/Location/View?r=1:109000-145001) (note that you only see MaizeCODE tracks because Til11 is not part of the NAM project). For gene expression data from both NAM and MaizeCODE projects, check out [this region of the B73 v5 genome](https://maize-pangenome-ensembl.gramene.org/Zea_maysb73/Location/View?r=1:109000-145001).


## Pathways

In this fifth release, there are 270 orthology-based maize pathways associated with 1,769 genes in the [Plant Reactome](https://plantreactome.gramene.org/) pathway knowledgebase. We utilize the Reactome pathway data model to represent plant metabolic, transport and signaling pathways, developmental processes, organ differentiation, and transcriptional regulatory networks. Manual biocuration is conducted in the reference _Oryza sativa_ Japonica species, and pathways are then projected via gene orthology to the B73 _Zea mays_, and other crops and model plants, lower plants and single-cell photoautotrophs.


## Funding

Core funding for the project is provided by the Agricultural Research Service of the U.S. Department of Agriculture (USDA ARS 8062-21000-051-00D). The MaizeCODE project was funded via a National Science Foundation award #1445025.

## Publications

Andorf C. M., Ross-Ibarra J., Seetharam A. S., Hufford M. B., Woodhouse M. R. (2025). A unified VCF data set from nearly 1,500 diverse maize accessions and resources to explore the genomic landscape of maize. G3 15(2):jkae281. https://doi.org/10.1093/g3journal/jkae281

Brandenburg, J.-T., Mary-Huard, T., Rigaill, G., Hearne, S. J., Corti, H., Joets, J., Vitte, C., Charcosset, A., Nicolas, S. D., & Tenaillon, M. I. (2017). Independent introductions and admixtures have contributed to adaptation of European maize and its American counterparts. PLoS Genetics, 13(3), e1006666. https://doi.org/10.1371/journal.pgen.1006666

Bukowski, R., Guo, X., Lu, Y., Zou, C., He, B., Rong, Z., Wang, B., Xu, D., Yang, B., Xie, C., Fan, L., Gao, S., Xu, X., Zhang, G., Li, Y., Jiao, Y., Doebley, J. F., Ross-Ibarra, J., Lorant, A., … Xu, Y. (2018). Construction of the third-generation Zea mays haplotype map. GigaScience, 7(4), 1–12. https://doi.org/10.1093/gigascience/gix134

Chia, J.-M., Song, C., Bradbury, P. J., Costich, D., de Leon, N., Doebley, J., Elshire, R. J., Gaut, B., Geller, L., Glaubitz, J. C., Gore, M., Guill, K. E., Holland, J., Hufford, M. B., Lai, J., Li, M., Liu, X., Lu, Y., McCombie, R., … Ware, D. (2012). Maize HapMap2 identifies extant variation from a genome in flux. Nature Genetics, 44(7), 803–807. https://doi.org/10.1038/ng.2313

Glaubitz, J. C., Casstevens, T. M., Lu, F., Harriman, J., Elshire, R. J., Sun, Q., & Buckler, E. S. (2014). TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline. PloS One, 9(2), e90346. https://doi.org/10.1371/journal.pone.0090346

Grzybowski, M. W., Mural, R. V., Xu, G., Turkus, J., Yang, J., & Schnable, J. C. (2023). A common resequencing-based genetic marker data set for global maize diversity. The Plant Journal: For Cell and Molecular Biology, 113(6), 1109–1121. https://doi.org/10.1111/tpj.16123

Haberer, G., Kamal, N., Bauer, E., Gundlach, H., Fischer, I., Seidel, M. A., Spannagl, M., Marcon, C., Ruban, A., Urbany, C., Nemri, A., Hochholdinger, F., Ouzunova, M., Houben, A., Schön, C.-C., & Mayer, K. F. X. (2020). European maize genomes highlight intraspecies variation in repeat and gene content. Nature Genetics, 52(9), 950–957. https://doi.org/10.1038/s41588-020-0671-9

Hufford, M. B., Seetharam, A. S., Woodhouse, M. R., Chougule, K. M., Ou, S., Liu, J., Ricci, W. A., Guo, T., Olson, A., Qiu, Y., Della Coletta, R., Tittes, S., Hudson, A. I., Marand, A. P., Wei, S., Lu, Z., Wang, B., Tello-Ruiz, M. K., Piri, R. D., … Kelly Dawe, R. (2021). De novo assembly, annotation, and comparative analysis of 26 diverse maize genomes. Science, 373(2021), 655–662. https://doi.org/10.1126/science.abg5289

Li, C., Guan, H., Jing, X., Li, Y., Wang, B., Li, Y., Liu, X., Zhang, D., Liu, C., Xie, X., Zhao, H., Wang, Y., Liu, J., Zhang, P., Hu, G., Li, G., Li, S., Sun, D., Wang, X., … Wang, H. (2022). Genomic insights into historical improvement of heterotic groups during modern hybrid maize breeding. Nature Plants, 8(7), 750–763. https://doi.org/10.1038/s41477-022-01190-2

Cahn J, Regulski M, Lynn J, Ernst E, de Santis Alves C, Ramakrishnan S, Chougule K, Wei S, Lu Z, Xu X, Ramu U, Drenkow J, Kramer M, Seetharam A, Hufford MB, McCombie WR, Ware D, Jackson D, Schatz MC, Gingeras TR, Martienssen RA. (2024). MaizeCODE reveals bi-directionally expressed enhancers that harbor molecular signatures of maize domestication. Nat Commun. 15(1):10854. doi: 10.1038/s41467-024-55195-w

Wang, Y., Li, W., Wang, L., Yan, J., Lu, G., Yang, N., Xu, J., Wang, Y., Gui, S., Chen, G., Li, S., Wu, C., Guo, T., Xiao, Y., Warburton, M. L., Fernie, A. R., Dresselhaus, T., & Yan, J. (2022). Three types of genes underlying the Gametophyte factor1 locus cause unilateral cross incompatibility in maize. Nature Communications, 13(1), 4498. https://doi.org/10.1038/s41467-022-32180-9

Yang, N., Liu, J., Gao, Q., Gui, S., Chen, L., Yang, L., Huang, J., Deng, T., Luo, J., He, L., Wang, Y., Xu, P., Peng, Y., Shi, Z., Lan, L., Ma, Z., Yang, X., Zhang, Q., Bai, M., … Yan, J. (2019). Genome assembly of a tropical maize inbred line provides insights into structural variation and crop improvement. Nature Genetics, 51(6), 1052–1059. https://doi.org/10.1038/s41588-019-0427-6

Zhang, X., & Qi, Y. (2021). Genetic architecture affecting maize agronomic traits identified by variance heterogeneity association mapping. Genomics, 113(4), 1681–1688. https://doi.org/10.1016/j.ygeno.2021.04.009
