## Vitis PanGenome Release 4.0
### Released: February 2024
## Summary
The Vitis PanGenome, a web portal for comparative plant genomics focused on grapevine varieties, has released its fourth version. It provides access to 29 grape reference genomes including 11 new grape genomes: 8 non-vinifera Vitis species (Cochetel _et al_, 2023), 2 Malbec varieties (Calderón _et al_, 2023), and one wild sylvestris variety VS-1 (Dong _et al_, 2023), and 8 non-grape species -including tomato, a fleshy fruit- selected for comparative analysis. The genome databases were built in direct collaboration with the [Gramene](http://gramene.org) project. Gene expression and orthology-based pathway association is available for the grapevine, _Vitis vinifera_ PN40024 reference v4 via the Search functionality, and were facilitated through collaborations with the [Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments), and the [Plant Reactome](https://plantreactome.gramene.org/) databases. The genomes sequenced by the VitisGen2 project are also available at [grapegenomics.com](http://www.grapegenomics.com) hosted by the Cantu Lab at the University of California in Davis.

Core funding for the Gramene Vitis PanGenome project is provided by the Agricultural Research Service of the [U.S. Department of Agriculture](http://www.usda.gov/) (USDA ARS 8062-21000-051-00D) to the Ware Lab at the Cold Spring Harbor Laboratory. We would like to thank the grapevine research community, especially the VitisGen project (currently as [VitisGen3](https://vitisgen3.umn.edu/); USDA NIFA SCRI 2022-51181-38240), [INTEGRAPE](https://integrape.eu/) (EU COST Action CA17111) and [GRAPEDIA](https://grapedia.org/) (partly funded by a COST Innovators Grant) collaborators for generously sharing their data, making valuable contributions, and providing feedback on the site. 


## Release Information
- [Overall Highlights](#overall-highlights)
- [Databases](#databases)
- [New Features](#features)
- [References](#references)

## Overall Highlights {#overall-highlights}
- 11 new grape reference genomes:
  - [_Vitis vinifera ssp. vinifera_ Malbec ‘Magdeleine’](https://vitis-ensembl.gramene.org/Vitis_malbecmagdeleine/Info/Index)
  - [_V. vinifera ssp. vinifera_ Malbec ‘Prunelard’](https://vitis-ensembl.gramene.org/Vitis_malbecprunelard/Info/Index)
  - [_V. vinifera ssp. sylvestris_ VS-1](https://vitis-ensembl.gramene.org/Vitis_vinifera/Info/Index)
  - [_Vitis acerifolia_](https://vitis-ensembl.gramene.org/Vitis_acerifolia/Info/Index)
  - [_Vitis aestivalis_](https://vitis-ensembl.gramene.org/Vitis_aestivalis/Info/Index)
  - [_Vitis berlanderi_](https://vitis-ensembl.gramene.org/Vitis_berlanderi/Info/Index)
  - [_Vitis girdiana_](https://vitis-ensembl.gramene.org/Vitis_girdiana/Info/Index)
  - [_Vitis monticola_](https://vitis-ensembl.gramene.org/Vitis_monticola/Info/Index)
  - [_Vitis mustangensis_](https://vitis-ensembl.gramene.org/Vitis_mustangensis/Info/Index)
  - [_Vitis riparia_](https://vitis-ensembl.gramene.org/Vitis_riparia/Info/Index)
  - [_Vitis rupestris_](https://vitis-ensembl.gramene.org/Vitis_rupestris/Info/Index)

 <!---[_V. vinifera ssp. vinifera_ Tempranillo ‘Albillo’](https://vitis-ensembl.gramene.org/Vitis_tempranilloalbillo/Info/Index)
  - [_V. vinifera ssp. vinifera_ Tempranillo ‘Benedicto’](https://vitis-ensembl.gramene.org/Vitis_tempranillobenedicto/Info/Index)-->
  
- 18 existing grape reference genomes including two versions of the partially inbred Pinot Noir reference (clone PN40024):
  [*Vitis vinifera* (L.) ssp. *vinifera* Pinot noir PN40024 v3](https://vitis-ensembl.gramene.org/Vitis_vinifera/Info/Index)
  and [*Vitis vinifera* (L.) ssp. *vinifera* Pinot noir PN40024 v4](https://vitis-ensembl.gramene.org/Vitis_viniferapn40024).

  The remaining grapevine genomes include 12 *Vitis vinifera* (L.) ssp. *vinifera* varieties:
  
  Cabernet Sauvignon (clone FPS08),
  Carménère (clone FPS02),
  Chardonnay (clone FPS04),
  Concord (interspecific hybrid PI 588077),
  Flame Seedless (clone Parlier),
  Merlot (clone FPS15),
  Zinfandel (clone FPS03),
  Black Corinth Seeded (seeded Black Corinth or Zante Currant),
  Black Corinth Seedless (seedless Black Corinth or Zante Currant),
  Cabernet Franc Cl. 04 (Cabernet Franc),
  Nebbiolo, and
  Riesling;
  two *Vitis vinifera* spp. *sylvestris* varieties:
  C1-2 (Badouin et al, 2020) and
  DVIT3351.27 (a male individual carrying a Ren1+ locus collected from Armenia);
  and two non-vinifera species:
  _Vitis arizonica_ (Arizona grape, canyon grape or uva del monte), and
  _Muscadinia rotundifolia_ cv. Trayshed (Muscadine), also known as _Vitis rotundifolia_.

**Note on updated genomes**: In this release, a new version of _Vitis arizonica_ replaced the existing one.

- Seven plant outgroup species (tomato, Japonica rice, B73 maize, Arabidopsis thaliana, grapevine, a vascular plant, and a single-celled green algae) and Drosophila melanogaster were used to build 33,275 protein-coding gene family trees. Tomato, a fleshy fruit, is being included for the first time in this release of the Gramene Vitis PanGenome.

- Our comparative genomics collection includes a total of 33,275 phylogenetic trees, of which 16,065 are grapevine specific. These family trees were constructed with 1,337,885 input proteins from 1,288,848 individual genes. Pairwise DNA alignments for version 4 of the Pinot noir PN40024 reference and _Arabidopsis thaliana_ are available in the [Gramene website](https://ensembl.gramene.org/Vitis_vinifera/Info/Index).

- Gene expression and orthology-based pathway projections are available for version 4 of the PN40024 grape reference genome via the Search interface.

- BLAST alignments are also available for all 37 plant genomes in the Vitis pangenome portal.
  

## Databases {#databases}
### Comparative Genomics

[**Gene Trees**](https://vitis-ensembl.gramene.org/prot_tree_stats.html). A total of 33,275 protein-coding gene family trees were constructed using the peptide encoded by the canonical transcript (i.e., a representative transcript for a given gene) for each of 1,288,848 individual genes (1,337,885 input proteins) from 38 plant genomes.

[**Whole-Genome Alignments**](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html). Pairwise genomic alignments is available for version 4 of the Pinot noir PN40024 reference assembly and _Arabidopsis thaliana_. New alignments across the grape genomes will be made available here in future releases.

[**Synteny**](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html). Synteny maps across all grape genomes and the PN40024 v4 reference will be made available in future releases.


### Variation

Over 300K SNPs identified by re-sequencing a collection of grape cultivars and wild Vitis species from the [USDA germplasm collection](http://www.ars.usda.gov/Main/docs.htm?docid=12254) (Myles et al, 2010) were remapped to version 4 of the Pinot noir PN40024 reference assembly and assigned unique standard rsIDs.

### Expression

Gene expression data is now available for the PN40024 grape v4 genome reference was curated and processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments). The set consists of 10 studies with baseline expression and 23 differential expression studies.

### Pathways

268 orthology-based pathways were projected from curated Japonica rice pathways to _Vitis vinifera_ v4 and linked to [Gramene’s Plant Reactome](https://plantreactome.gramene.org/PathwayBrowser/#/SPECIES=9030216). The pathways are associated with 1,304 grapevine PN40024 v4 genes. 

## New Features {#features}

This release of the site integrates two features in the gene search interface:

1. Papers tab lists publications describing gene function. Search for [curated genes](https://vitis.gramene.org/?fq_field=capabilities&fq_value=pubs&category=Curated&name=publication)
2. Sequences tab providing quick access to gene, transcript and protein sequences for a gene model

   
## References {#references}

1. Cochetel N, Minio A, Guarracino A, Garcia JF, Figueroa-Balderas R, Massonnet M, Kasuga T, Londo J, Garrison E, Gaut B, Cantu D (2023) A super-pangenome of the North American wild grape species. Genome Biology 24:290. [doi: 10.1186/s13059-023-03133-2](https://doi.org/10.1186/s13059-023-03133-2).
2. Calderón L, Carbonell-Bejerano P, Muñoz P, Bree L, Sola C, Bergamin D, Tulle W, Gomez-Talquenca S, Lanz C, Royo C, Ibáñez J, Martinez-Zapater JM, Weigel D, Lijavetzky D. 2023. Diploid genome assembly of the Malbec grapevine cultivar enables haplotype-aware analysis of transcriptomic differences underlying clonal phenotypic variation. biorxiv. [doi: 10.1101/2023.11.30.569420](https://doi.org/10.1101/2023.11.30.5694200).
3. 	Dong Y, Duan S, Xia Q, Liang Z, Dong X, Margaryan K, et al. Dual domestications and origin of traits in grapevine evolution. Science. 2023;379: 892–901. [doi:10.1126/science.add8655](http://doi.org/0.1126/science.add8655).
4.	Myles S, Chia JM, Hurwitz B, Simon C, Zhong GY, Buckler E, Ware D (2010) Rapid genomic characterization of the genus vitis. PLoS ONE. 5:e8219. [doi: 10.1371/journal.pone.0008219](https://doi.org/10.1371/journal.pone.0008219).


