
## Vitis PanGenome Release 3.0
### Released: May 30, 2022
## Summary
The Vitis PanGenome, a web portal for comparative plant genomics focused on grapevine varieties, has released its third version. It provides access to 18 grape reference genomes, together with seven other species selected for comparative analysis. The genome databases were built in direct collaboration with the [Gramene](http://gramene.org) project. Gene expression and orthology-based pathway association is available for the grapevine PN40024 reference v3 via the Search functionality, and were facilitated through collaborations with the [Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments), and the
[Plant Reactome](https://plantreactome.gramene.org/) databases. Core funding for the project is provided by the National Sciene Foundation ([NSF IOS-1127112](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1127112) and the Agricultural Research Service of the [U.S. Department of Agriculture](http://www.usda.gov/) (USDA ARS 8062-21000-041-00D) to the Ware Lab at the Cold Spring Harbor Laboratory. 

We would like to thank the grapevine research community, especially the VitisGen2 (https://www.vitisgen2.org; USDA NIFA SCRI 51181-26829) and the EU-COST action CA 17111 (http://www.integrape.eu) collaborators for generously sharing their data, making valuable contributions, and providing feedback on the site. The genomes sequenced by the VitisGen2 are also available at [grapegenomics.com](http://www.grapegenomics.com) hosted by the Cantu Lab at the University of California in Davis. Funding for the Flame Seedless genome was provided by the California Table Grape Commission, and the Concord genome was sequenced and assembled by 10X Genomics. The genome assembly of the Cabernet sauvignon variety clone FPS08 was funded by E.&J. Gallo Winery and J. Lohr Vineyards and Wines. The genome assembly of the Carménère variety clone FPS02 was funded by the Chilean Economic Development Agency (CORFO; Project 13CEI2-21852), Viña Concha y Toro, and Viña San Pedro. The genome assembly of the Chardonnay variety clone FPS04 was funded by UC Davis Chile Life Sciences Innovation Center, Viña Concha y Toro, E. & J. Gallo Winery, J. Lohr Vineyards and Wines, Viña San Pedro, Louis P. Martini Endowment in Viticulture, and National Science Foundation Plant Genome Research grant #1741627. The genome assembly of the Merlot variety clone FPS15 was funded by UC Davis Chile Life Sciences Innovation Center, Viña Concha y Toro, and Viña San Pedro. The genome assembly of the Zinfandel variety clone FPS03 was funded by Louis P. Martini Endowment in Viticulture. The genome assembly of *Vitis vinifera* spp. *sylvestris* clone DVIT3351.27 was funded by National Science Foundation Plant Genome Research grant #1741627.


## Release Information
- [Overall Highlights](#overall-highlights)
- [Databases](#databases)
- [Statistics](#statistics)
- [References](#references)

## Overall Highlights {#overall-highlights}
- Seven new grape reference genomes 
- Eleven existing grape reference genomes including two versions of the partially inbred Pinot Noir reference (clone PN40024): [*Vitis vinifera* (L.) ssp. *vinifera* Pinot noir PN40024 v3](https://vitis-ensembl.gramene.org/Vitis_vinifera/Info/Index) and [*Vitis vinifera* (L.) ssp. *vinifera* Pinot noir PN40024 v4](https://vitis-ensembl.gramene.org/Vitis_viniferapn40024). The remaining grapevine genomes include seven *Vitis vinifera* (L.) ssp. *vinifera* varieties: [Cabernet Sauvignon (clone FPS08)](https://vitis-ensembl.gramene.org/Vitis_cabsauvignon), [Carménère (clone FPS02)](https://vitis-ensembl.gramene.org/Vitis_carmenere), [Chardonnay (clone FPS04)](https://vitis-ensembl.gramene.org/Vitis_chardonnay), [Concord (interspecific hybrid PI 588077)](https://vitis-ensembl.gramene.org/Vitis_concord10x), [Flame Seedless (clone Parlier)](https://vitis-ensembl.gramene.org/Vitis_flamepac), [Merlot (clone FPS15)](https://vitis-ensembl.gramene.org/Vitis_merlot), and [Zinfandel (clone FPS03)](https://vitis-ensembl.gramene.org/Vitis_zinfandel); plus two *Vitis vinifera* spp. *sylvestris* varieties: [C1-2 (Badouin et al, 2020)](https://vitis-ensembl.gramene.org/Vitis_sylc12) and [DVIT3351.27](https://vitis-ensembl.gramene.org/Vitis_syldvit3351) (a male individual carrying a Ren1+ locus collected from Armenia).
- Six plant outgroup species (Japonica rice, B73 maize, Arabidopsis thaliana, grapevine, a vascular plant, and a single-celled green algae) and Drosophila melanogaster were used to build 24,050 protein-coding gene family trees.
- Our comparative genomics collection includes a total of 27,620 phylogenetic trees, of which xxx are grapevine specific. These family trees were constructed with 885,518 input proteins from 841,363 individual genes. Pairwise DNA alignments and synteny maps are available for version 3 of the Pinot noir PN40024 reference in the [Gramene website](https://ensembl.gramene.org/Vitis_vinifera/Info/Index).
- Gene expression and orthology-based pathway projections are available for version 3 of the PN40024 grape reference genome via the Search interface.
- BLAST alignments available for all 25 genomes in the Vitis pangenome portal.

## Databases {#databases}
### Comparative Genomics

[**Gene Trees.**](https://vitis-ensembl.gramene.org/prot_tree_stats.html) A total of 27,620 protein-coding gene family trees were constructed using the peptide encoded by the canonical transcript (i.e., a representative transcript for a given gene) for each of 841,363 individual genes (885,518 input proteins) from 25 plant genomes.

[**Whole-Genome Alignments**](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html). 76 pairwise genomic alignments are available for version 3 of the Pinot noir PN40024 reference assembly in the [Gramene website](https://ensembl.gramene.org/Vitis_vinifera/Info/Index). New alignments across the grape genomes will be made available here in future releases.

[**Synteny**](https://ensembl.gramene.org/info/genome/compara/compara_analyses.html). Eight synteny maps are available for version 3 of the Pinot noir PN40024 reference assembly in the [Gramene website](https://ensembl.gramene.org/Vitis_vinifera/Info/Index). New synteny maps across the grape genomes will be made available here in future releases.


### Variation

Genetic variation is available for version 3 of the Pinot noir PN40024 reference assembly in the [Gramene website](https://ensembl.gramene.org/Vitis_vinifera/Info/Index) for will be available in future releases. Genetic variation data sets will be made availale here in future releases.

### Expression

Gene expression data for the PN40024 grape v3 genome reference was curated and processed through the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/plant/experiments). The set consists of 10 studies with baseline expression and 23 baseline studies.

### Pathways

269 orthology-based grape pathways associated with 1,076 grapevine PN40024 v3 genes. These were projected from curated Japonica rice pathways and are linked to [Gramene’s Plant Reactome](https://plantreactome.gramene.org/PathwayBrowser/#/SPECIES=9030216).

## Statistics {#statistics}
| Species | Classification | Note | Assembly | Literature reference | Accession | Expression | Phylogenetic Trees | Pathways | BioProject or Genomics Data Source|
|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
| *Vitis vinifera* L. ssp. *vinifera* Pinot noir PN40024 v3 | Eudicotyledons | Dicot reference | IGGP_12x (**V3**) | [Jaillon *et al* (2007)](https://www.nature.com/articles/nature06148) | GCA_000003745.2 | Search only | Y | Search only | - |
| *Vitis vinifera* L. ssp. *vinifera* Pinot noir PN40024 v4 | Eudicotyledons | Dicot reference | v1 (**V4**) | [INTEGRAPE](https://integrape.eu/resources/genes-genomes/genome-accessions/) | PN40024_40X_REF_AnnoV2 | - | Y | - | - |
| *Vitis vinifera* L. ssp. *vinifera* Cabernet Sauvignon clone FPS08 | Eudicotyledons | | Cabernet_Sauvignon | [Massonnet *et al* (2020)](https://www.nature.com/articles/s41467-020-16700-z) & [Chin *et al* (2016)](https://www.nature.com/articles/nmeth.4035) | VvCabSauv08_v1 | - | Y | - | [PRJNA593045](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA593045), [Data](https://doi.org/10.5281/zenodo.3827985), [PRJNA316730](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA316730), [PRJNA433195](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA433195) & [Cantu Lab](http://cantulab.github.io/data.html) |
| *Vitis vinifera* L. ssp. *vinifera* Carménère clone FPS02 | Eudicotyledons | | Carmenere | [Minio *et al* (2019)](https://www.g3journal.org/content/9/5/1331) | VvCarFPS02VCR702_v1 | - | Y | - | [PRJNA517468](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA517468), [Cantu Lab](http://cantulab.github.io/data.html) |
| *Vitis vinifera* L. ssp. *vinifera* Chardonnay clone FPS04 | Eudicotyledons | | Chardonnay | [Zou *et al* (2019)](https://www.nature.com/articles/s41477-019-0507-8) | VvChar04_v1 | - | Y | - | [PRJNA550461](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA550461) |
| *Vitis vinifera* L. ssp. *vinifera* interspecific crossing Concord PI588077 | Eudicotyledons | | 10x201806 |  [Zou *et al* (2020)](https://www.nature.com/articles/s41467-019-14280-1) | 10x201806 | - | Y | - | [PRJNA281110](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA281110) |
| *Vitis vinifera* L. ssp. *vinifera* Flame Seedless clone Parlier | Eudicotyledons | | pacbio201806 | [Zou *et al* (2020)](https://www.nature.com/articles/s41467-019-14280-1) |   | - | Y | - | [PRJNA281110](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA281110) |
| *Vitis vinifera* L. ssp. *vinifera* Merlot clone FPS15 | Eudicotyledons | | Merlot | [Massonnet *et al* (2020)](https://www.nature.com/articles/s41467-020-16700-z) | VITVvi_vMerlFPS15_v1.1 | - | Y | - | [PRJNA593045](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA593045), [Data](https://doi.org/10.5281/zenodo.3827985) |
| *Vitis vinifera* L. ssp. *vinifera* Zinfandel clone FPS03 | Eudicotyledons | | Zinfandel | [Vondras *et al* (2019)](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-6211-2) | VvZin03_v1 | - | Y | - | [PRJNA527006](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA527006) & [Cantu Lab](http://cantulab.github.io/data.html) |
| *Vitis vinifera* L. ssp. *sylvestris* C1-2 (Badouin et al, 2020) | Eudicotyledons | | Sylvestris_C1-2 | [Badouin *et al* (2020)](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02131-y) | Sylvestris_C1-2 | - | Y | - | [PRJEB37020](https://www.ncbi.nlm.nih.gov/bioproject/PRJEB37020) |
| *Vitis vinifera* L. ssp. *sylvestris* DVIT3351.27 (Ren1 male) | Eudicotyledons | | Sylvestris_DVIT3351.27 | [Massonnet *et al* (2020)](https://www.nature.com/articles/s41467-020-16700-z) | VITVvi_sDVIT3351.27_v1.1 | - | Y | - | [PRJNA593045](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA593045), [Data](https://doi.org/10.5281/zenodo.3827985) |

## References {#references}

1.	Jaillon, O., Aury, J.M., Noel, B., Policriti, A., Clepet, C., Casagrande, A., Choisne, N., Aubourg, S., Vitulo, N., Jubin, C. et al. (2007) The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla. Nature, 449, 463-467.
2.	Massonnet, M., Cochetel, N., Minio, A., Vondras, A.M., Lin, J., Muyle, A., Garcia, J.F., Zhou, Y., Delledonne, M., Riaz, S. et al. (2020) The genetic basis of sex determination in grapes. Nat Commun, 11, 2902.
3.	Chin, C.S., Peluso, P., Sedlazeck, F.J., Nattestad, M., Concepcion, G.T., Clum, A., Dunn, C., O'Malley, R., Figueroa-Balderas, R., Morales-Cruz, A. et al. (2016) Phased diploid genome assembly with single-molecule real-time sequencing. Nat Methods, 13, 1050-1054.
4.	Minio, A., Massonnet, M., Figueroa-Balderas, R., Castro, A. and Cantu, D. (2019) Diploid Genome Assembly of the Wine Grape Carmenere. G3 (Bethesda), 9, 1331-1337.
5.	Zhou, Y., Minio, A., Massonnet, M., Solares, E., Lv, Y., Beridze, T., Cantu, D. and Gaut, B.S. (2019) The population genetics of structural variants in grapevine domestication. Nat Plants, 5, 965-979.
6.	Zou, C., Karn, A., Reisch, B., Nguyen, A., Sun, Y., Bao, Y., Campbell, M.S., Church, D., Williams, S., Xu, X. et al. (2020) Haplotyping the Vitis collinear core genome with rhAmpSeq improves marker transferability in a diverse genus. Nat Commun, 11, 413.
7.	Vondras, A.M., Minio, A., Blanco-Ulate, B., Figueroa-Balderas, R., Penn, M.A., Zhou, Y., Seymour, D., Ye, Z., Liang, D., Espinoza, L.K. et al. (2019) The genomic diversification of grapevine clones. BMC Genomics, 20, 972.
8.	Badouin, H., Velt, A., Gindraud, F., Flutre, T., Dumas, V., Vautrin, S., Marande, W., Corbi, J., Sallet, E., Ganofsky, J. et al. (2020) The wild grape genome sequence provides insights into the transition from dioecy to hermaphroditism during grape domestication. Genome Biol, 21, 223.
9.	Tello-Ruiz, M.K., Naithani, S., Gupta, P., Olson, A., Wei, S., Preece, J., Jiao, Y., Wang, B., Chougule, K., Garg, P. et al. (2021) Gramene 2021: harnessing the power of comparative genomics and pathways for plant research. Nucleic Acids Res, 49, D1452-D1463.
10.	Papatheodorou, I., Moreno, P., Manning, J., Fuentes, A.M., George, N., Fexova, S., Fonseca, N.A., Fullgrabe, A., Green, M., Huang, N. et al. (2020) Expression Atlas update: from tissues to single cells. Nucleic Acids Res, 48, D77-D83.
11.	Naithani, S., Gupta, P., Preece, J., D'Eustachio, P., Elser, J.L., Garg, P., Dikeman, D.A., Kiff, J., Cook, J., Olson, A. et al. (2020) Plant Reactome: a knowledgebase and resource for comparative pathway analysis. Nucleic Acids Res, 48, D1093-D1103.



