<!-----
NEW: Check the "Suppress top comment" option to remove this info from the output.

Conversion time: 6.7 seconds.


Using this Markdown file:

1. Paste this output into your source file.
2. See the notes and action items below regarding this conversion run.
3. Check the rendered output (headings, lists, code blocks, tables) for proper
   formatting and use a linkchecker before you publish this page.

Conversion notes:

* Docs to Markdown version 1.0β29
* Tue Jun 15 2021 09:39:50 GMT-0700 (PDT)
* Source doc: Sorghum bicolor BTx623 - Quick tutorial
* This document has images: check for >>>>>  gd2md-html alert:  inline image link in generated source and store images to your server. NOTE: Images in exported zip file from Google Docs may not appear in  the same order as they do in your doc. Please check the images!

----->


<p style="color: red; font-weight: bold">>>>>>  gd2md-html alert:  ERRORs: 0; WARNINGs: 0; ALERTS: 15.</p>
<ul style="color: red; font-weight: bold"><li>See top comment block for details on ERRORs and WARNINGs. <li>In the converted Markdown or HTML, search for inline alerts that start with >>>>>  gd2md-html alert:  for specific instances that need correction.</ul>

<p style="color: red; font-weight: bold">Links to alert messages:</p><a href="#gdcalert1">alert1</a>
<a href="#gdcalert2">alert2</a>
<a href="#gdcalert3">alert3</a>
<a href="#gdcalert4">alert4</a>
<a href="#gdcalert5">alert5</a>
<a href="#gdcalert6">alert6</a>
<a href="#gdcalert7">alert7</a>
<a href="#gdcalert8">alert8</a>
<a href="#gdcalert9">alert9</a>
<a href="#gdcalert10">alert10</a>
<a href="#gdcalert11">alert11</a>
<a href="#gdcalert12">alert12</a>
<a href="#gdcalert13">alert13</a>
<a href="#gdcalert14">alert14</a>
<a href="#gdcalert15">alert15</a>

<p style="color: red; font-weight: bold">>>>>> PLEASE check and correct alert issues and delete this message and the inline alerts.<hr></p>


**_Sorghum bicolor_ BTx623**

_Types of data available for this variant and tips on how to find it._

**Querying a_ Sorghum bicolor_ BTx623 Gene**

Search Results Summary

The results of a gene search are organized in five data categories:



*   Location - Genomic context
    *   Gramene
    *   Pangenome site
    *   phytozome
*   Expression - Baseline gene expression from [EBI-Expression Atlas](https://www.ebi.ac.uk/gxa)
*   Homology - Customizable Ensembl Compara gene family trees with displays:
    *   Alignment Overview
    *   Multiple-Sequence Alignment
    *   Neighborhood Conservation View
*   Pathways - Association to pathways in [Gramene’s Plant Reactome](https://plantreactome.gramene.org)
*   Xrefs - Cross-references to other databases

The example below shows the default view of the results when searching for the [SORBI_3006G095600](https://dev.sorghumbase.org/genes?filters={%22status%22:%22init%22,%22operation%22:%22AND%22,%22negate%22:false,%22marked%22:false,%22leftIdx%22:0,%22rightIdx%22:3,%22children%22:[{%22fq_field%22:%22id%22,%22fq_value%22:%22SORBI_3006G095600%22,%22name%22:%22SORBI_3006G095600%22,%22category%22:%22Gene%22,%22leftIdx%22:1,%22rightIdx%22:2,%22negate%22:false,%22showMenu%22:false,%22marked%22:true}],%22showMarked%22:true,%22showMenu%22:false,%22moveCopyMode%22:%22%22,%22searchOffset%22:0,%22rows%22:20}&genomes=) gene (msd2).



<p id="gdcalert1" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image1.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert2">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>


![alt_text](images/image1.png "image_tooltip")


**Gene page**

A [gene page](https://ensembl.sorghumbase.org/Sorghum_bicolor/Gene/Summary?g=SORBI_3006G095600;r=6:46566240-46571064), [transcript page](https://ensembl.sorghumbase.org/Sorghum_bicolor/Transcript/Summary?db=core;g=SORBI_3006G095600;r=6:46566240-46571064;t=OQU81659), [(SNP) variant page](https://ensembl.sorghumbase.org/Sorghum_bicolor/Variation/Explore?db=core;g=SORBI_3006G095600;r=6:46566240-46571064;t=OQU81659;v=tmp_6_47487513_G_A;vdb=variation;vf=7758800), and [structural variant page](https://ensembl.sorghumbase.org/Sorghum_bicolor/StructuralVariation/Explore?db=core;g=SORBI_3006G095600;r=6:46566240-46571064;sv=nsv856002;svf=54084;t=OQU81659;v=tmp_6_47487513_G_A;vdb=variation;vf=7758800) (tabs) in the Ensembl browser.



<p id="gdcalert2" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image2.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert3">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>


![alt_text](images/image2.png "image_tooltip")


<p id="gdcalert3" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image3.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert4">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>


![alt_text](images/image3.png "image_tooltip")


**Comparative Genomics**

The following comparative genomics data for BTx623 are available in the SorghumBase genome browser:



*   Gene trees (Ensembl Compara pipeline) - [Example](http://ensembl.sorghumbase.org/Sorghum_bicolor/Gene/Compara_Tree?g=SORBI_3006G095600;r=6:46566240-46571064;collapse=2831659,2829950,2830021)



<p id="gdcalert4" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image4.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert5">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>


![alt_text](images/image4.png "image_tooltip")




*   Genome-wide alignment to _Oryza sativa_ Japonica (see [example region](http://ensembl.sorghumbase.org/Sorghum_bicolor/Location/Multi?align=23;db=core;g=SORBI_3006G095600;r=6:46563088-46605099;r1=4:22284737-22339738:1;s1=Oryza_sativa)).



<p id="gdcalert5" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image5.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert6">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>


![alt_text](images/image5.png "image_tooltip")




*   Synteny maps with:
    *   Rio
    *   TX430
    *   TX2783
    *   TX436
    *   _Zea mays _V4 (see [example](http://ensembl.sorghumbase.org/Sorghum_bicolor/Location/Synteny?r=6%3A46563088-46605099&db=core&g=SORBI_3006G095600&otherspecies=Zea_maysb73v4))
    *   _Oryza sativa _Japonica



<p id="gdcalert6" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image6.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert7">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>


![alt_text](images/image6.png "image_tooltip")


BTx623

Chr 6


    **Regulation**

**Epigenomics**

Genomewide epigenetic marks related to root system morphology and architecture in sorghum BTx623 deposited under BioProject [PRJNA454504](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA454504/) (Gladman et al, manuscript in prep.). Three repetitions available for each of seven samples: leaf lower (vegetative whorl), leaf upper (growing anthesis), panicle (upper anthesis), root bottom (anthesis), root top (vegetative), stem 1 cm (vegetative) and stem mid internode (anthesis).



*   The [example](https://ensembl.sorghumbase.org/Sorghum_bicolor/Location/View?db=core;g=SORBI_3006G095600;r=6:46566240-46571064) below shows two samples with very similar patterns of epigenetic variation for the msd2 gene region.



<p id="gdcalert7" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image7.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert8">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>


![alt_text](images/image7.png "image_tooltip")



#### 
    **Baseline Gene Expression (Atlas)**

Baseline gene expression data from seven sorghum BTx623 datasets curated and processed by the [EMBL-EBI Expression Atlas](https://www.ebi.ac.uk/gxa/) [[(Emms et al. 2016; Makita et al. 2015; Davidson et al. 2012; Turco et al. 2017; Wang et al. 2018; Olson et al. 2014)](https://paperpile.com/c/H8kp7d/bA0bX+t7q1R+8Eil0+YI4gS+7omsp+1dzar) and BioProject [PRJNA293229](https://www.ncbi.nlm.nih.gov//bioproject/PRJNA293229)/[SRP062564](https://trace.ncbi.nlm.nih.gov/Traces/sra?study=SRP062564) by Thurber et al (2015)].



*   The [example](https://ensembl.sorghumbase.org/Sorghum_bicolor/Gene/ExpressionAtlas?db=core;g=SORBI_3004G141800;r=4:41625307-41663480;t=OQU84910;v=S4_41077528;vdb=variation;vf=112207) below shows Baseline gene expression for the msd2 gene. 



<p id="gdcalert8" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image8.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert9">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>


![alt_text](images/image8.png "image_tooltip")



### 
    **Variation**


#### Variation in SorghumBase is available for short variants (genetic variation, which in turn may be naturally occurring or chemically induced), longer than 50 nucleotides (structural variants) and QTL variants associated with physical traits. 


#### 
    **Genetic Variation**

Genetic variation data for a sorghum gene is available graphically and in tabular form, and for each variant, a Variant page provides more detailed information. Below are provided examples of each of these data representations.



*   An [image](https://ensembl.sorghumbase.org/Sorghum_bicolor/Transcript/Variation_Transcript/Image?db=core;g=SORBI_3006G095600;r=6:46566240-46571064;t=OQU81659;v=tmp_6_47486074_T_C;vdb=variation;vf=3821694) of all the genetic variants mapping to the smaller transcript of the msd2 gene.



<p id="gdcalert9" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image9.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert10">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>


![alt_text](images/image9.png "image_tooltip")
 



*   A partial [table](https://ensembl.sorghumbase.org/Sorghum_bicolor/Transcript/Variation_Transcript/Table?db=core;g=SORBI_3006G095600;r=6:46566240-46571064;t=OQU81659;v=tmp_6_47486074_T_C;vdb=variation;vf=3821694) of all the genetic variants mapping to the smaller transcript of the msd2 gene.

    <p style="text-align: right">


<p id="gdcalert10" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image10.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert11">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>


<img src="images/image10.png" width="" alt="alt_text" title="image_tooltip">
</p>



<p style="text-align: right">


<p id="gdcalert11" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image11.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert12">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>


<img src="images/image11.png" width="" alt="alt_text" title="image_tooltip">
</p>




*   An example of a [Variant summary page](https://ensembl.sorghumbase.org/Sorghum_bicolor/Variation/Explore?db=core;g=SORBI_3006G095600;r=6:46566240-46571064;t=OQU81659;v=tmp_6_47487513_G_A;vdb=variation;vf=7758800) for a chemically induced variant (tmp_6_47487513_G_A) with a smaller inset of the [genotype frequency panel](https://ensembl.sorghumbase.org/Sorghum_bicolor/Variation/Population?db=core;g=SORBI_3006G095600;r=6:46566240-46571064;t=OQU81659;v=tmp_6_47487513_G_A;vdb=variation;vf=7758800#14_tablePanel) showing the accession (ARS105) bearing the mutation (G|A).



<p id="gdcalert12" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image12.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert13">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>


![alt_text](images/image12.png "image_tooltip")


<p style="text-align: right">


<p id="gdcalert13" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image13.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert14">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>


<img src="images/image13.png" width="" alt="alt_text" title="image_tooltip">
</p>



#### 
    **Naturally occurring genetic variation** - Single Nucleotide Polymorphisms (SNPs)


#### 
    Currently in SorghumBase, there are two SNPs data sets for sorghum BTx623: 



*   The Morris SNP dataset [(Morris et al. 2013)](https://paperpile.com/c/H8kp7d/YVPG) consists of ~265,000 SNPs genotyped in 378 accessions from the US sorghum association panel (SAP).
*   The Mace SNP dataset [(Mace et al. 2013)](https://paperpile.com/c/H8kp7d/SDiz) includes almost 6.5 million SNPs genotyped in 45 _Sorghum bicolor_ lines including the BTx623 reference genome plus 2 _S. propinquum_ lines reported by Mace and colleagues [(Mace et al. 2013)](https://paperpile.com/c/H8kp7d/SDiz). The data was obtained by resequencing the genomes of the 44 _S. bicolor_ lines representing the primary gene pool and spanning dimensions of geographic origin, end-use and taxonomic group (i.e., major races of cultivated _S. bicolor_, landraces, improved inbreds, progenitors, wild and weedy), and the first resequenced genome of _S. propinquum_, all of which were mapped to the BTx623 S. _bicolor_ reference genome.

#### 
    **Chemically induced variation** -  Ethyl methanesulfonate (EMS)-induced mutations



#### Currently in SorghumBase, there is one collection of EMS-induced mutant lines. EMS is a chemical commonly used to cause point mutations, that is, to change single nucleotides in the DNA of a plant seed. 



*   The Xin EMS dataset [(Jiao et al. 2016)](https://paperpile.com/c/H8kp7d/13nP) includes almost 1.5 million EMS-induced G/C to A/T transition mutations annotated from 252 M3 families selected from the 6,400 sorghum mutant library in BTx623 background described by Xin and colleagues [(Xin et al. 2008)](https://paperpile.com/c/H8kp7d/jJZy). Genomic DNA used for sequencing was pooled from 20 M3 plants per M2 family [(Jiao et al. 2016)](https://paperpile.com/c/H8kp7d/13nP). 

### 
    **Structural Variation**



#### Data for structural variation for the sorghum BTx623 reference has been imported from the[ Database of Genomic Variants Archive](http://www.ebi.ac.uk/dgva/) (dGVA) from a single study containing around 28 thousand structural variations described by [(Zheng et al. 2011)](https://paperpile.com/c/H8kp7d/oxfd). [Here is an example](https://ensembl.sorghumbase.org/Sorghum_bicolor/StructuralVariation/Explore?db=core;r=1:96756-143685;sv=nsv838266;svf=32426;vdb=variation).



<p id="gdcalert14" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image14.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert15">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>


![alt_text](images/image14.png "image_tooltip")



### 
    **Phenotypic Variation - **Quantitative Trait Locus (QTLs)** **


#### Data corresponding to 5,843 QTL features for 220 sorghum traits were imported from[ Sorghum QTL Atlas](https://aussorgm.org.au/sorghum-qtl-atlas/) and are provided with predicted syntenic locations in maize and rice.



*   [Example region](https://ensembl.sorghumbase.org/Sorghum_bicolor/Location/View?db=core;g=SORBI_3006G095600;r=7:61190510-61277060;sv=nsv856002;svf=54084;t=OQU81659;vdb=variation) with QTLs associated with multiple traits including greenbug resistance, fresh biomass, and flag leaf height. Hint: For additional regions with QTL data in the current sorghum assembly (v.3), use the[ physical or genetic (cM) coordinates](http://aussorgm.org.au/dev/wp-content/uploads/2018/08/Consensus-Map_cM-and-bp-coordinates.xlsx) kindly provided by the Sorghum QTL Atlas team.

#### 
    

<p id="gdcalert15" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image15.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert16">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>


![alt_text](images/image15.png "image_tooltip")



    **Literature References**


    Gladman, N. et al.  “Sorghum root epigenetic landscape during limiting phosphorus conditions.” _Manuscript in preparation._


    [Aken, Bronwen L., Sarah Ayling, Daniel Barrell, Laura Clarke, Valery Curwen, Susan Fairley, Julio Fernandez Banet, et al. 2016. “The Ensembl Gene Annotation System.” Database: The Journal of Biological Databases and Curation 2016 (June). https://doi.org/10.1093/database/baw093.](http://paperpile.com/b/H8kp7d/D11I)


    [Casa, Alexandra M., Gael Pressoir, Patrick J. Brown, Sharon E. Mitchell, William L. Rooney, Mitchell R. Tuinstra, Cleve D. Franks, and Stephen Kresovich. 2008. “Community Resources and Strategies for Association Mapping in Sorghum.” Crop Science 48 (1): 30–40.](http://paperpile.com/b/H8kp7d/9fn8y)


    [Davidson, Rebecca M., Malali Gowda, Gaurav Moghe, Haining Lin, Brieanne Vaillancourt, Shin-Han Shiu, Ning Jiang, and C. Robin Buell. 2012. “Comparative Transcriptomics of Three Poaceae Species Reveals Patterns of Gene Expression Evolution.” The Plant Journal: For Cell and Molecular Biology 71 (3): 492–502.](http://paperpile.com/b/H8kp7d/8Eil0)


    [Emms, David M., Sarah Covshoff, Julian M. Hibberd, and Steven Kelly. 2016. “Independent and Parallel Evolution of New Genes by Gene Duplication in Two Origins of C4 Photosynthesis Provides New Insight into the Mechanism of Phloem Loading in C4 Species.” Molecular Biology and Evolution 33 (7): 1796–1806.](http://paperpile.com/b/H8kp7d/bA0bX)


    [Goodstein, David M., Shengqiang Shu, Russell Howson, Rochak Neupane, Richard D. Hayes, Joni Fazo, Therese Mitros, et al. 2012. “Phytozome: A Comparative Platform for Green Plant Genomics.” Nucleic Acids Research 40 (Database issue): D1178–86.](http://paperpile.com/b/H8kp7d/OkGO)


    [Jiao, Yinping, John Burke, Ratan Chopra, Gloria Burow, Junping Chen, Bo Wang, Chad Hayes, Yves Emendack, Doreen Ware, and Zhanguo Xin. 2016. “A Sorghum Mutant Resource as an Efficient Platform for Gene Discovery in Grasses.” The Plant Cell 28 (7): 1551–62.](http://paperpile.com/b/H8kp7d/13nP)


    [Mace, Emma S., Shuaishuai Tai, Edward K. Gilding, Yanhong Li, Peter J. Prentis, Lianle Bian, Bradley C. Campbell, et al. 2013. “Whole-Genome Sequencing Reveals Untapped Genetic Potential in Africa’s Indigenous Cereal Crop Sorghum.” Nature Communications 4: 2320.](http://paperpile.com/b/H8kp7d/SDiz)


    [Makita, Yuko, Setsuko Shimada, Mika Kawashima, Tomoko Kondou-Kuriyama, Tetsuro Toyoda, and Minami Matsui. 2015. “MOROKOSHI: Transcriptome Database in Sorghum Bicolor.” Plant & Cell Physiology 56 (1): e6.](http://paperpile.com/b/H8kp7d/t7q1R)


    [McCormick, Ryan F., Sandra K. Truong, Avinash Sreedasyam, Jerry Jenkins, Shengqiang Shu, David Sims, Megan Kennedy, et al. 2018. “The Sorghum Bicolor Reference Genome: Improved Assembly, Gene Annotations, a Transcriptome Atlas, and Signatures of Genome Organization.” The Plant Journal: For Cell and Molecular Biology 93 (2): 338–54.](http://paperpile.com/b/H8kp7d/Vv7a)


    [Morris, Geoffrey P., Punna Ramu, Santosh P. Deshpande, C. Thomas Hash, Trushar Shah, Hari D. Upadhyaya, Oscar Riera-Lizarazu, et al. 2013. “Population Genomic and Genome-Wide Association Studies of Agroclimatic Traits in Sorghum.” Proceedings of the National Academy of Sciences of the United States of America 110 (2): 453–58.](http://paperpile.com/b/H8kp7d/YVPG)


    [Olson, Andrew, Robert R. Klein, Diana V. Dugas, Zhenyuan Lu, Michael Regulski, Patricia E. Klein, and Doreen Ware. 2014. “Expanding and Vetting Sorghum Bicolor Gene Annotations through Transcriptome and Methylome Sequencing.” The Plant Genome 7 (2): lantgenome2013.08.0025.](http://paperpile.com/b/H8kp7d/1dzar)


    [Paterson, A. H., J. E. Bowers, R. Bruggmann, I. Dubchak, J. Grimwood, H. Gundlach, G. Haberer, et al. 2009. “The Sorghum Bicolor Genome and the Diversification of Grasses.” Nature 457 (7229): 551–56.](http://paperpile.com/b/H8kp7d/5g7D)


    [Turco, Gina M., Kaisa Kajala, Govindarajan Kunde-Ramamoorthy, Chew-Yee Ngan, Andrew Olson, Shweta Deshphande, Denis Tolkunov, et al. 2017. “DNA Methylation and Gene Expression Regulation Associated with Vascularization in Sorghum Bicolor.” The New Phytologist 214 (3): 1213–29.](http://paperpile.com/b/H8kp7d/YI4gS)


    [Wang, Bo, Michael Regulski, Elizabeth Tseng, Andrew Olson, Sara Goodwin, W. Richard McCombie, and Doreen Ware. 2018. “A Comparative Transcriptional Landscape of Maize and Sorghum Obtained by Single-Molecule Sequencing.” Genome Research 28 (6): 921–32.](http://paperpile.com/b/H8kp7d/7omsp)


    [Xin, Zhanguo, Ming Li Wang, Noelle A. Barkley, Gloria Burow, Cleve Franks, Gary Pederson, and John Burke. 2008. “Applying Genotyping (TILLING) and Phenotyping Analyses to Elucidate Gene Function in a Chemically Induced Sorghum Mutant Population.” BMC Plant Biology. https://doi.org/10.1186/1471-2229-8-103.](http://paperpile.com/b/H8kp7d/jJZy)


    [Zheng, Lei-Ying, Xiao-Sen Guo, Bing He, Lian-Jun Sun, Yao Peng, Shan-Shan Dong, Teng-Fei Liu, et al. 2011. “Genome-Wide Patterns of Genetic Variation in Sweet and Grain Sorghum (Sorghum Bicolor).” Genome Biology 12 (11): R114.](http://paperpile.com/b/H8kp7d/oxfd)
